# -*- coding: utf-8 -*-
import numpy as np
import matplotlib.pyplot as plt
import nltk
import re
from nltk.stem.lancaster import LancasterStemmer

import string
from gensim import corpora, models, similarities

from collections import defaultdict, Counter
from __builtin__ import True


from sklearn import manifold

STOPLIST = set(nltk.corpus.stopwords.words('english'))
STEMMER = nltk.stem.porter.PorterStemmer()

def tokenize(documents, frequency_threshold = 1, stemmed = False, export_chunck = True, grammar = r'kT: {(<JJ>* <NN.*>+ <IN>)? <JJ>* <NN.*>+}', ngram = 1):
    
    tokens = [[w for w in nltk.word_tokenize(document.lower().translate(None, string.punctuation)) if not w in STOPLIST] for document in documents]
    
    if stemmed:
        tokens = [[STEMMER.stem(w) for w in token] for token in tokens] 
    
    if frequency_threshold > 0:
        temp = sum(tokens, [])
        all_tokens = set(stem for stem in set(temp) if temp.count(stem) == frequency_threshold)
        tokens = [[w for w in token if w not in all_tokens] for token in tokens] 

    return tokens


class NLPModel:
    
    STOPLIST = set(nltk.corpus.stopwords.words('english'))
    STEMMER = nltk.stem.porter.PorterStemmer()
    DIC_PATH = '/Users/sephon/Downloads/nlp/deerwester.dict'
    
    DEFAULT_FREQ_THRE = 1
    DEFAULT_STEMMED = False
    DEFAULT_EXPORT_CHUNCK = False
    DEFAULT_GRAMMAR = r'kT: {(<JJ>* <NN.*>+ <IN>)? <JJ>* <NN.*>+}'
    DEFAULT_NGRAM = 1
    
    tfidf = None
    dictionary = None
    similarity_index = None
    
    def __init__(self):
        self.dictionary = None
    
    def create_similarity_index_by_corpus(self, corpus):
        self.similarity_index = similarities.SparseMatrixSimilarity(self.tfidf[corpus], num_features=len(self.dictionary))
        
    ###
    def create_models(self, documents, frequency_threshold = DEFAULT_FREQ_THRE, stemmed = DEFAULT_STEMMED, export_chunck = DEFAULT_EXPORT_CHUNCK, grammar = DEFAULT_GRAMMAR, ngram = DEFAULT_NGRAM):
        
        texts = self.tokenize(documents,frequency_threshold = frequency_threshold, stemmed = stemmed, export_chunck = export_chunck, grammar = grammar, ngram = ngram)

        # Create dictionary
        self.creat_dictionary(texts)
        corpus = self.text2corpus(texts)
        
        # Create TF-IDF model
        self.create_tfidf_model_from_corpus(corpus)
        # Create TF-IDF similarity model
        self.create_similarity_index_by_corpus(corpus)
        

    def get_similarity_score_from_corpus(self, corpus):
        
        sims = self.similarity_index[self.tfidf[corpus]]
        for s in sims:
            s = sorted(enumerate(s), key=lambda item: -item[1])
        
    ####
    def get_similarity_score(self, documents, frequency_threshold = DEFAULT_FREQ_THRE, stemmed = DEFAULT_STEMMED, export_chunck = DEFAULT_EXPORT_CHUNCK, grammar = DEFAULT_GRAMMAR, ngram = DEFAULT_NGRAM):
        try:
            corpus = self.document2corpus(documents, frequency_threshold = frequency_threshold, stemmed = stemmed, export_chunck = export_chunck, grammar = grammar, ngram = ngram)
            sims = self.similarity_index[self.tfidf[corpus]]
            return sims
             
        except:
            print "Unavailable models"
    
    @ staticmethod
    def sort_similarity_score(sims):  
        similarities = []
        for i, s in enumerate(sims):
            s = sorted(enumerate(s), key=lambda item: -item[1])
            similarities.append(s)
        return similarities
        
    def create_tfidf_model_from_corpus(self, corpus):
        self.tfidf = models.TfidfModel(corpus)
        
    def create_tfidf_model(self, documents, frequency_threshold = DEFAULT_FREQ_THRE, stemmed = DEFAULT_STEMMED, export_chunck = DEFAULT_EXPORT_CHUNCK, grammar = DEFAULT_GRAMMAR, ngram = DEFAULT_NGRAM):
        if self.dictionary is not None:
            corpus = self.document2corpus(documents, frequency_threshold = frequency_threshold, stemmed = stemmed, export_chunck = export_chunck, grammar = grammar, ngram = ngram)
            self.tfidf = models.TfidfModel(corpus)
        else:
            print "Unavailable Dictionary"
        
    def get_tfidf_corpus(self, corpus):
        return self.tfidf[corpus]
        
    def creat_dictionary(self, texts):
        self.dictionary = corpora.Dictionary(texts)
#         for d in self.dictionary:
#             print self.dictionary[d]
        
    def save_dictionary(self, path):
        if self.dictionary is not None:
            self.dictionary.save(path)
            
    def load_dicionary(self, path):
        self.dictionary = corpora.Dictionary.load(path)
        
    def document2corpus(self, documents, frequency_threshold = DEFAULT_FREQ_THRE, stemmed = DEFAULT_STEMMED, export_chunck = DEFAULT_EXPORT_CHUNCK, grammar = DEFAULT_GRAMMAR, ngram = DEFAULT_NGRAM):
        
        texts = self.tokenize(documents, frequency_threshold = frequency_threshold, stemmed = stemmed, export_chunck = export_chunck, grammar = grammar, ngram = ngram)
        return self.text2corpus(texts)
    
    def text2corpus(self, texts):
        return [self.dictionary.doc2bow(text) for text in texts]
        
        
    @staticmethod
    def tokenize(documents, frequency_threshold = DEFAULT_FREQ_THRE, stemmed = DEFAULT_STEMMED, export_chunck = DEFAULT_EXPORT_CHUNCK, grammar = DEFAULT_GRAMMAR, ngram = DEFAULT_NGRAM):
    
        tokens = [[w for w in nltk.word_tokenize(document.lower().translate(None, string.punctuation)) if not w in STOPLIST] for document in documents]
        
        if stemmed:
            tokens = [[STEMMER.stem(w) for w in token] for token in tokens] 
        
        if frequency_threshold > 0:
            temp = sum(tokens, [])
            all_tokens = set(stem for stem in set(temp) if temp.count(stem) == frequency_threshold)
            tokens = [[w for w in token if w not in all_tokens] for token in tokens] 
    
        return tokens


tt = ["Human machine interface for lab abc, computer applications in new york",
            "A survey of user opinion of computer; system response time",
            "The EPS user interface management system",
            "System and human system engineering testing of EPS",
            "Relation of user perceived response time to error measurement",
            "The generation of random binary unordered trees",
            "The intersection graph of paths in trees",
            "Graph minors IV Widths of trees and well quasi ordering",
            "Graph minors A survey"
            ]

viziometrics = ["Scientific results are communicated visually in the literature through diagrams, visualizations, and photographs. These information-dense objects have been largely ignored in bibliometrics and scientometrics studies when compared to citations and text. In this paper, we use techniques from computer vision and machine learning to classify more than 8 million figures from PubMed into 5 figure types and study the resulting patterns of visual information as they relate to impact. We find that the distribution of figures and figure types in the literature has remained relatively constant over time, but can vary widely across field and topic.  Remarkably, we find a significant correlation between scientific impact and the use of visual information, where higher impact papers tend to include more diagrams, and to a lesser extent more plots.  To explore these results and other ways of extracting this visual information, we have built a visual browser to illustrate the concept and explore design alternatives for supporting viziometric analysis and organizing visual information. We use these results to articulate a new research agenda -- viziometrics -- to study the organization and presentation of visual information in the scientific literature.",
                "Figure 1  VizioMetrics.org system overview. We store the images in Amazon S3 service. Image paths, figure captions,paper metadata and classification result are stored in the database. The figure analysis system acquires the file keys from the database, downloads the image files, and feeds them into the figure processing pipeline. The final classification results are stored in the database as the sources for the application prototype.",
                "Figure 2  Multi-chart figure dismantling. The figure shows the intermediate steps for dismantling multi-chart figures. The splitting algorithm recursively segments the raw images into several sub-images. The merging algorithm then aggregates auxiliary fragments with nearby standalone figures to produce the final segmentation.",
                "Figure 3. Recognizing multi-chart images. After splitting the figure into distinct blocks, the dismantling algorithm marks the effective figure regions (EFR) then downsamples the EFR into n x n blocks that form a n^2 x 1 feature vector. These vectors are used to train the classifier.",
                "Figure 4. The distribution of figure types across journals show an emphasis on plots and diagrams relative to tables, and identify visualization-heavy venues such as Cell Deat and Disease. We considered the top 49 highest-impact journals in PMC that had at least 850 papers available in the corpus, where impact is measured as Article Influence (AI) (the black bar). Each stacked bar shows the average density of each figure type across all papers published in the journal. The density of a figure type is the number of instances of that type divided by the page count. The category Others contains 288,953 papers from other journals.", 
                "Figure 5. Figure distribution by research topic show that microbiology topics tend to emphasize visual presentation of ideas. Topics were determined by the journal categories in Thomson Reuters JCR.We show the highest-impact 49 topics that have at least 1000 papers, where impact is the average of all papers assigned to that category. The category Others includes 216,380 papers from other topics and papers without topic labels.",
                "Figure 6. The distribution of figure types in the PMC corpus over time. The top figure shows the number of papers increasing dramatically in the mid-2000s, which can be explained by a change in sponsor rules: NIH required authors to submit their papers to PMC. The hump of impact between 1997 and 2005 may be attributable to author bias in voluntarily uploading their highest-impact papers. After 2006, the increasing uses of plots and tables may be attributable to increased emphasis on data-intensive research. The density of photos and diagrams are consistently flat over time. The bottom plot provides context: the average page length per paper over time, and the number of papers in the corpus over time.", 
                "Figure 7. We choose five specific journals for closer inspection: Nature (highest impact), Cell Death and Disease (highest figure density), British Medical Journal (lowest figure density), Genome Biology (unusually low proportion of photos) and PLoS One (largest number of papers). Nature, Cell Death and Disease and Genome Biology exhibit a recent increase in plots-per-page, consistent with the overall trend. We conjecture that the articles in these high-impact journals are becoming more data-centric. Moreover, Nature and especially Cell Death and Disease show a heavy use of figures, in part because these journals tend to have greater proportions of multi-chart figures (67% for Nature and 82% for Cell Death and Disease relative to 30% for the entire image age set.) The British Medical Journal shows a different trend in which figure density gradually decrease; the mechanism behind this trend is unclear. PLoS One shows no siginficant change from its launch in 2006.",
                "Figure 8. Impact versus (A) figure density and (B) proportion of figures. We rank papers by ALEF and group them into 4 bins. Papers with the same Eigenfactor are grouped into the same set. Any two papers with Eigenfactor difference within 1E-12 are regarded as having the same impact, which is why the bins are not evenly distributed. For each set, we average the densities and proportions of 4 figure types.", 
                "Figure 9. The user interface of the VizioMetrics.org search engine. Result figures are either arranged via (a) the brick-wall layout or (b) a conventional layout bundling figures with literature title. Figures are labeled by different colors based on their types. (c) Clicking figures will show article details such as authors, abstract, figure captions, hyperlink to full PDFs and related figures. We also provide a verification form to encourage user verifying our machine-labelled figure type and help us gather more ground-truth label.",
                "In this study, we intend to launch a new field of study called viziometrics that extends  prior work in bibliometrics and scientometrics but focuses on the role of visual information encodings. We developed a figure processing pipeline that automatically classifies figures into equations, diagrams, plots, photos, and tables. By integrating the figure-type labels and article metadata, we analyzed the patterns across journals, over time, and relationships to impact.  In different disciplines, we found that the role of the five figure types can vary widely. For instance, clinical papers tend to have higher photo density and computational papers tend to have higher diagram and plot density. In respect of visual patterns over time, we found a growing use of plots, perhaps suggesting increasing emphasis on data-intensive methods.  Our key result is that high-impact papers tend to have more diagrams per page and a higher proportion of diagrams relative to other figure types. A possible interpretation is that clarity is critical for impact: illustrating an original idea may be more influential than quantitative experimental results. We also described a new application to search and browse scientific figures, potentially enabling new kinds of search tasks. The Viziometrics.org systems affords search by keyword as well as figure type, and shows results in a figure-centric layout. We believe more interesting and useful applications can be inspired by the concept of viziometrics. We also encourage people to use our publicly available corpus and software to explore this area of research and create a new community of interest."]

www = ["We present VizioMetrix, a platform that extracts visual information from the scientific literature and makes it available for use in new information retrieval applications and for studies that look at patterns of visual information across millions of papers.  New ideas are conveyed visually in the scientific literature through figures --- diagrams, photos, visualizations, tables --- but these visual elements remain ensconced in the surrounding paper and difficult to use directly to facilitate information discovery tasks or longitudinal analytics.  Very few applications in information retrieval, academic search, or bibliometrics make direct use of the figures, and none attempt to recognize and exploit the type of figure, which can be used to augment interactions with a large corpus of scholarly literature. The VizioMetrix platform processes a corpus of documents, classifies the figures, organizes the results into a cloud-hosted databases, and drives three distinct applications to support bibliometric analysis and information retrieval. The first application supports information retrieval tasks by allowing rapid browsing of classified figures.  The second application supports longitudinal analysis of visual patterns in the literature and facilitates data mining of these figures.  The third application supports crowdsourced tagging of figures to improve classification, augment search, and facilitate new kinds of analyses. Our initial corpus is the entirety of PubMed Central (PMC), and will be released to the public alongside this paper; we welcome other researchers to make use of these resources.", 
       "Screenshot of search engine interface (viziometrics.org). We use different colors to highlight different figure types (e.g., red indicates diagrams) (A) shows the grid layout, which is designed for reviewing many images (B) shows the alternative layout, which bundles figures from the same paper and related papers.  Related papers are selected based on out and in-citations and then ranked by the ALEF score.  This is made to look more like a paper, whereas the grid layout provides a general overview on a particular topic. (c) is the page showing figure and paper details. A simple crowdsourced labelling interface is embedded in the page to gather human labels.",
       "Screenshot of the bulk-labelling interface. (A) Instructions for using the interface. (B) Indicator for the type of figure the user is asked to label (e.g., photos). (C) The total number figures that have been labelled by the user. (D) Controls to allow users to label all figures directly. (E) Controls to allow users to refresh the pool of figures. (F) submits the result. (G) Zoom control to afford fluid inspection of figures and groups of figures.",
       "Scholarly impact versus figure density. We rank PMC papers by ALEF and group them into 4 tiers: top 5%, 5th to 25th percentile, 25th to 50th percentile, and the bottom 50%. Any two papers with ALEF difference within 1E-12 are regarded as having equal impact and forced to be in the same bin. Hence, the boundary between 2nd tier and 3rd tier shifts to 23th percentile (papers in 23th to 25 percentile have equal ALEF) and the last bin increases to 55% (papers in bottom 55% have ALEF of zero), where are the closest positions to 25th and 50% respectively. For each tier, we average the densities (total number of figure per paper page) of 4 figure categories: table, photo, plot, and diagram. We ignore equations here because they are text. The top 5% paper tend to have statistically higher densities of plots and diagrams.",
       "VizioMetrix's Architecture. The gray box illustrates the processing system on which VizioMetrix is based.  This includes a data pipeline, which parses articles files, classifies figures, and calculates the article influence.  The data pipeline then pushes the results into the VizioMetrix database. The blue box lists the three applications powered by the database.",
       "We present VizioMetrix, a platform for mining millions of figures from the biomedical sciences.  Our hope is that the platform will catalyze future research for improving scholarly search and facilitating large-scale analysis of these figures and new figure-centric applications.  VizioMetrix provides a figure-oriented search service for general academic users and an open data resource for researches interested in mining scholarly figures. We provide a data processing pipeline that extracts the metadata from scholarly papers, extracts the figures and classifies them into different types. We also develop a crowdsourced labelling application for further labeling and subsequent improvement of our machine learning methods and methods of others.  This platform is needed since mechanical turkers do not usually have the domain knowledge for labeling figure types.  We plan to assign a Document Object Identifier (DOI) to our dataset and making it freely available to the general public.  We hope this platform reduces the activation energy needed to analyze scholarly figures at scale and provokes new and exciting questions."]

# http://journals.plos.org/plosone/article?id=info%3Adoi%2F10.1371%2Fjournal.pone.0157941
#strike: figure 7  10,9,7,3,6,8,4,1,2,5
PLOS_1 = ["Although several studies are devoted to determining the diversity of Antarctic heterobranch sea slugs, new species are still being discovered. Among nudibranchs, Doto antarctica Eliot, 1907 is the single species of this genus described from Antarctica hitherto, the type locality being the Ross Sea. Doto antarctica was described mainly using external features. During our Antarctic research on marine benthic invertebrates, we found D. antarctica in the Weddell Sea and Bouvet Island, suggesting a circumpolar distribution. Species affiliation is herein supported by molecular analyses using cytochrome c oxidase subunit I, 16S rRNA, and histone H3 markers. We redescribe D. antarctica using histology, micro-computed tomography (micro-CT), and 3D-reconstruction of the internal organs. Moreover, we describe a new, sympatric species, namely D. carinova Moles, Avila & Wagele n. sp., and provide an anatomical comparison between the two Antarctic Doto species. Egg masses in both species are also described here for the first time. We demonstrate that micro-CT is a useful tool for non-destructive anatomical description of valuable specimens. Furthermore, our high resolution micro-CT data reveal that the central nervous system of both Doto species possesses numerous accessory giant cells, suggested to be neurons herein. In addition, the phylogenetic tree of all Doto species sequenced to date suggests a scenario for the evolution of the reproductive system in this genus: bursa copulatrix seems to have been reduced and the acquisition of a distal connection of the oviduct to the nidamental glands is a synapomorphy of the Antarctic Doto species. Overall, the combination of thorough morphological and anatomical description and molecular analyses provides a comprehensive means to characterize and delineate species, thus suggesting evolutionary scenarios.",
        "Table 1. Sampling stations where D. antarctica and D. carinova Moles, Avila & Wagele n. sp. were collected.",
        "Fig 1. Photographs of D. antarctica (left column: A,C,E,G) and D. carinova Moles, B Live picture right after collection, showing the D. carinova n. sp. spawning on top of the gorgonian Primnoisis antarctica (Isididae).C-D Lateral and dorsal view of the preserved animals. E-F Ventral view of the preserved animals. G-H Lateral view of the preserved egg masses.",
        "Fig 2. Micro-CT reconstructions of the internal organs of D. antarctica (left column) and D. carinova Moles, Avila & Wagele n. sp. (right column). A-B Right antero-lateral view of all reconstructed organs. C-D Left antero-lateral view of the circulatory, digestive, excretory, and nervous systems. E-F Right lateral view of the male reproductive system. G-H Left lateral view of the reproductive system (mucus gland is not depicted here since it covers the whole view). am ampulla; adiv ampulla diverticulum; au auricle; bc bursa copulatrix; cis circulatory sinuses; cgl capsule gland; cpg cerebropleural ganglion; dg distal gonoduct; dgdiv digestive gland diverticula; dgl digestive gland (only depicted in D. carinova n. sp.); gc giant cells; gon gonad; ht heart; int intestine; kid kidney; megl membrane gland; mugl mucus gland; oes oesophagus; od odontophore; ot oral tube; ov oviduct; peg pedal ganglion; pen penis; per pericardium; pg proximal gonoduct; pha pharynx; pro prostate; rad radula; rhg rhinophoral ganglion; rhi rhinophore; rhs rhinophoral sheath; sgl salivary gland; sin sinus; sp sphincter; sto stomach; vag vagina; vd vas deferens; ven ventricle.",
        "Fig 3. Histological slides of the glandular structures in D. antarctica. A Epidermis of the rhinophoral sheath. B Detail of one cerata tubercle showing defensive glandular cells (white punctuation in live animals). C Oral glands. D Giant neurones chain attached to the cerebropleural ganglion; a thin cortex, large nucleus, and nucleolus of each cell can be seen. E Detail of the prostate glandular cells. F Detail of the capsule glandular cells. G Detail of the membrane glandular cells. H Detail of the glandular mucus cells. gc giant cells; lum lumen; muc glandular mucus cell; n nucleus; nu nucleolus; ogl oral glands; pha pharynx; rhn rhinophoral nerve; sin sinus vessel; sgl salivary gland; smc specialised multivacuolised cell; wgc white glandular cells.",
        "Fig 4. Scanning electron micrographs (SEM) of the radular teeth of D. antarctica.",
        "Fig 5. Left antero-lateral view of the micro-CT reconstruction of the nervous system of D. carinova Moles, Avila & Wagele n. sp. cpg cerebropleural ganglion; gc giant cells; peg pedal ganglion; rhg rhinophoral ganglion; sin sinus.",
        "Fig 6. Histological slide of outer surface of the egg mass in Doto carinova Moles, Avila & Wagele n. sp.",
        "Table 2. Differential characters among D. carinova Moles, Avila & Wagele n. sp. and D. antarctica.",
        "Fig 7. Phylogenetic tree of Doto species based on the combined COI, 16S, and H3 genes using Bayesian inference (BI) and maximum-likelihood (ML). Numbers on nodes indicate posterior probability values (BI) and bootstrap support values (ML). Specimens sequenced are in bold; Doto antarctica specimens are coloured in red. Schematic drawings of the reproductive system of Doto species are depicted (A-D), as well as the unsequenced D. uva Marcus, 1955 (E). bc bursa copulatrix; gd gonoduct; nigl nidamental glands; pen penis; rs receptaculum seminis; vd vas deferens.",
        "New Dotidae species have been usually described based only on external morphological and radular characters. Nonetheless, internal organ organisation and egg mass structure is desirable for describing Doto species. Micro-CT and histology has demonstrated to be very useful techniques to reconstruct the internal anatomy of these two Doto species. Two new occurrences of D. antarctica were recorded in Bouvet Island and the eastern Weddell Sea. These specimens are morphologically and genetically characterised herein and appeared related to D. antarctica from the Ross Sea, which strongly suggests a circumpolar distribution. We also described D. carinova n. sp. occurring in sympatry with D. antarctica in the Weddell Sea. Although some distinguishing characters can be size-related, the lower number of tubercles on the cerata, the different form of the rhinophoral sheath, the shape and arrangement of the salivary glands, ampulla, and prostate in the large specimen of D. carinova n. sp., as well as differences in the egg masses and cnidarian substrate indicate separate evolutionary lineages. A phylogenetic hypothesis including various species of Doto from various regions showed a trend towards the reduction of bursa copulatrix and distal connection of oviduct to the nidamental glands with separate pathways for eggs and allosperm. Furthermore, we identified and described the nervous system of Doto species that contains accessory giant cells that might represent neurones with neuronal/secretory function. Future studies may unravel the properties and function of these peculiar giant cells. Moreover, further studies should revisit and check the identification of Doto species collected in former Antarctic cruises, because the present study provides new characters that may allow distinguishing among species.",
        ]

# http://journals.plos.org/plosone/article?id=info%3Adoi%2F10.1371%2Fjournal.pone.0158665 
#strike figure 2  1,2,4,3
PLOS_2 = ["Grunt-like calls are present in the vocal repertoire of many group-living mammals and seem to facilitate social interactions between lower and higher-ranking members. The white-lipped peccary (Tayassu pecari) lives in stable hierarchical mixed-sex groups and like non-human primates, usually emits grunt-like calls following aggressive interactions, mainly during feeding contexts. We investigated the possible functions of peccaries' grunt-like calls and their relationship to the individuals' social rank, identity, and sexual dimorphism. We observed that low-ranking individuals emitted grunt-like calls more often than high-ranking ones, and that the alpha male never emitted this vocalization. Moreover, the mean minimum frequency of grunt-like calls decreased as the peccary's rank increased. The findings revealed differences among individual grunts, but the low accuracy of cross-validation (16%) suggests that individual recognition in peccaries may be less important than an honest signal of individual social status. In addition, the absence of differences in the acoustic parameters of grunt-like calls between males and females points to the lack of sexual dimorphism in this species. We verified that after hearing grunt calls, dominant opponents were more likely to cease attacking a victim, or at least delay the continuation of conflict, probably decreasing the severity of agonistic interactions. Our findings are particularly important to improve the current understanding of the role of grunt-like calls in herd-living mammals with linear dominant hierarchies, and strongly suggest that they are involved in the maintenance of herd social stability and cohesion.",
          "Fig 1. Grunt call emitted by a white-lipped peccary (Tayassu pecari).",
          "Table 1. Mean values (standard deviation) of the acoustic parameters for grunts emitted by white-lipped peccaries according to the ranking position. Lower numbers correspond to higher positions in the dominance hierarchy. All measurements were taken from 50 samples from each individual. The percentage of correct cross-classified grunt calls for each individual based on the conventional (DFA) and permuted discriminant analysis (pDFA) is also presented, based on 37 calls for each individual.",
          "Fig 2. Distribution of discriminant scores for each individual along the two canonical discriminant functions.",
          "Table 2. Coefficients of the two main discriminant functions (DF) indicating the relative contribution of each acoustic parameter (variable) for discriminating between individuals' grunt calls. The values in bold indicate the parameters with higher loading for the discriminant functions (DF1 and DF2)."]
 
# http://journals.plos.org/plosone/article?id=info%3Adoi%2F10.1371%2Fjournal.pone.0159035
# strike figure 1  3,1,2,5,4
PLOS_3 = ["Throughout human history, slow-renewal biological resource populations have been predictably overexploited, often to the point of economic extinction. We assess whether and how this has occurred with timber resources in the Brazilian Amazon. The asynchronous advance of industrial-scale logging frontiers has left regional-scale forest landscapes with varying histories of logging. Initial harvests in unlogged forests can be highly selective, targeting slow-growing, high-grade, shade-tolerant hardwood species, while later harvests tend to focus on fast-growing, light-wooded, long-lived pioneer trees. Brazil accounts for 85% of all native neotropical forest roundlog production, and the State of Para for almost half of all timber production in Brazilian Amazonia, the largest old-growth tropical timber reserve controlled by any country. Yet the degree to which timber harvests beyond the first-cut can be financially profitable or demographically sustainable remains poorly understood. Here, we use data on legally planned logging of ~17.3 million cubic meters of timber across 314 species extracted from 824 authorized harvest areas in private and community-owned forests, 446 of which reported volumetric composition data by timber species. We document patterns of timber extraction by volume, species composition, and monetary value along aging eastern Amazonian logging frontiers, which are then explained on the basis of historical and environmental variables. Generalized linear models indicate that relatively recent logging operations farthest from heavy-traffic roads are the most selective, concentrating gross revenues on few high-value species. We find no evidence that the post-logging timber species composition and total value of forest stands recovers beyond the first-cut, suggesting that the commercially most valuable timber species become predictably rare or economically extinct in old logging frontiers. In avoiding even more destructive land-use patterns, managing yields of selectively-logged forests is crucial for the long-term integrity of forest biodiversity and financial viability of local industries. The logging history of eastern Amazonian old-growth forests likely mirrors unsustainable patterns of timber depletion over time in Brazil and other tropical countries.",
          "Fig 1. Map of the state of Para, the second largest in Brazil, showing the geographic distribution of 446 AUTEF management plans for which the species composition of timber stocks were declared based on local forest inventories. Color gradient shows the total fraction of low-value timber species (price class D) by volume (m3) declared within each management plan. Standing volumetric stocks of low-value timber species, which comprise the cheapest lumber in local and regional markets, and represent the strongest negative predictor of concession scale logging revenues per unit area. Forest and deforested areas as of 2012 are indicated in green and purple, respectively.",
          "Fig 2. Relationship between volumetric proportions of timber species aggregated by price brackets (classes A, B, C, and D) and mean gross revenues per hectare expected from each authorized concession area. Timber value classes are ordered top to bottom from the highest (A) to the lowest (D). Symbols are colour-coded from low (blue) to high (red) values according to the respective proportion of each price class contributing to the overall timber revenue of each concession site. Solid lines represent a smoother ( = 0.8) running through all data points.",
          "Fig 3. Coefficient estimates (95% confidence intervals) showing the magnitude and direction of effects of different forest site and landscape scale variables on timber revenue (R$/ha), species selectivity ranked by timber prices (J' evenness), and the volumetric genus-level composition of timber offtake (NMDS1). Concession-scale timber revenue, tree species selectivity, and volumetric composition of offtakes were modelled with generalized linear models using the following variables: AREA-net concession areas authorized for timber extraction; % FOREST-percentage of forest cover within a 10-km buffer outside concessions; ROAD DISTANCE-linear distance between each concession and the nearest heavy-traffic paved road; BASAL AREA-predicted pre-logging forest basal area based on an interpolation of 2,345 one-hectare plots from the RADAMBRASIL forest inventory program; and FRONTIER AGE-number of years since the onset of large-scale timber exploitation. Explanatory variables were standardized prior to analyses. For full summary of averaged models, see Table B in S1 File.",
          "Fig 4. Relationships between linear distances to major access roads and timber revenue (R$/ha), species selectivity ranked by timber prices (J' evenness), and the volumetric genera composition of timber offtake (NMDS1). Higher revenues per ha (R$/ha) and higher levels of species selectivity (lower Pielou's J-evenness values) are observed in forest management plans farther away from heavy-use roads. Multivariate patterns of volumetric composition at the genus level (NMDS axis 1), which was inversely correlated with volumetric densities of low-value timber, was strongly positively related to distances from heavy-use roads.",
          "Fig 5. Relationships between the baseline abundance of timber species within contrasting market price classes at four major logging frontiers examined in this study, in terms of the total proportional basal area density (m2/ha) and total volume density (m3/ha) per 1-ha plot sampled in the early 1970s by the RADAMBRASIL forest inventory program. High-value and low-value timber species are aggregated within price classes A + B and C + D, respectively."] 

# http://journals.plos.org/plosone/article?id=10.1371%2Fjournal.pone.0159770
# strike fig 8   1,8,6,2,7,5,3,4
PLOS_4 = ["Helminth infections and nutrition can independently alter the composition and abundance of the gastrointestinal microbiota, however, their combined effect is poorly understood. Here, we used the T. retortaeformis-rabbit system to examine how the helminth infection and host restriction from coprophagy/ready-to-absorb nutrients affected the duodenal microbiota, and how these changes related to the acquired immune response at the site of infection. A factorial experiment was performed where the bacterial community, its functionality and the immune response were examined in four treatments (Infect, Infect+Collar, Control+Collar and Control). Helminths reduced the diversity and abundance of the microbiota while the combination of parasites and coprophagic restriction led to a more diversified and abundant microbiota than infected cases, without significantly affecting the intensity of infection. Animals restricted from coprophagy and free from parasites exhibited the richest and most abundant bacterial community. By forcing the individuals to absorb nutrients from less digested food, the coprophagic restriction appears to have facilitated the diversity and proliferation of bacteria in the duodenum. Changes in the microbiota were more clearly associated with changes in the immune response for the infected than the nutrient restricted animals. The functional and metabolic characteristics of the duodenal microbiota were not significantly different between treatments. Overall, infection and diet affect the gut microbiota but their interactions and outcome can be complex. These findings can have important implications for the development of control measures to helminth infections where poor nutrition/malnutrition can also be a concern.",
          "Fig 1. Abundance and alpha diversity of the duodenum microbiota. Abundance and diversity by treatment (Infect: I, Infect+Collar: IC, Control+Collar: CC and Control: C) at the 3 taxonomic levels (time points pooled together). The mean, 25-75% percentiles, maximum and minimum and outliers from the Shannon diversity index are reported. Diversity is significantly lower in I than in CC or C groups. The number of taxa is similar among treatments.",
          "Fig 2. Relative microbiota abundance by animal. Abundance at the 3 taxonomic levels in the four treatments (Infect: I, Infect+Collar: IC, Control+Collar: CC and Control: C). Only the taxa with a relative abundance greater than 1% at each taxonomic level are presented. The white space that adds up to 100% abundance should be interpreted as 'other' taxa.",
          "Fig 3. Expression of cytokine, transcription factor and function genes in the duodenal mucosa. Mean values (2-ct s.e.) by treatment (Infect: I, Infect+Collar: IC, Control+Collar: CC and Control: C) and sampling time (day post initial infection). Data have been standardized by the housekeeping gene HPRT and the mean values from the baseline animals sampled at day 0. Most of the variables show significant differences among treatments but not sampling time, higher values are observed in the I and IC groups.",
          "Fig 4. Antibody IgA from the duodenal mucus. Mean (s.e.) Optical Density (O.D.) for the species-specific (left y-axis, full cycle) and the total (right y-axis, full square) IgA by treatment (Infect: I, Infect+Collar: IC, Control+Collar: CC and Control: C) and sampling time (days post initial infection). Both specific and total IgA are higher in I and IC than in CC or C.",
          "Fig 5. Relationships between microbiota and immune response. Spearman rank correlations grouping animals by treatment (Infect: I, Infect+Collar: IC, Control+Collar: CC, and Control: C) and using only the bacteria that showed a between-individual coefficient of variation (CV) greater than 150. Negative relationships are in grades of blue while positive relationships are in grades of red; a correlation coefficient quantifies each grade, null correlations (0) are depicted in white; significant correlations (p<0.05) are highlighted with a star. Significant correlations are more common in the Infect group than the remaining treatments.",
          "Fig 6. Relationships between microbiota and T. retortaeformis abundance. Spearman rank correlations grouping animals by treatment (Infect -I, Infect+Collar -IC), or treatment and time point, and: Panel A- the bacteria families with CV>150 or Panel B- the functions at the kEGG pathway module 2 that exhibited a coefficient of variation CV among individuals greater than 70,000. Additional details on the color-coding of the correlations are reported in Fig 5, while the activities related to the kEGG function IDs are listed in Fig 7. No significant correlations are detected, except in two cases.",
          "Fig 7. Relative abundance of gene functionality in the duodenum by animal. Data are presented by treatment (Infect: I, Infect+Collar: IC, Control+Collar: CC, and Control: C) at the kEGG pathway level 2. Only the functions with a relative abundance greater than 1% were displayed. The white space that adds up to 100% abundance should be considered as 'other' functions. The following function IDs and related activities were selected: CellProc1 = Cellular Processes: Cell Motility; CellProc2 = Cellular Processes: Transport and Catabolism; EnvInfo1 = Environmental Information: Processing Membrane Transport; EnvInfo2 = Environmental Information: Processing Signal Transduction; GenInfo1 = Genetic Information: Processing Folding Sorting and Degradation; GenInfo2 = Genetic Information: Processing Replication and Repair; GenInfo3 = Genetic Information: Processing Transcription; GenInfo4 = Genetic Information: Processing Translation; Metab1 = Metabolism: Amino Acid Metabolism; Metab2 = Metabolism: Biosynthesis of Other Secondary Metabolites; Metab3 = Metabolism: Carbohydrate Metabolism; Metab4 = Metabolism: Energy Metabolism; Metab5 = Metabolism: Enzyme Families; Metab6 = Metabolism: Glycan Biosynthesis and Metabolism; Metab7 = Metabolism: Lipid Metabolism; Metab8 = Metabolism: Metabolism of Cofactors and Vitamins; Metab9 = Metabolism: Metabolism of Other Amino Acids; Metab10 = Metabolism: Metabolism of Terpenoids and Polyketides; Metab11 = Metabolism: Nucleotide Metabolism; Metab12 = Metabolism: Xenobiotics Biodegradation and Metabolism.",
          "Fig 8. Relationships between gene functionality and immune response. Spearman rank correlations grouping animals by treatment (Infect: I, Infect+Collar: IC, Control+Collar: CC, and Control: C) and using the kEGG pathway level 2 functions with a CV>70,000. Additional details are reported in Figs 5 and 7. Significant correlations are mainly observed with a type 1 response."]

# http://journals.plos.org/plosone/article?id=10.1371%2Fjournal.pone.0156905
# figure 4     1,4,2,3
PLOS_5 = ["Eukaryotic DNA is strongly bent inside fundamental packaging units: the nucleosomes. It is known that their positions are strongly influenced by the mechanical properties of the underlying DNA sequence. Here we discuss the possibility that these mechanical properties and the concomitant nucleosome positions are not just a side product of the given DNA sequence, e.g. that of the genes, but that a mechanical evolution of DNA molecules might have taken place. We first demonstrate the possibility of multiplexing classical and mechanical genetic information using a computational nucleosome model. In a second step we give evidence for genome-wide multiplexing in Saccharomyces cerevisiae and Schizosacharomyces pombe. This suggests that the exact positions of nucleosomes play crucial roles in chromatin function.",
          "Fig 1. Nucleosomal DNA model with bp step dependent mechanical properties. (A) The rigid base-pair model is forced, using 28 constraints (indicated by red spheres), into a lefthanded superhelical path that mimics the DNA conformation in the nucleosome crystal structure [4]. (B) Fraction of dinucleotides GC and AA/TT/TA at each position along the nucleosome model found in 10 million high affinity sequences produced by MMC at 100 k. The solid and dashed lines indicate minor and major groove bending sites; the nucleosome dyad is at 0 bp. The model recovers the basic nucleosome positioning rules [1, 3]. (C) Same as (B), but on top of 1200 coding sequences (produced by sMMC). The same periodic signals are found albeit with a smaller amplitude.",
          "Fig 2. Mechanical energy landscape along a 500 bp stretch of the YAL002W gene of S. cerevisiae.(A) Elastic energy of the nucleosome model as a function of position obtained from a Monte Carlo simulation at 50 k. (B) Effective energy including excluded volume between nucleosomes. In both, (A) and (B), the vertical lines indicate experimentally determined nucleosome positions from the unique nucleosome map [25]. (C) The top graph shows a fraction of the original landscape from (A), the five landscapes below are produced via sMMC with the nucleosome positioned at the corresponding dashed vertical line. The minima can be shifted freely on top of genes, proving that multiplexing is possible.",
          "Fig 3. Mechanisms underlying multiplexing.(A) Energy landscape (black dashed curve) of a sequence with highly optimized nucleosome affinity at -5 bp, produced by MMC at very low temperature (15 k). The colored curves are landscapes for three synonymous mutants (three different codon frames) that are optimized via sMMC for high affinity at position 0. The maximum cannot be turned into a minimum in this case, signaling that multiplexing would not be possible on genomes if they were selected for highest nucleosome affinity. (B) Distribution of AA, TT and TA dinucleotides around minor groove bending site -25 bp for the shifted nucleosome from Fig 2C bottom. Dashed blue curves: natural preferences (attained through MMC), red curves: distribution obtained from sMMC on that particular stretch of the YAL002W gene; both simulations are performed at 100 k. Though not optimal, sMMC brings in AA at a position close to its preferred position (amplitude almost 1) indicating the plasticity of the mechanical code.",
          "Fig 4. Multiplexing in two eukaryotic genomes. (A) Normalized Fourier amplitudes for the distribution of the synonymous codons for threonine along nucleosomes on top of genes (purple curve) and for the distribution of the corresponding trinucleotides along nucleosomes outside genes (green curve) [25]. The peaks at 10 bp (indicated by an arrow) are due to nucleosome positioning that appears weaker on top of genes but might signal multiplexing instead. (B) Same as (A), but for S. pombe [26]. (C) The normalized 10 bp amplitude inside vs. outside genes of all 20 amino acids for the two yeast species. The arrows indicate threonine. All points below the line have smaller amplitudes inside genes, a hallmark of multiplexing.",]
# http://journals.plos.org/plosone/article?id=10.1371%2Fjournal.pone.0159813
# strike figure 8     6,9,2,4,3,1,8,5,7
PLOS_6 = ["It is important to account for the movement behaviour of fishes when designing effective marine protected areas (MPAs). Fish movements occur across different spatial and temporal scales and understanding the variety of movements is essential to make correct management decisions. This study describes in detail the movement patterns of an economically and commercially important species, Diplodus sargus, within a well-enforced Mediterranean MPA. We monitored horizontal and vertical movements of 41 adult individuals using passive acoustic telemetry for up to one year. We applied novel analysis and visualization techniques to get a comprehensive view of a wide range of movements. D. sargus individuals were highly territorial, moving within small home ranges (< 1 km2), inside which they displayed repetitive diel activity patterns. Extraordinary movements beyond the ordinary home range were observed under two specific conditions. First, during stormy events D. sargus presented a sheltering behaviour, moving to more protected places to avoid the disturbance. Second, during the spawning season they made excursions to deep areas (> 50 m), where they aggregated to spawn. This study advances our understanding about the functioning of an established MPA and provides important insights into the biology and management of a small sedentary species, suggesting the relevance of rare but important fish behaviours.",
          "Fig 1. Study area and the acoustic receiver array. Black triangles correspond to acoustic receivers placed in May 2007 and white triangles to the receivers placed in September 2008. Dotted circles represent the average detection range (150 m). Red bold letters stand for the Diplodus sargus capture locations: TAS, Tascons; POR, Portitxol; ARQ: Arquets; FAL: Falaguer. The topographic base map (1:5.000) and the sea bottom bionomic map [35] are freely accessible through the Institut Cartografic i Geologic de Catalunya (www.icgc.cat) under Creative Commons Attribution License (CC BY 4.0).",
          "Fig 2. Daily presence-absence plot of tagged Diplodus sargus within the monitored area. Colours represent the different protection zones of the MPA; Red: no-take zone (NT), Blue: partial reserve (PR), Green: no-reserve (NR). Fish tag and release dates (blue circles), expected transmitter battery dead dates (red squares) and the final withdrawal of the receiver array (vertical green line) are shown.",
          "Table 1. Summary of the monitoring information of tagged Diplodus sargus. DD: Detection Days; TP: total Tracking Period (d); RI: Residence Index, CA: Core Area; HR: Home Range.",
          "Fig 3. Mean spatial Utilization Distribution estimates of Diplodus sargus, for each capture location. The concentric polygons represent, from inside out, the areas covering the 50% (core area), 75%, 90%, and 95% (home range) of the volume of the Utilization Distribution computed by the Brownian Bridge Movement Model. The topographic base map (1:5.000) is freely accessible through the Institut Cartografic I Geologic de Catalunya (www.icgc.cat) under Creative Commons Attribution License (CC BY 4.0).",
          "Fig 4. Utilization Distribution Overlap Index (UDOI) calculated for each pair of Diplodus sargus individuals. Yellow blocks represent high UDOIs observed when comparing individuals captured in the same location. Green and blue blocks represent lower degrees of overlap and the purple areas represent no-overlapping HRs.",
          "Fig 5. Spatial chronogram plot for one tagged Diplodus sargus individual (#14), showing a clear diel movement pattern. This individual was observed at receivers 5, 6 and 7 during the day (green areas) and moved to receivers 9 and 10 during the night. Some punctual excursions to receivers 12, 13 and 14 are visible in December, March and April. White areas represent 30 min intervals with no receptions. See Fig 1 for the placement of the receivers.",
          "Fig 6. Distribution of phase-transition values (PT) calculated for the mean depth between consecutive day-night (A) and day-day (B) phases, for each Diplodus sargus individual. Lower and upper boundaries of the boxes represent the first and third quantiles of value distributions. Significant results of the Wilcoxon signed-rank tests against the null hypothesis of a median of zero are expressed by the symbols above the boxes; +: p < 0.05; *: p < 0.01.",
          "Fig 7. Daily mean sea surface temperature and wave height (A) and percentages of fish observed at different depths (B) and distances from their HR boundary (C). Red lines in the back point out days in which extraordinary activities were detected by the cluster analysis. Dotted black rectangles highlight periods with recurrent high wave conditions.",
          "Fig 8. Metric multidimensional scaling analysis depicting similarities between days in fish occurrence at different depths and distances-to-HR. The two main groups of days detected by the cluster analysis (ordinary and extraordinary) are shown in different colours."]

# http://journals.plos.org/plosone/article?id=10.1371%2Fjournal.pone.0159967
# figure 5 **   4,7,1,8,5,2,6,3
PLOS_7 = ["The aim of this study was to develop ethosomes loaded with cryptotanshinone (CPT) and formulate them as a topical gel for the treatment of acne. Ethosomes were prepared and evaluated for vesicle size, CPT loading and encapsulation efficiency. Optimized ethosomes were formulated as Carbomer 974 gels and compared with conventional hydroethanolic gels for transdermal permeation and skin deposition in vitro. The anti-acne activity and skin irritation of the gel was investigated in rabbits. Optimized ethosomes had an average vesicle size of 69.1 + 1.9 nm with CPT loading and encapsulation efficiency of 0.445 + 0.007 mg/mL and 40.31 + 0.67%, respectively. The transdermal flux and skin deposition of the optimized ethosomal gel were 2.5- and 2.1-times those of conventional gels. The ethosomal gel revealed better anti-acne effect with only slight skin irritation. This study demonstrates that ethosomal formulation is an effective dermal delivery system for CPT, and that CPT ethosomal gels are promising future acne treatments.",
          "Table 1. Formulation of ethosomes.",
          "Fig 1. Images of cryptotanshinone loaded ethosomes.",
          "Table 2. Characterization of ethosomes.",
          "Fig 2. In vitro transdermal profiles (A) and skin deposition (B) of ethosomal gel and conventional gel, (n = 6) **, P<0.01.",
          "Table 3. Transdermal parameters of ethosomal and conventional gels through excised pig skin (n = 6).",
          "Fig 3. Influence of cryptotanshinone loaded ethosomal gel on epidermis thickness (A) and lymphatic cell numbers in a radon area (10 um x 10 um) of dermis (B). #P>0.05, *P<0.05, **P<0.01.",
          "Fig 4. Effect of cryptotanshinone loaded ethosomal gel on the histopathology in the rabbit ear acne model induced by oleic acid. A: normal rabbit ear, B: acne model treated by blank ethosomal gel, C: conventional cryptotanshinone gel, D: cryptotanshinone loaded ethosomal gel. (400x).",
          "Fig 5. Photomicrographs (400x) of hematoxylin-eosin stained normal rabbit back skin (A), rabbit skin treated with blank ethosomal gel (B) and rabbit skin treated with cryptotanshinone loaded ethosomal gel (C)."
          ]

# http://journals.plos.org/plosone/article?id=10.1371%2Fjournal.pone.0159867
# figure 6   1,7,4,3,5,6,2
PLOS_8 = ["Dihydrotestosterone (DHT) is regarded as the most potent natural androgen and is implicated in the development and progression of castration resistant prostate cancer (CRPC). Under castrate conditions, DHT is produced from the metabolism of the adrenal androgen precursors, DHEA and androstenedione. Recent studies have shown that the adrenal steroid 11B-hydroxyandrostenedione (11OHA4) serves as the precursor to the androgens 11-ketotestosterone (11kT) and 11-ketodihydrotestosterone (11kDHT). In this study we comprehensively assess the androgenic activity of 11kT and 11kDHT. This is the first study, to our knowledge, to show that 11kT and 11kDHT, like T and DHT, are potent and efficacious agonists of the human androgen receptor (AR) and induced both the expression of representative AR-regulated genes as well as cellular proliferation in the androgen dependent prostate cancer cell lines, LNCaP and VCaP. Proteomic analysis revealed that 11kDHT regulated the expression of more AR-regulated proteins than DHT in VCaP cells, while in vitro conversion assays showed that 11kT and 11kDHT are metabolized at a significantly lower rate in both LNCaP and VCaP cells when compared to T and DHT, respectively. Our findings show that 11kT and 11kDHT are bona fide androgens capable of inducing androgen-dependant gene expression and cell growth, and that these steroids have the potential to remain active longer than T and DHT due to the decreased rate at which they are metabolised. Collectively, our data demonstrates that 11kT and 11kDHT likely play a vital, but overlooked, role in the development and progression of CRPC.",
          "Fig 1. Biosynthesis of 11kT and 11kDHT from the adrenal androgen precursor 11OHA4. Enzymes: 11BHSD2, 11B-hydroxysteroid dehydrogenase; 17BHSD2, 17B-hydroxysteroid dehydrogenase; SRD5A1, steroid 5a-reductase type 1; 3aHSD2, 3a-hydroxysteroid dehydrogenase. Steroids: 11OHA4, 11B-hydroxyandrostenedione; 11kA4, 11-ketoandrostenedione; 11kT, 11-ketotestosterone; 11OH-5a-dione, 11OH-5a-androstanedione; 11k-5a-dione, 11-keto-5a-androstanedione; 11kDHT, 11-ketodihydrotestosterone; 11OHAST, 11B-hydroxyandrosterone; 11kAST, 11-ketoadrenosterone; 11k-3a-adiol, 11-keto-5a-androstane-3a,17B-diol.",
          "Fig 2. Binding of DHT, T, 11kDHT and 11kT to the human AR (A) and transactivation via an ARE (B and C). Binding affinities, agonist potencies and efficacies of DHT, 11kDHT, T and 11kT relative to the synthetic AR agonist mibolerone are summarised in (D). Whole cell binding assays (A) were conducted in COS-1 cells transiently transfected with pSVARo. Cells were incubated with 0.2 nM [3H]-Mib in the absence and presence of increasing concentrations of either unlabelled Mib, DHT, 11kDHT, T and 11kT for 16 hours. Results are plotted as % specific binding where the total specific binding of [3H]-Mib only is set to 100% and binding of unlabelled steroid is set as a % binding relative to that. Whole cell binding results are shown as means + SEM of three independent experiments performed in triplicate. Transactivation assays (B and C) where performed in COS-1 cells transiently transfected with the pSVARo expression vector and the 4xSC ARE1.2-luc reporter. Agonist activity was measured by incubating cells in the presence of increasing concentrations of either Mib, DHT, T, 11kDHT or 11kT for 24 h. Induction is shown as % luciferase activity expressed in relative light units (rlu's), with the maximal response of Mib (10-5 M) set to 100%. Luciferase assays are shown as means + SEM of six independent experiments performed in quadruplicate.",
          "Fig 3. Induction of AR-regulated gene expression in LNCaP cells by DHT, 11kDHT and 11kT. Cells were incubated with CS-FCS-supplemented media for 24 hours prior to treatment with 1 or 10 nM steroid for an additional 24 hours prior to analysis by qPCR. Gene expression was calculated relative to the geometric mean of the reference genes ALAS and PBGD. Fold change over vehicle was calculated using the method described by Pfaffl et al [48]. Results are shown as means + SEM of three independent experiments performed in triplicate. Data from individual experiments was all normalized using log transformation and mean-centering prior to analysis [49].",
          "Fig 4. Induction of AR-regulated gene expression in VCaP cells by DHT, 11kDHT, T and 11kT. Cells were incubated with CS-FCS supplemented media for 24 hours prior to treatment with 1 or 10 nM steroid for an additional 24 hours prior to analysis by qPCR. Gene expression was calculated relative to the geometric mean of the reference genes ALAS and PBGD and are expressed as the fold change over the vehicle control. Results are shown as means + SEM of three independent experiments performed in triplicate. Data from individual experiments was all normalized using log transformation and mean-centering prior to analysis [49].",
          "Table 1. Regulation of AR-regulated proteins by DHT, 11kDHT, T and 11kT in VCaP cells. Cells were incubated with CS-FCS supplemented media for 48 hours prior to treatment with 1 nM steroid. Proteins were subsequently identified using mass spectrometry. Fold changes were calculated relative to the vehicle control. Statistically significant changes are indicated (P<0.05). Results are representative of three independent experiments.",
          "Fig 5. Induction of cell proliferation in LNCaP and VCaP cells by DHT, 11kDHT, T and 11kT. Cells were incubated with media supplemented with CS-FCS for 24 hours prior to treatment with 0.1, 1 or 10 nM steroids. Resazurin assays were carried out on day 7 (LNCaP) or day 10 (VCaP) after treatment. Results are shown as means + SEM of three independent experiments with eight replicates each.",
          "Fig 6. Metabolism of DHT, 11kDHT, T and 11kT by LNCaP and VCaP cells. Steroids were analysed by ultra-performance convergence chromatography-mass spectrometry (UPC2-MS/MS). Results are representative of two independent experiments performed in triplicate."
          ]

# http://journals.plos.org/plosone/article?id=10.1371%2Fjournal.pone.0159865
# figure 5    4,2,5,6,1,7,3,8
PLOS_9 = ["In soybean, flavonoid 3'5'-hydroxylase (F3'5'H) and dihydroflavonol-4-reductase (DFR) play a crucial role in the production of anthocyanin pigments. Loss-of-function of the W1 locus, which encodes the former, or W3 and W4, which encode the latter, always produces white flowers. In this study, we searched for new genetic components responsible for the production of white flowers in soybean and isolated four white-flowered mutant lines, i.e., two Glycine soja accessions (CW12700 and CW13381) and two EMS-induced mutants of Glycine max (PE1837 and PE636). F3'5'H expression in CW12700, PE1837, and PE636 was normal, whereas that in CW13381 was aberrant and missing the third exon. Sequence analysis of F3'5'H of CW13381 revealed the presence of an indel (~90-bp AT-repeat) in the second intron. In addition, the F3'5'H of CW12700, PE1837, and PE636 harbored unique single-nucleotide substitutions. The single nucleotide polymorphisms resulted in substitutions of amino acid residues located in or near the SRS4 domain of F3'5'H, which is essential for substrate recognition. 3D structure modeling of F3'5'H indicated that the substitutions could interfere with an interaction between the substrate and heme group and compromise the conformation of the active site of F3'5'H. Recombination analysis revealed a tight correlation between all of the mutant alleles at the W1 locus and white flower color. On the basis of the characterization of the new mutant alleles, we discussed the biological implications of F3'5'H and DFR in the determination of flower colors in soybean.",
          "Fig 1. Schematic representation of the simplified biosynthesis pathway for anthocyanins. Gene loci are indicated in white letters with a black highlight. ANS, anthocyanin synthase; CHS, chalcone synthase; CHI, chalcone isomerase; DFR, dihydroflavonol-4-reductase; F3H, flavanone 3-hydroxylase; F3'H, flavonoid 3'-hydroxylase; F3'5'H, flavonoid 3'5'-hydroxylase.",
          "Fig 2. Photographic images showing flower colors of the plant materials. IT182932 (purple flower), CW12700 (white), and CW13381 (white) are Glycine soja accessions. EMS mutant lines, PE1837 (white) and PE636 (white), were isolated from the progeny of Glycine max 'Pungsannamul' (purple) treated with EMS. (A) A whole flower. (B) A banner petal.",
          "Table 1. Segregation and co-segregation analyses.",
          "Fig 3. Expression profiles of W1, W3, and W4 and schematic representation of different F3'5'H alleles from Pungsannamul. (A) RT-PCR analysis of W1, W3, and W4 in purple-flowered lines (IT182932 and Pungsannamul) and white-flowered lines (CW12700, CW13381, PE1837, and PE636). GmActin, a housekeeping gene, was used as the loading control. M, a 1-kb molecular marker. (B) 3'-Rapid amplification of cDNA ends (RACE) analysis of the W1 locus from IT182932 and CW13381. (C) Mutational changes in the recessive alleles at the W1 locus are indicated in white boxes. Exons are indicated in grey boxes. W1-U and W1-L denote the 5' and 3' parts of W1, respectively. GS-F was used in 3'-RACE analysis as the gene specific forward primer.",
          "Fig 4. Amino acid alignment of F3'5'H proteins and 3D-modeling simulation of the effect of amino acid substitutions on structures of F3'5'H. (A) GenBank accession numbers of F3'5'H proteins are as follows: Camellia sinensis, AAY23287; Clitoria ternatea, BAF49293; Cyclamen persicum, ACX37698; Gossypium hirsutum, ACH56524; Iochroma calycinum, AIY22750; Lycium ruthenicum, AGT57963; Medicago truncatula, XP_013459330; Nekemias grossedentata, AGO02173; Petunia occidentalis, BAF34571; Petunia reitzii, BAF34572; Pisum sativum, ADW66160; Ribes nigrum, AGI16385; Vitis amurensis, ACN38269; and Vitis vinifera, BAE47007. Identical amino acids are in grey. Relevant amino acid substitutions are indicated in white letters with a black highlight. The thick black box indicates the SRS4 domain. N (Asn) - k (Lys) in w1-s1; A (Ala) - T (Thr) in w1-p1; and S (Ser) - F (Phe), and A (Ala) - D (Asp) in w1-p2. (B) 3D-model structure of the F3'5'H active site consisting of the SRS domains, a heme (green), and a flavone (yellow). (C) 3D-model structure showing the N300k substitution in CW12700 (w1-s1). The Asn300 residue is shown in the magenta stick, and the Lys300 substitute is overlaid by the black line and mesh. (D) 3D-model structure showing the A304T substitution in PE1837 (w1-p1). The Ala304 residue is shown in the magenta stick, and the Thr304 substitute is represented by the black line and mesh. (E) 3D-model structure showing the S316F and A318D substitutions in PE636 (w1-p2). The Ser316 and Ala318 residues are shown in cyan sticks, and the Phe316 and Asp318 substitutes are overlaid by black lines and meshes.",
          "Fig 5. Derived cleaved amplified polymorphic sequence (dCAPS) analysis of CW12700 (w1-s1), PE1837 (w1-p1), and PE636 (w1-p2) and indel analysis of CW13381 (w1-s2). (A) dCAPS analysis of F2 populations derived from the breeding crosses, such as CW12700 x IT182932, PE1837 x Pungsannamul, and PE636 x Pungsannamul. The PCR products were digested by SpeI, PstI or HaeIII. (B) Indel analysis of the F2 population derived from a cross between CW13381 and IT182932. M, a 1-kb molecular marker; H, heterozygous F2 plants with purple flowers; A, wild-type F2 plants with purple flowers; and B, homozygous F2 plants with white flowers.",
          "Table 2. Plant materials of Glycine soja and Glycine max used in this study.",
          "Table 3. List of primers."]

# http://journals.plos.org/plosone/article?id=10.1371%2Fjournal.pone.0159918
# figure 2  1,3,2
PLOS_10 = ["An individual's reputation and group membership can produce automatic judgments and behaviors toward that individual. Whether an individual's social reputation impacts interactions with affiliates has yet to be demonstrated. We tested the hypothesis that during initial encounters with others, existing knowledge of their social network guides behavior toward them. Participants learned reputations (cooperate, defect, or equal mix) for virtual players through an iterated economic game (EG). Then, participants learned one novel friend for each player. The critical question was how participants treated the friends in a single-shot EG after the friend-learning phase. Participants tended to cooperate with friends of cooperators and defect on friends of defectors, indicative of a decision making bias based on memory for social affiliations. Interestingly, participants' explicit predictions of the friends' future behavior showed no such bias. Moreover, the bias to defect on friends of defectors was enhanced when affiliations were learned in a social context; participants who learned to associate novel faces with player faces during reinforcement learning did not show reputation-based bias for associates of defectors during single-shot EG. These data indicate that when faced with risky social decisions, memories of social connections influence behavior implicitly.",
           "Fig 1. Schematic of task phases. (a) Economic Game task. On each trial, participants were presented with a virtual player's image and had to choose to cooperate or defect. After a 1s fixation screen, the outcome of the trial was presented for 3s. This outcome consisted of the participant and player's choices as well as the points earned or lost according to the payoff matrix (gray inset). (b) Friend association task (social framing task). Half of the participants learned associations between players form the initial EG phase and new faces in a social framing friend association task. On each trial, participants were presented with a player or novel face and instructed to choose the friend from among two faces presented below. The trial continued only after a response was made, at which point, a fixation screen was presented for 1s followed by a 3s feedback consisting of the correct friendship association. Each friendship pair was repeated six times over the course of the friend association task. (c) Associative face learning task (nonsocial framing task). The other half of participants learned associations between players and new faces in a nonsocial framing association task. Participants were instructed to choose the face that led to a reward without reference to a social relationship among faces. The participants' feedback consisted of either 5 star coins for correct trials or a red X for incorrect trials. The participants' goal was to acquire as many points as they could; accordingly, they saw a running total of their points in the upper right hand corner of the display. As in the friend association task, each face-face pair was presented six times during this phase.",
           "Fig 2. Establishing reputations and memory of associations for the social (saturated) and nonsocial framing groups (lighter). (a) Average points won during the six trials within a mini-block during the iterated EG task separated by player type (cooperator-green, defector-red, and 50/50 -blue). Points represent individual subject point totals and solid lines depict the player type regression lines from the linear mixed effects model. (b) Average percentage of category responses during the categorization task on day 2 separated by actual player type and further by categorized player type. Error bars represent 95% confidence intervals of the mean. c) Average associative memory performance at the end of day 1 learning (left bars) and day 2 memory test (right bars) separated by reputation condition type (cooperator-green, defector-red, 50/50 -blue, no reputation-yellow). Error bars represent 95% confidence intervals of the mean.",
           "Fig 3. Cooperation behavior in one-shot EG and explicit predictions of cooperative behavior. (a) Average percentage of cooperative plays with associated faces during one-shot EG separated by the associated player reputation type for the social (saturated colors) and nonsocial (lighter colors) framing groups. (b) Average predicted cooperation in future EG games for players and associated faces separated by framing group. Error bars represent 95% confidence intervals of the mean."]

# http://journals.plos.org/plosone/article?id=10.1371%2Fjournal.pone.0159899
# figure5   5,1,2,4,3
PLOS_11 = ["kruppel-like factor 12 (kLF12) is a transcription factor that plays a role in normal kidney development and repression of decidualization. kLF12 is frequently elevated in esophageal adenocarcinoma and has been reported to promote gastric cancer progression. Here, we examined the role of kLF12 in colorectal cancer (CRC). Indeed, kLF12 promotes tumor growth by directly activating early growth response protein 1 (EGR1). The levels of kLF12 and EGR1 correlate synergistically with a poor prognosis. These results indicate that kLF12 likely plays an important role in CRC and could serve as a potential prognostic marker and therapeutic target.",
           "Fig 1. kLF12 promotes tumor growth in vitro and in vivo. (A) kLF12 protein levels in CRC cell lines. Actin served as a loading control. (B) kLF12 expression (top) and cell viability (bottom) of LS174T cells stably transfected with either GFP or kLF12. Actin served as a loading control. C. kLF12 expression (top) and cell viability (bottom) of HCT116 cells stably transfected with either a vector containing nonsilencing control shRNA (shCon) or one of two kLF12 shRNAs (shkLF12-1 and shkLF12-2). Actin served as a loading control. D. Tumor weight in mice orthotopically injected with either LS174T/GFP or LS174T/kLF12 cells (n = 8 for each group). E. Tumor weight in mice orthotopically injected with either HCT116/shCon, HCT116/shkLF12-1, or HCT116/shkLF12-2 cells (n = 9 for each group).",
           "Fig 2. EGR1 is a direct target of kLF12 mediating cell viability. (A) Chromatin immunoprecipitation (ChIP) assay showed binding of kLF12 to motif 2 of the EGR1 promoter, but not to motif 1 in LS-174T cells. Immunoprecipitation with IgG antibody was used as a control. (B) Transient co-transfection of cells was performed with EGR1 promoter wild type (WT), or mutant luciferase reporter plasmids with renilla luciferase control plasmids and non-targeting or kLF12 siRNA. The luciferase activity was determined. (C) EGR1 protein levels in CRC cell lines. Actin served as a loading control. (D) EGR1 mRNA (left) and protein (right) levels in LS174T cells stably transfected with either GFP or kLF12. Actin served as a loading control. E. EGR1 mRNA (left) and protein (right) levels in HCT116 cells stably transfected with either a vector containing nonsilencing control shRNA (shCon) or one of two kLF12 shRNAs (shkLF12-1 and shkLF12-2). Actin served as a loading control.",
           "Fig 3. kLF12 and EGR1 is co-expressed in vivo. (A) Immunohistochemistry of EGR1 in nude mice injected with either LS174/GFP cells as control, or with LS174 cells stably transfected with kLF12 (LS174/kLF12). (B) Immunohistochemistry of kLF12 and EGR1 in matching sections taken from two CRC patients (Patient #1 and #2). Magnification x10. (C) Pearson correlation of kLF12 and EGR1 mRNA expression in a cohort of 232 CRC patients (Moffitt cohort, n = 177 and Vanderbilt Medical Center cohort, n = 55).",
           "Fig 4. kLF12 enhances cell viability by activating EGR1. (A) Protein levels of kLF12 and EGR1 (left) and cell viability (right) of LS174T/GFP and LS174T/kLF12 cells transfected with either non-targeting siRNA as control (con) or EGR1 siRNA. Actin served as a loading control. (B) EGR1 protein levels (left) and cell viability (right) of LS174T cells transfected with either GFP (LS174T/GFP) or EGR1 (LS174T/EGR1). (C) Tumor weight in mice orthotopically injected with either LS174T/GFP or LS174T/EGR1 cells (n = 8 for each group).",
           "Fig 5. kLF12 and EGR1 expression levels are synergistically correlated with worse prognosis in CRC. kaplan-Meier Disease free survival (DFS) curves of a cohort of 232 CRC patients (Moffitt cohort, n = 177; Vanderbilt Medical Center cohort, n = 55) with either high or low mRNA levels of kLF12 (A), EGR1 (B), or both (C). Vertical bars denote censored patients."]

# http://journals.plos.org/plosone/article?id=10.1371%2Fjournal.pone.0159735
# figure 4   8,2,7,5,1,9,3,4,6
PLOS_12 = ["The Iberian Peninsula has been the focus of attention of numerous studies dealing with mitochondrial DNA (mtDNA) variation, most of them targeting the control region segment. In the present study we sequenced the control region of 3,024 Spanish individuals from areas where available data were still limited. We also compiled mtDNA haplotypes from the literature involving 4,588 sequences and 28 population groups or small regions. We meta-analyzed all these data in order to shed further light on patterns of geographic variation, taking advantage of the large sample size and geographic coverage, in contrast with the atomized sampling strategy of previous work. The results indicate that the main mtDNA haplogroups show primarily clinal geographic patterns across the Iberian geography, roughly along a North-South axis. Haplogroup HV0 (where haplogroup U is nested) is more prevalent in the Franco Cantabrian region, in good agreement with previous findings that identified this area as a climate refuge during the Last Glacial Maximum (LGM), prior to a subsequent demographic re-expansion towards Central Europe and the Mediterranean. Typical sub-Saharan and North African lineages are slightly more prevalent in South Iberia, although at low frequencies; this pattern has been shaped mainly by the transatlantic slave trade and the Arab invasion of the Iberian Peninsula. The results also indicate that summary statistics that aim to measure molecular variation, or AMOVA, have limited sensitivity to detect population substructure, in contrast to patterns revealed by phylogeographic analysis. Overall, the results suggest that mtDNA variation in Iberia is substantially stratified. These patterns might be relevant in biomedical studies given that stratification is a common cause of false positives in case-control mtDNA association studies, and should be also considered when weighting the DNA evidence in forensic casework, which is strongly dependent on haplotype frequencies.",
           "Fig 1. Map showing the location of the main Iberian regions analyzed in the present study. The pie charts show the frequency values for the main haplogroup categories in these main Iberian regions. Note that the category 'Other' in the pie charts denotes an artificial polyphyletic cluster (paragroup). Other haplogroup labels represent the following haplogroup categories: L = L(x M,N); N = N(xR,I,X,W).",
           "Table 1. Summary statistics of HVS-I sequences in Spain and Portugal (excluding islands) carried out on main geographic regions. All the computations were carried out on the common sequence segment of the HVS-I that ranges from 16024 to 16362. Population codes are as follows: a) Cantabrian coast = Galicia + Asturias + Cantabria + Basque Country + Navarra; b) Iberian Plateau = Castilla y Leon + Castilla La Mancha + La Rioja + Extremadura + Madrid + Aragon; c) Mediterranean coast = Murcia + Valencia + Catalonia; d) Andalusia; e) North Portugal = Viana do Castelo + Braga + Vila Real + Porto + Braganca; f) Centre Portugal = Aveiro + Viseu + Guarda + Coimbra + Castelo Branco + Leiria + Lisboa + Santarem + Portalegre; and g) South Portugal = Setubal + Evora + Beja + Faro. Note that sum of sample sizes of population sub-sets do not fully match with the total numbers mentioned above (4,588 from the literature + 3024 newly generated in the present study) because there was not enough geographic information for a few of them (the same applies to Tables 2 and 3).",
           "Table 2. Summary statistics of HVS-I sequences in Spain and Portugal (excluding islands) carried out on main political regions (Autonomous Regions). All the computations were carried out on the common sequence segment of the HVS-I that ranges from 16024 to 16362.",
           "Table 3. Summary statistics of HVS-I sequences in Spain (including islands) carried out on Spanish provinces. All the computations were carried out on the common sequence segment that ranges from 16024 to 16362.",
           "Fig 2. Geographic maps of sample sizes (A) and diversity indices (B and C).",
           "Fig 3. Geographic maps of the haplogroup frequencies. See legend of Fig 1 for more information.",
           "Table 4. Analysis of molecular variance (AMOVA) accounting for main geographic regions and Autonomous Regions (P-value < 0.0000).",
           "Fig 4. (A) Dispersal of H14a and H14a2 haplogroups along the Mediterranean coast. (B) Maximum parsimony tree of haplogroups represented by the Iberian mitogenomes sequenced in the present study. The position of the revised Cambridge reference sequence (rCRS) is indicated for reading sequence motifs [53]. Mitochondrial DNA variants are indicated along the branches of the phylogenetic tree. Mutations are transitions unless a suffix A, C, G, or T indicates a transversion. A suffix + indicates insertions. Variants underlined represent recurrent mutations in this tree while a prefix @ indicates a back mutation. Mutational hotspot variants at positions 16182, 16183, and 16519, as well as variation around position 310 and length or point heteroplasmies were not considered for the phylogenetic reconstruction. The color of the circles indicates their geographic origin according to the legend inset. The Iberian complete genomes obtained in the present study are indicated with yellow circles. More details on the geographic or ethnic origin of all the mitogenomes used in this network are provided in S3 Table. Haplogroups in orange boxes represent new branches of the mtDNA phylogeny, and are therefore not present in Phylotree Build 17.",
           "Table 5. TMRCA for haplogroup H14 and its sub-lineages."]

# http://journals.plos.org/plosone/article?id=10.1371%2Fjournal.pone.0159736
# fig4  2,5,4,3,1
PLOS_13 = ["Sampling the biodiversity is an essential step for conservation, and understanding the efficiency of sampling methods allows us to estimate the quality of our biodiversity data. Sex ratio is an important population characteristic, but until now, no study has evaluated how efficient are the sampling methods commonly used in biodiversity surveys in estimating the sex ratio of populations. We used a virtual ecologist approach to investigate whether active and passive capture methods are able to accurately sample a population's sex ratio and whether differences in movement pattern and detectability between males and females produce biased estimates of sex-ratios when using these methods. Our simulation allowed the recognition of individuals, similar to mark-recapture studies. We found that differences in both movement patterns and detectability between males and females produce biased estimates of sex ratios. However, increasing the sampling effort or the number of sampling days improves the ability of passive or active capture methods to properly sample sex ratio. Thus, prior knowledge regarding movement patterns and detectability for species is important information to guide field studies aiming to understand sex ratio related patterns.",
           "Table 1. Model sensitivity to each parameter. Parameters were individually modified in order to test their effect on the number of times sex ratio was correctly estimated. A negative effect means that an increase in the parameter caused a reduction in the number of times sex ratio was properly estimated. Movement probability and detectability were the only parameters that caused biased estimations of sex ratios.",
           "Fig 1. The influence of differences in movement pattern and detectability between males and females on sampling method efficiency in estimating a population's sex ratio. Sex ratio distributions (left) and number of times sex ratio was recorded as equal (right) in 100 simulations when changing a) differences in movement pattern between males and females; b) differences in detectability between males and females. The number of times sex ratio was recorded as equal was determined by the number of times that the chi-square test performed on the sex ratio sampled in each simulation found a non-significant result. Red circles represent passive capture methods, and blue squares are active search.",
           "Fig 2. The influence of number of sampling days on the efficiency of the sampling methods in estimating a population's sex ratio when there are differences in the movement pattern between males and females. Left graphs represent sex ratio distributions and right graphs represent the number of times the sex ratio was recorded as equal in 100 simulations with varying sampling days. From (a) to (d), increasing difference in movement pattern: a) movement probability of males = 0.6 and females = 0.4; b) males = 0.7 and females = 0.3; c) males = 0.8 and females = 0.2; d) males = 0.9 and females = 0.1. The number of times sex ratio was recorded as equal was determined by the number of times that the chi-square test performed on the sex ratio sampled in each simulation found a non-significant result. Red circles represent passive capture methods, and blue squares are active searches.",
           "Fig 3. The influence of number of sampling days on the efficiency of the sampling methods in estimating a population's sex ratio when there are differences in detectability between males and females. Left graphs represent sex ratio distributions and right graphs represent the number of times the sex ratio was recorded as equal in 100 simulations with varying sampling days. From (a) to (d), decreasing difference in detectability: a) males = 0.1 and females = 0.9; b) males = 0.2 and females = 0.8; c) males = 0.3 and females = 0.7; d) males = 0.4 and females = 0.6. The number of times sex ratio was recorded as equal was determined by the number of times that the chi-square test performed on the sex ratio sampled in each simulation found a non-significant result. Red circles represent passive capture methods, and blue squares are active searches. Note the efficiency in pitfall traps is constant during the simulations, since detectability implemented in our simulation is related only to active searches.",
           "Fig 4. The influence of passive sampling effort on the efficiency of the sampling methods in estimating a population's sex ratio when there are differences in movement pattern between males and females. Left graphs represent sex ratio distributions and right graphs represent the number of times the sex ratio was recorded as equal in 100 simulations with varying passive sampling efforts. From (a) to (d), increasing difference in movement pattern. a) male movement probability = 0.6 and female = 0.4; b) male = 0.7 and female = 0.3; c) male = 0.8 and female = 0.2; d) male = 0.9 and female = 0.1. The number of times sex ratio was recorded as equal was determined by the number of times that the chi-square test performed on the sex ratio sampled in each simulation found a non-significant result. Red circles represent passive capture methods, and blue squares are active searches."]

# http://journals.plos.org/plosone/article?id=10.1371%2Fjournal.pone.0159704
# fig 5     4,3,2,5,7,1,6
PLOS_14 = ["Schistosoma japonicum insulin receptors (SjIRs) have been identified as encouraging vaccine candidates. Interrupting or blocking the binding between host insulin and the schistosome insulin receptors (IRs) may result in reduced glucose uptake leading to starvation and stunting of worms with a reduction in egg output. To further understand how schistosomes are able to exploit host insulin for development and growth, and whether these parasites and their mammalian hosts compete for the same insulin source, we identified insulin binding sites on the SjIRs. Based on sequence analysis and the predicted antigenic structure of the primary sequences of the SjIRs, we designed nine and eleven peptide analogues from SjIR-1 and SjIR-2, respectively. Using the Octet RED system, we identified analogues derived from SjIR-1 (10) and SjIR-2 (20, 21 and 22) with insulin-binding sequences specific for S. japonicum. Nevertheless, the human insulin receptor (HIR) may compete with the SjIRs in binding human insulin in other positions which are important for HIR binding to insulin. However, no binding occurred between insulin and parasite analogues derived from SjIR-1 (2, 7 and 8) and SjIR-2 (14, 16 and 18) at the same locations as HIR sequences which have been shown to have strong insulin binding affinities. Importantly, we found two analogues (1 and 3), derived from SjIR-1, and two analogues (13 and 15) derived from SjIR-2, were responsible for the major insulin binding affinity in S. japonicum. These peptide analogues were shown to have more than 10 times (in kD value) stronger binding capacity for human insulin compared with peptides derived from the HIR in the same sequence positions. Paradoxically, analogues 1, 3, 13 and 15 do not appear to contain major antigenic determinants which resulted in poor antibody responses to native S. japonicum protein. This argues against their future development as peptide-vaccine candidates.",
           "Fig 1. Schematic model of the SjIRs with approximate locations of mapped peptide analogues 1-22. Peptides 1-10 (in red) are located on SjIR-1; peptides 11, 13-22 (in blue) are located on SjIR-2. Peptide 9 is located on the HIR (its approximate location relative to the sequence in the SjIRs is indicated). Peptide 12 is the reverse analogue of peptide 9. L1, L1 subdomain; L2, L2 subdomain; CR, cysteine rich domain; FnIII-1-3, fibronectin domains 1, 2 and 3; TM, transmembrane domain; Tk, tyrosine kinase domain.",
           "Table 1. Details of peptide analogues screened for insulin binding activity including sequence information and their location within the SjIR proteins.",
           "Fig 2. The binding affinity between human insulin and analogues 1, 13 and 28 derived from SjIR-1, SjIR-2 and HIR, respectively, determined using the Octet RED system. Binding is shown between human insulin and analogues 1 (A), 13 (B) and 28 (C) at different peptide concentrations (uM). The real time binding response (nm) was measured in seconds (sec). The parameters of the binding response (nm) and the kD value (M) of the binding between insulin and the analogues at different concentrations are shown. The coefficient of determination (R^2) of these interactions was close to 1.0, indicating a good curve fit. The left panels show that an increased binding response was observed when the concentration of peptide analogues was increased from low to high, as shown on the right in the tables.",
           "Fig 3. The binding affinity between human insulin and analogues 3, 15 and 29 derived from SjIR-1, SjIR-2 and HIR, respectively, measured using the Octet RED system. Binding between human insulin and analogues 3 (A), 15 (B) and 29 (C) is shown at different concentrations (uM). The real time binding response (nm) was measured in seconds (sec). The parameters of the binding response (nm) and the kD value (M) of the binding between insulin and the analogues at different concentrations are shown in the tables on the right. The coefficient of determination (R^2) for all these interactions was close to 1.0, indicating a good curve fit.",
           "Table 2. Details of the peptide analogues screened for human insulin binding activity.",
           "Fig 4. Circular dichroism spectra for peptide analogues 1 (top left), 3 (top right), 13 (bottom left), and 15 (bottom right).",
           "Fig 5. Molecular modelling of the SjIR-1 and SjIR-2 proteins showing the location of peptide analogues 1, 3, 13, and 15. The top panel shows the location on the SjIR-1 protein of analogues 1 (left), 3 (middle), and 1 and 3 combined (right). The bottom panel shows the location on the SjIR-2 protein of analogues 13 (left), 15 (middle), and 13 and 15 combined (right). Cyan represents the protein with peptide analogues identified as yellow (B-sheet), red (coiled-coil) or green (the secondary structure is absent)."
           ]

# http://journals.plos.org/plosone/article?id=10.1371%2Fjournal.pone.0159646
# fig 8   8,7,6,4,1,2,3,5
PLOS_15 = ["The multi-functional molecular motor cytoplasmic dynein performs diverse essential roles during mitosis. The mechanistic importance of the dynein Light Intermediate Chain homologs, LIC1 and LIC2 is unappreciated, especially in the context of mitosis. LIC1 and LIC2 are believed to exist in distinct cytoplasmic dynein complexes as obligate subunits. LIC1 had earlier been reported to be required for metaphase to anaphase progression by inactivating the kinetochore-microtubule attachment-sensing arm of the spindle assembly checkpoint (SAC). However, the functional importance of LIC2 during mitosis remains elusive. Here we report prominent novel roles for the LIC2 subunit of cytoplasmic dynein in regulating the spindle assembly checkpoint. LIC2 depletion in mammalian cells led to prolonged metaphase arrest in the presence of an active SAC and also to stretched kinetochores, thus implicating it in SAC inactivation. Quantitative fluorescence microscopy of SAC components revealed accumulation of both attachment- and tension-sensing checkpoint proteins at metaphase kinetochores upon LIC2 depletion. These observations support a stronger and more diverse role in checkpoint inactivation for LIC2 in comparison to its close homolog LIC1. Our study uncovers a novel functional hierarchy during mitotic checkpoint inactivation between the closely related but homologous LIC subunits of cytoplasmic dynein. These subtle functional distinctions between dynein subpopulations could be exploited to study specific aspects of the spindle assembly checkpoint, which is a key mediator of fidelity in eukaryotic cell division.",
           "Fig 1. Localization of LICs in Mitosis. A) Confocal immunofluorescence images of Hela cells depicting the localization of LIC1 (red) and LIC2 (green), shown by arrows. B) Fluorescence images of Hela cells showing the presence of LIC2 (green, using the ThermoScientific antibody) at kinetochores (CREST, red) in prometaphase. C) Fluorescence imaging after treatment of cells with anti-LIC2 siRNA, showing loss of LIC2 signal from spindle poles and kinetochores (arrow in inset). DAPI was used to visualize chromosomes. Scalebar is 5 um in all images that are not zoomed.",
           "Fig 2. LIC2 is required for metaphase to anaphase progression. A) Metaphase index (% of total cells present in metaphase +/- SD) in Hela cells treated with respective siRNAs as indicated. (3 experiments, n = approximately 500 cells per experiment). B) & C) Western blots showing siRNA mediated LIC1 and LIC2 specific knockdown. LIC2a and LIC2b represent two different siRNA sequences against human LIC2 [23]. The band marked by the arrow represents human LIC2 that gets reduced upon siRNA treatment, the upper bands are non-specifically recognized by the LIC2 antibody (ThermoScientific). Actin = loading control. D) Rescue of the LIC2-depletion induced metaphase arrest by transgenic expression of rat LIC2 (3 experiments, n = at least 500 cells per experiment). E) Western blots showing specific depletion of LIC2 upon treatment with LIC2-specific siRNAs, using a different LIC2 antibody (Abcam). LIC2a and LIC2b represent two different siRNA sequences against human LIC2 [23]. The band marked by the arrow represents human LIC2 that appears at the same molecular weight as in C.",
           "Fig 3. LIC2 depletion leads to prolonged metaphase arrest. A) Still images from a movie featuring four different representative siRNA treated Hela cells (control, LIC2 and LIC 1 + 2) showing prolonged arrest in metaphase, followed by delayed anaphase onset (LIC2 > 80 min) or cell death (LIC2 > 4 hrs). B) Average metaphase to anaphase timing, (3 experiments, n = at least 100 cells per experiment). NEB = Nuclear Envelope Breakdown. Statistical significance was calculated by the T-test between two groups. C) Fraction of mitotic cells arrested for more than 80 minutes in metaphase upon respective siRNA treatment. (3 experiments, n = at least 100 cells per experiment). D) Fractions of metaphase cells dying after prolomged metaphase arrest (> 4 hrs) from the cells in C. p values in all panels are * p <0.05, ** p<0.01, *** p<0.001. Scalebar is 75 um in all images as indicated.",
           "Fig 4. LIC2 acts on the spindle assembly checkpoint (SAC) to ensure mitotic progression. A) Metaphase index in Hela cells after respective siRNA treatments of Mad2 and BubR1 alone or co-depleted with LIC2, as indicated on the x-axis (3 experiments, n = at least 200 cells per experiment). *** p<0.001, ns = not significant (p > 0.05). B) Western blot showing depletion of Mad2 by siRNA treatment. Actin = loading control.",
           "Fig 5. LIC2 depletion leads to increased inter-kinetochore tension in metaphase. A) Confocal immunofluorescence images of metaphase Hela cells stained for kinetochores (red, CREST) and chromatin (DAPI). White lines indicate representative distance measurements between sister kinetochores. The insets on the right are zoomed to equivalent levels in both upper and lower panels. B) Average inter-kinetochore distance (um) on y-axis. 3 experiments; n = at least 30 metaphase cells per experiment (10 pairs of sister chromatids measured per cell). ** = p < 0.01). Scalebar is 5 um in all images that are not zoomed.",
           "Fig 6. LIC2 removes key SAC proteins from metaphase kinetochores to inactivate the spindle assembly checkpoint. Representative images showing accumulation of SAC proteins A) Mad1, C) Zw10 and F) BubR1 at metaphase kinetochores upon prolonged metaphase arrest following indicated siRNA treatments. B, D, E, G) Left panels: integrated fluorescence intensities of SAC proteins normalized to the respective kinetochore (CREST) intensities for B) Mad1, D) Zw10, E) Mad2 and G) BubR1 at metaphase kinetochores. Y-axis = mean fluorescence intensity (+/- SEM) from 3 independent experiments, n = at least 18 metaphase cells per experiment for LIC1 and LIC2 depletion. Right panels: fold increase in normalized fluorescence intensity over control (GFP siRNA) for the various SAC proteins. Scalebar is 5 um in all images.",
           "Fig 7. LIC2 biochemically interacts with key SAC proteins at metaphase. A) Western blots of affinity purified mitotic LIC2 (LIC2-MTAP) probing for interaction of SAC proteins Mad1, Zw10 and BubR1 (right panel), and with the empty tag control (left panel). B-actin was used as the loading control. Control indicates empty tag (MTAP vector alone) pulldown, LIC2 indicates LIC2-MTAP (see Materials and Methods). B) Upper panels: Immunoblots probing for the pulldown of dynein subunits DHC (dynein heavy chain) and IC-74 (dynein intermediate chain) upon affinity purification of the empty MTAP tag or LIC2-MTAP. Lower panels show successful pulldown of the respective bait proteins as positive controls for the affinity purification reaction. IN = input, FT = flow through, WS = wash, AP = affinity pulldown precipitate.",
           "Fig 8. A model for key LIC2 functions in SAC silencing during mitosis. Novel functions of mitotic LIC2 uncovered from this study in silencing the spindle assembly checkpoint (SAC) are shown. LIC2-dynein strips attachment sensing SAC proteins (Mad1, Mad2, Zw10) from metaphase kinetochores, like LIC1-dynein. LIC2-dynein has the additional capability of causing removal of tension-sensing SAC protein BubR1, which is lacking in LIC1-dynein. The model offers a possible mechanistic explanation for the differential effects of the two LICs in mediating metaphase to anaphase progression."]

# http://journals.plos.org/plosone/article?id=10.1371%2Fjournal.pone.0159871
# fig 9  **4,8,2,10,7,11,9,3,1,6,5
PLOS_16 = ["The bacterial RecA protein plays a role in the complex system of DNA damage repair. Here, we report the functional and structural characterization of the Herbaspirillum seropedicae RecA protein (HsRecA). HsRecA protein is more efficient at displacing SSB protein from ssDNA than Escherichia coli RecA protein. HsRecA also promotes DNA strand exchange more efficiently. The three dimensional structure of HsRecA-ADP/ATP complex has been solved to 1.7 A resolution. HsRecA protein contains a small N-terminal domain, a central core ATPase domain and a large C-terminal domain, that are similar to homologous bacterial RecA proteins. Comparative structural analysis showed that the N-terminal polymerization motif of archaeal and eukaryotic RecA family proteins are also present in bacterial RecAs. Reconstruction of electrostatic potential from the hexameric structure of HsRecA-ADP/ATP revealed a high positive charge along the inner side, where ssDNA is bound inside the filament. The properties of this surface may explain the greater capacity of HsRecA protein to bind ssDNA, forming a contiguous nucleoprotein filament, displace SSB and promote DNA exchange relative to EcRecA. Our functional and structural analyses provide insight into the molecular mechanisms of polymerization of bacterial RecA as a helical nucleoprotein filament.",
           "Fig 1. ATPase profile from EcRecA and HsRecA in the presence and absence of SSB protein. (A) Reaction 1: contained 5 uM nt M13mp18 cssDNA and 3 uM HsRecA or EcRecA, were previously incubated per 20 min at 37C, following, 3 uM ATP and 0.5 uM SSB. Reaction 2: reaction 1 without SSB protein addition. Time 0 min indicates the addition of ATP and SSB. (B) The reaction contained 5 uM nt M13mp18 cssDNA, 3 uM ATP and 0.35 uM SSB, were previously incubated per 10 min at 37C, following addition of 3 uM HsRecA or EcRecA (time 0 min).",
           "Fig 2. DNA strand exchange promoted by the wild-type HsRecA and EcRecA proteins. (A) The three DNA strand exchange containing 10 uM nt M13mp18 cssDNA and 3.5 uM HsRecA or EcRecA, were previously incubated per 20 min at 37C, and then 3 uM ATP and 1 uM SSB were added and incubated for an additional 10 min. The minutes shown represents the time of reaction after addition of 20 uM nt M13mp18 ldsDNA. (B) The percentage of duplex substrate converted into the nicked circular duplex (NC product) is plotted against the time.",
           "Fig 3. Electronic microscopy of EcRecA and HsRecA filaments in the absence or presence of EcSSB. (A) EcRecA-cssDNA filaments without EcSSB, (B) HsRecA-cssDNA filaments without EcSSB, (C) EcRecA-cssDNA filaments with EcSSB, (D) HsRecA-cssDNA filament with EcSSB, (E) Avarege length of 10 HsRecA and EcRecA filaments in the conditions assayed. Letters above each bar refer back to panels A-D. Reactions containing 6.7 uM HsRecA or EcRecA and 20 uM M13mp18 cssDNA were incubated at 37C for 20 min, with or without 2 uM Ec SSB. After 10 min, filaments were stabilized by a 3 min incubation with 3 uM ATPrS and spread on an Alcian grid.",
           "Fig 4. Ribbon diagram of the monomeric crystal structure of HsRecA protein. Regions that could be modeled were indicated by the last residue-number. HsRecA protein is composed of N-terminal domain (NTD), a central core ATPase domain and a large C-terminal domain (CTD). The core ATPase domain contains one Ca2+ ion (magenta sphere), coordinated by Asn119 and Asp120 and the ATPase activity site is partially occupied by ATP and ADP. Figure was prepared using the STRIDE program for secondary structure assignment [55], and visualized using PyMOL [56].",
           "Table 1. Data collection and refinement statistics.",
           "Fig 5. Omit maps (mFo-DFc) at the ATP binding site: (A) Refinement with ATP at 100% occupancy; the negative electron density over the r-phosphate indicates that it should not be at this full occupancy. (B) Refinement with ATP at 39% occupancy; the positive electron density over the corresponding ADP moiety indicates that it should be at full occupancy. The omit maps are contoured at +3 (green) and -3 (red) t levels.",
           "Fig 6. 3D structural and amino acid sequence alignment from H. seropedicae (HsRecA), E. coli (EcRecA), M. tuberculosis (MtRecA), M. smegmatis (MsRecA), D. radiodurans (DrRecA), N. gonorrhoeae (NgRecA), P. aeruginosa (PaRecA), M. voltae (MvRadA), P. furiosus (RadA), H. sapiens (HsRad51), P. furiosus (Rad51), S. cerevisiae (ScRad51 and Dmc1). Part of the N-terminus has been removed for clarity. The position of the core ATPase domain and the C-terminus are indicated. Functional motifs are indicated above their corresponding amino acid sequences: N-PM and core-PM, N-terminal and core polymerization motif, respectively, the ATP binding Walker A and B motifs, the putative DNA binding sites Loop L1 and L2. The secondary structure of HsRecA and HsDmc1 are indicated at the top and bottom, respectively. The 3D structural and amino acid alignment was performed using the MultiProt and T-Coffee programs [61,62]. The results are visualized using ESPript-http://espript.ibcp.fr [63].",
           "Table 2. Summary of hydrogen bonds formed between the N-PM and the Core-PM in RecA protein structures. The residues and their atoms involved in the interaction are indicated, as well as the atomic distances. Interactions were determined using the CCP4 application (Protein Interfaces, Surfaces and Assemblies-PISA) [52,64,65].",
           "Fig 7. Superposition inactive and presynaptic monomers of EcRecA protein over the inactive HsRecA monomer. EcRecA protein is colored in gray and HsRecA in cyan. (A) Superposition of the inactive EcRecA structure (pdb entry 1XMV) over the HsRecA structure. Both structures have a B-loop motif in the N-PM. In the EcRecA structure, a Mg2+ ion interacts with the N-PM. In the HsRecA structure, a Ca2+ ion interacts with the Core-PM. (B) Superposition of presynaptic monomers of EcRecA structure (pdb entry 3CMU:A) over the HsRecA structure. The N-PM in the active EcRecA structure assumes a B-strand motif. The nucleotides ADP and ADP-AlF4, Mg2+ and Ca2+ ions, and the ssDNA bound in the EcRecA structure was removed for clarity. Figure was prepared using the STRIDE program for secondary structure assignment [55]. The superposition was generated using the MultiProt program [61] and visualized using PyMOL [56].",
           "Fig 8. Electrostatic potential distribution on the solvent-accessible surface of HsRecA protein structure. (A) Side view of two subsequent helical hexamers. (B) and (C), 5' and 3' views, relative to the axial direction of the filament, respectively. The surface potential representation has charge levels from -3kT/e (red) to +3kT/e (blue). The electrostatic potential distribution was generated using the APBS program, side chain atom not ordered in the crystal were added using the PDB2PQR program and protonation states at pH 7.5 were assigned with the PROPkA program [67-69], and visualized using PyMOL [56].",
           "Fig 9. Ribbon diagram of two subsequent helical hexamers of HsRecA protein. Each monomer is colored differently. The hexameric structure has 61-fold symmetry, the helical filament has a pitch of 91.3 A. Figures were prepared using the STRIDE program for secondary structure assignment [55], and visualized using PyMOL [56]."]

# http://journals.plos.org/plosone/article?id=10.1371%2Fjournal.pone.0159596
# fig 6  6,3,1,4,5,2
PLOS_17 = ["Blood testing has been used as an essential tool to diagnose diseases for decades. Recently, there has been a rapid developing trend in using Quantitative Phase Imaging (QPI) methods for blood cell screening. Compared to traditional blood testing techniques, QPI has the advantage of avoiding dyeing or staining the specimen, which may cause damage to the cells. However, most existing systems are bulky and costly, requiring experienced personnel to operate. This work demonstrates the integration of one QPI method onto a smartphone platform and the application of imaging red blood cells. The adopted QPI method is based on solving the Intensity Transport Equation (ITE) from two de-focused pupil images taken in one shot by the smartphone camera. The device demonstrates a system resolution of about 1 um, and is ready to be used for 3D morphological study of red blood cells.",
           "Fig 1. Conjugate planes in the conventional WCS setup.",
           "Fig 2. Smartphone-based wavefront sensor with extended system magnification.",
           "Fig 3. Resulting image of using the smartphone-based system to image the USAF resolution target. The system demonstrates the capability of resolving the 6th element of group 7 on the resolution target, which has a line width around 2 um.",
           "Fig 4. (a) Regular optical grating is consisted of periodical straight lines; (b) Quadratically distorted grating lines in a circular aperture; (c) Fabricated grating masks for smartphone devices, each of the dark circles is a mask; (d) System structure of the prototyping system.",
           "Fig 5. (a) The theoretical thickness map of a 8 um diameter polymer microsphere; (b) the reconstructed thickness map of the polymer microsphere using the reported system.",
           "Fig 6. Exemplary results of using the smartphone-based wavefront sensor for red blood cell imaging. (a) A raw photo taken by the smartphone camera, consisting of 3 images of the pupil; (b) Pre-selected and cut images for the wavefront computing algorithm; (c) Calculated phase map; (d) Reconstructed height map."]

# http://journals.plos.org/plosone/article?id=10.1371%2Fjournal.pone.0159566
# fig 6   5,3,7,4,6,1,2
PLOS_18 = ["Diospyros is the largest genus in Ebenaceae, comprising more than 500 species with remarkable economic value, especially Diospyros kaki Thunb., which has traditionally been an important food resource in China, korea, and Japan. Complete chloroplast (cp) genomes from D. kaki, D. lotus L., D. oleifera Cheng., D. glaucifolia Metc., and Diospyros 'Jinzaoshi' were sequenced using Illumina sequencing technology. This is the first cp genome reported in Ebenaceae. The cp genome sequences of Diospyros ranged from 157,300 to 157,784 bp in length, presenting a typical quadripartite structure with two inverted repeats each separated by one large and one small single-copy region. For each cp genome, 134 genes were annotated, including 80 protein-coding, 31 tRNA, and 4 rRNA unique genes. In all, 179 repeats and 283 single sequence repeats were identified. Four hypervariable regions, namely, intergenic region of trnQ_rps16, trnV_ndhC, and psbD_trnT, and intron of ndhA, were identified in the Diospyros genomes. Phylogenetic analyses based on the whole cp genome, protein-coding, and intergenic and intron sequences indicated that D. oleifera is closely related to D. kaki and could be used as a model plant for future research on D. kaki; to our knowledge, this is proposed for the first time. Further, these analyses together with two large deletions (301 and 140 bp) in the cp genome of D. 'Jinzaoshi', support its placement as a new species in Diospyros. Both maximum parsimony and likelihood analyses for 19 taxa indicated the basal position of Ericales in asterids and suggested that Ebenaceae is monophyletic in Ericales.",
           "Fig 1. Gene maps of Diospyros chloroplast genomes. Genes on the outside of the large circle are transcribed clockwise and those on the inside are transcribed counterclockwise. The genes are color-coded based on their function. Dashed area represents the GC composition of the chloroplast genome.",
           "Table 1. Genes located on Diospyros chloroplast genomes.",
           "Fig 2. Global alignment of Ebenaceae genome and other published chloroplast genomes in Ericales using VISTA. Y-axis indicates the range of identity (50%-100%). Alignment was performed using D. kaki as a reference.",
           "Fig 3. Phylogenetic trees based on whole genome sequences of Diospyros. (a) Maximum likelihood tree, (b) Maximum parsimony tree.",
           "Fig 4. Indels (>5 bp) identified based on multiple sequence alignment of five Diospyros cp genomes. Insertions are shown above and deletions below the horizontal axis. Indel distribution was positioned using D. kaki as a reference.",
           "Fig 5. The comparison of inverted-repeat (IR) and single-copy (SC) borders among nine chloroplast genomes. Gene annotation or portions are represented by gray boxes above or below.",
           "Fig 6. Phylogenetic tree of the asterid clade. The tree was reconstructed based on 61 protein-coding sequences shared by 19 angiosperm species. The numbers at the nodes indicate bootstrap values (1000 replications)."]

# http://journals.plos.org/plosone/article?id=10.1371%2Fjournal.pone.0159545
# fig 4 **   1,5,3,2,4,6
PLOS_19 = ["Dalmatian sage (Salvia officinalis L., Lamiaceae) is a well-known aromatic and medicinal Mediterranean plant that is native in coastal regions of the western Balkan and southern Apennine Peninsulas and is commonly cultivated worldwide. It is widely used in the food, pharmaceutical and cosmetic industries. knowledge of its genetic diversity and spatiotemporal patterns is important for plant breeding programmes and conservation. We used eight microsatellite markers to investigate evolutionary history of indigenous populations as well as genetic diversity and structure within and among indigenous and cultivated/naturalised populations distributed across the Balkan Peninsula. The results showed a clear separation between the indigenous and cultivated/naturalised groups, with the cultivated material originating from one restricted geographical area. Most of the genetic diversity in both groups was attributable to differences among individuals within populations, although spatial genetic analysis of indigenous populations indicated the existence of isolation by distance. Geographical structuring of indigenous populations was found using clustering analysis, with three sub-clusters of indigenous populations. The highest level of gene diversity and the greatest number of private alleles were found in the central part of the eastern Adriatic coast, while decreases in gene diversity and number of private alleles were evident towards the northwestern Adriatic coast and southern and eastern regions of the Balkan Peninsula. The results of Ecological Niche Modelling during Last Glacial Maximum and Approximate Bayesian Computation suggested two plausible evolutionary trajectories: 1) the species survived in the glacial refugium in southern Adriatic coastal region with subsequent colonization events towards northern, eastern and southern Balkan Peninsula; 2) species survived in several refugia exhibiting concurrent divergence into three genetic groups. The insight into genetic diversity and structure also provide the baseline data for conservation of S. officinalis genetic resources valuable for future breeding programmes.",
           "Fig 1. Within-population microsatellite diversity and genetic relationships of 30 Dalmatian sage populations. Populations are numbered as for Table 1. (A) Distribution of sampled populations: circles indicate indigenous populations; triangles indicate cultivated/naturalised populations. Symbol colours correspond to allelic richness (Nar): white < 5, yellow 5-7, orange 7-9 and red >9. (B) Unrooted Fitch-Margoliash tree based on Cavalli-Sforza's chord distance. Bootstrap support values greater than 50% of 1,000 replicates are given near the branches.",
           "Fig 2. Genetic structure of 30 Dalmatian sage populations as estimated by the software STRUCTURE. The population numbering corresponds to Table 1. (A) Population structure assuming k = 4. (B) Proportions of membership for k = 2 to 4 clusters are given. Each individual plant is represented by a single vertical line divided into colours. Each colour represents one cluster, and the length of the coloured segment shows the individual's estimated proportion of membership in that cluster. White lines separate populations that are labelled below the figure.",
           "Table 1. Sampling sites, genetic diversity and genetic bottlenecks assessed from eight microsatellite markers in 30 Dalmatian sage populations.",
           "Table 2. Analysis of molecular variance for the partitioning of microsatellite diversity.",
           "Fig 3. Isolation by distance analysis among 23 indigenous Dalmatian sage populations.",
           "Fig 4. Ecological Niche Modelling of Salvia officinalis under (a) present, (b) Last Glacial Maximum CCSM conditions and (c) Last Glacial Maximum MIROC conditions. The colour codes indicate habitat suitability, from light blue (low suitability) to dark blue (high suitability) as calculated by the model."
           ]

# http://journals.plos.org/plosone/article?id=10.1371%2Fjournal.pone.0159497
# fig 6 **     6,7,4,5,1,3,2
PLOS_20 = ["In prostate radiotherapy, the influence of bladder volume variation on the dose absorbed by the target volume and organs at risk is significant and difficult to predict. In addition, the resolution of a typical medical image is insufficient for visualizing the bladder wall, which makes it more difficult to precisely evaluate the dose to the bladder wall. This simulation study aimed to quantitatively investigate the relationship between the dose received by organs at risk and the intravesical volume in prostate radiotherapy. The high-resolution Visible Chinese Human phantom and the finite element method were used to construct 10 pelvic models with specific intravesical volumes ranging from 100 ml to 700 ml to represent bladders of patients with different bladder filling capacities during radiotherapy. This series of models was utilized in six-field coplanar 3D conformal radiotherapy simulations with different planning target volume (PTV) margins. Each organ's absorbed dose was calculated using the Monte Carlo method. The obtained bladder wall displacements during bladder filling were consistent with reported clinical measurements. The radiotherapy simulation revealed a linear relationship between the dose to non-targeted organs and the intravesical volume and indicated that a 10-mm PTV margin for a large bladder and a 5-mm PTV margin for a small bladder reduce the effective dose to the bladder wall to similar degrees. However, larger bladders were associated with evident protection of the intestines. Detailed dosimetry results can be used by radiation oncologists to create more accurate, individual water preload protocols according to the patient's anatomy and bladder capacity.",
           "Fig 1. Flowchart of organ deformation and radiotherapy simulation.",
           "Fig 2. FE models constructed in Hypermesh. Right views of (a) intestines, (b) prostate, (c) bladder, and (d) seminal vesicles. (e) Anterior view and (f) posterior view of the VCH pelvic part.",
           "Table 1. Properties of the organs in the pelvic region.",
           "Fig 3. Ten pelvic models with different bladder volumes. Only the bladder and bone are shown for easier comparison.",
           "Fig 4. (A) Bladder wall displacement after a 100 ml increase in bladder volume. The average values and standard deviations of the simulated results and clinical results of Pinkawa et al. [25] are compared. (B) Prostate displacement after a 100-ml increase in bladder volume.",
           "Fig 5. Effective dose and volume fraction received more than 50 Gy of the bladder in radiotherapy with different PTV margins.",
           "Fig 6. (A) Intestines volume that received more than 50 Gy for different PTV margins and (B) an intuitive representation of the distribution of the dose (Gy) to the intestines when intravesical volume varied from 100 ml to 700 ml with 5 (a-g) and 10 mm (h-n) PTV margins."]

# http://journals.plos.org/plosone/article?id=10.1371%2Fjournal.pone.0159752
# fig 4 **      10,8,11,5,6,12,2,4,7,1,9,3
PLOS_21 = ["In demyelinating diseases, changes in the quality and quantity of the extracellular matrix (ECM) may contribute to demyelination and failure of myelin repair and axonal sprouting, especially in chronic lesions. To characterize changes in the ECM in canine distemper demyelinating leukoencephalitis (DL), histochemical and immunohistochemical investigations of formalin-fixed paraffin-embedded cerebella using azan, picrosirius red and Gomori`s silver stain as well as antibodies directed against aggrecan, type I and IV collagen, fibronectin, laminin and phosphacan showed alterations of the ECM in CDV-infected dogs. A significantly increased amount of aggrecan was detected in early and late white matter lesions. In addition, the positive signal for collagens I and IV as well as fibronectin was significantly increased in late lesions. Conversely, the expression of phosphacan was significantly decreased in early and more pronounced in late lesions compared to controls. Furthermore, a set of genes involved in ECM was extracted from a publically available microarray data set and was analyzed for differential gene expression. Gene expression of ECM molecules, their biosynthesis pathways, and pro-fibrotic factors was mildly up-regulated whereas expression of matrix remodeling enzymes was up-regulated to a relatively higher extent. Summarized, the observed findings indicate that changes in the quality and content of ECM molecules represent important, mainly post-transcriptional features in advanced canine distemper lesions. Considering the insufficiency of morphological regeneration in chronic distemper lesions, the accumulated ECM seems to play a crucial role upon regenerative processes and may explain the relatively small regenerative potential in late stages of this disease.",
           "Table 1. Numbers of animals, investigated areas and groups in histochemistry and immunohistochemistry.",
           "Table 2. Used primary antibodies, supplier, clonality, immunogen, demasking of antigens (pretreatment) and dilution.",
           "Table 3. List of the different groups and numbers of animals per group in microarray and real time quantitative PCR (RT-qPCR) analysis.",
           "Table 4. Target gene, sequence, amplicon length and GenBank accession of primers used for RT-qPCR analysis.",
           "Fig 1. Histochemical staining of CDV-infected canine brains. A: Cerebellum, white matter, acute lesion with azan-positive vascular walls (arrow). B: Cerebellum, white matter, chronic lesion, extensive reticular, extracellular, intralesional deposition of azan-positive blue material (arrows). Azan stain. C: Cerebellum, white matter, acute lesion, dark red reaction of vascular wall (arrow). Modified picrosirius red stain. D: Cerebellum, white matter, chronic lesion, intralesional grid-like picrosirius red reaction (arrows). Modified picrosirius red stain. E: Cerebellum, white matter, acute lesion, black signal around blood vessels. Gomori`s silver stain. F: Cerebellum, white matter, chronic lesion, branched filamentous reaction in the center of the lesion and around blood vessels (arrows). Gomori's silver stain. A-F: Insets show vascular structures in a higher magnification. All scale bars = 50 um.",
           "Table 5. Morphometrically determined areas of extracellular matrix molecule deposition in the cerebella of CDV-infected dogs and controls (positive area per total white matter area [%]).",
           "Table 6. p-values of the multiple pair-wise comparisons of geometric means (Tukey-kramer test) of histochemical stains (logarithmic data).",
           "Fig 2. Immunohistochemical detection of aggrecan, phosphacan and fibronectin in CDV-infected canine brains. A: Cerebellum, white matter, acute lesion, minimal to mild extracellular expression of aggrecan as well as in the cytoplasm of glial cells (arrow). B: Cerebellum, white matter, chronic lesion, prominent intralesional aggrecan deposition mainly in macrophages (arrows). Aggrecan immunohistochemistry (DAB). C: Cerebellum, white matter, acute lesion, prominent phosphacan immunoreactivity (arrows). D: Cerebellum, white matter, chronic lesion, with moderate to severe reduction of phosphacan immunoreactivity within the lesion (asterisks) corresponding to perivascular inflammatory cell infiltration. Phosphacan immunohistochemistry (DAB). E: Cerebellum, white matter, acute lesion, mild expression of fibronectin in vascular walls (arrow). F: Cerebellum, white matter, chronic lesion, extensive intralesional reticular extracellular deposition of fibronectin throughout the lesion (arrows). Fibronectin immunohistochemistry (DAB). All scale bars = 50 um.",
           "Table 7. p-values of the multiple pair-wise comparisons of geometric means (Tukey- kramer test) immunohistochemistry (logarithmic data).",
           "Fig 3. Immunohistochemical detection of collagen type I and IV as well as laminin in CDV-infected canine brains. A: Cerebellum, white matter, acute lesion, mild to moderate type I collagen expression associated with basement membranes. B: Cerebellum, white matter, chronic lesion, extracellular deposition of type I collagen, associated with vascular walls (arrow). Type I collagen immunohistochemistry (DAB). C: Cerebellum, white matter, acute lesion, mild to moderate type IV collagen expression around basement membranes of vascular walls (arrow). D: Cerebellum, white matter, chronic lesion, extensive extracellular deposition of type IV collagen (arrows) and intracellular type IV collagen detection in macrophages. Type IV collagen immunohistochemistry (DAB). E: Cerebellum, white matter, acute lesion, laminin expression is associated with basement membranes of blood vessels (arrow). F: Cerebellum, white matter, chronic lesion, with nearly complete absence of laminin immunoreactivity in the demyelinating lesion. Laminin immunohistochemistry (DAB). All scale bars = 50 um.",
           "Fig 4. Venn diagram of differentially expressed ECM-associated genes in canine distemper. The Venn diagram displays the intersection of differentially expressed genes associated with the synthesis and degradation of extracellular matrix in acute, subacute, and chronic canine distemper lesions as obtained by re-analysis of a previously published microarray study (ArrayExpress E-MEXP-3917)[8].",
           "Table 8. List of differentially expressed genes (DEGs) by microarray analysis."]

# http://journals.plos.org/plosone/article?id=10.1371%2Fjournal.pone.0159777
# fig 5 ** 4,3,5,1,2,6
PLOS_22 = ["We fabricated droplet-based microfluidic platform for copolymerizable microspheres with acrydite modified DNA probe. The copolymerizable 3-D polyacrylamide microspheres were successfully produced from microcontinuous-flow synthesis with on-channel solidification. DNA copolymerization activity, surface presentation and thermostability were assessed by using fluorescent labeled complementary probe. The binding performance was only visible on the surface area of oligo-microspheres. We show that the resulting oligo-microspheres can be directly integrated into a streamlined microsphere-PCR protocol for amplifying ssDNA. Our microspheres could be utilized as a potential material for ssDNA analysis such as DNA microarray and automatic DNA SELEX process.",
           "Fig 1. A graphical summary of the Microsphere-PCR protocol. The Microsphere-PCR synthesizes ssDNA copy of a DNA template.",
           "Table 1. Sequences of oligonucleotides used to amplify ssDNA based on microsphere-PCR.",
           "Fig 2. Fabricated polymeric droplet-based microfluidic platform and on-flow microsphere synthesis. (A) Droplet-based microfluidic platform, (B) Flow-focusing geometry. (C) Overall experimental set-up for producing polyacrylamide microspheres. (D) On-flow synthesis of polyacrylamide microsphere in the main channel. The inset figure shows the ccaptured image being formed microspheres.",
           "Fig 3. DNA presentation and fluorescence image of probe binding to the surface of 3-D microsphere. (A) The fluorescent signal demonstrated that the oligonucleotides (5'Acrydite-modified DNA probe, Ap) on the surface of the microsphere were capable of hybridizing to the complementary fluorescent labeled DNA probes. (B) The complementary binding performance was visible only on the surface area of Ap-contained microsphere. (C) Z-stack fluorescence images of a single microsphere captured with 3D confocal fluorescence microscopy at different Z-planes. Non-specific interaction or absorption of complementary DNA probes to any of the inside area was not observed in Z-stack fluorescent images. Images were collected at 7.6 um intervals using a 488 laser.",
           "Fig 4. Thermostability of Ap-contained oligo-microsphere.",
           "Fig 5. Microsphere-PCR for amplifying ssDNA. (A) The DNA amplicons were run on a 2% agarose gel followed by EtBr staining. (B) Agarose gel configuration to determine the relative location of PCR bands and their intensity from asymmetric PCR and microsphere-PCR experiments. M: Synthesized ssDNA Marker (76 nt, 100 nt), Lane 1: Asymmetric PCR (primer (F/R) ratio_20: 1), Lane 2: Microshperes-PCR. (C) Melting Tm analysis for synthetic DNA hybrids (Reference Duplex, RD) and Microsphere-PCR dsDNA byproducts. Co-polymerized Ap-microspheres was tested as a control. (ND: Not detected)."]

# http://journals.plos.org/plosone/article?id=10.1371%2Fjournal.pone.0159648
# fig 2  6,3,1,4,2,5
PLOS_23 = ["Massively parallel sequencing (MPS) combined with bioinformatic analysis has been widely applied to detect fetal chromosomal aneuploidies such as trisomy 21, 18, 13 and sex chromosome aneuploidies (SCAs) by sequencing cell-free fetal DNA (cffDNA) from maternal plasma, so-called non-invasive prenatal testing (NIPT). However, many technical challenges, such as dependency on correct fetal sex prediction, large variations of chromosome Y measurement and high sensitivity to random reads mapping, may result in higher false negative rate (FNR) and false positive rate (FPR) in fetal sex prediction as well as in SCAs detection. Here, we developed an optimized method to improve the accuracy of the current method by filtering out randomly mapped reads in six specific regions of the Y chromosome. The method reduces the FNR and FPR of fetal sex prediction from nearly 1% to 0.01% and 0.06%, respectively and works robustly under conditions of low fetal DNA concentration (1%) in testing and simulation of 92 samples. The optimized method was further confirmed by large scale testing (1590 samples), suggesting that it is reliable and robust enough for clinical testing.",
           "Fig 1. Discovery of six special regions in Y chromosome. A and B show the six highly covered regions (highlighted by red line) in Y chromosome in female and male group. They were located at the same positions of Y chromosome in two groups. C) There are detection counts of six regions in Y chromosome of 92 samples. D) The comparison of Y chromosome reads percentage in the female and male fetus groups in before and after filtering out the reads in six regions of Y chromosome.",
           "Table 1. Basic information of 92 samples of pregnant women.",
           "Table 2. The value for calculation of fetal DNA concentration.",
           "Table 3. The calculated percentage of Y chromosome using original and optimized methods.",
           "Table 4. The six regions discovered in Y chromosome.",
           "Fig 2. Testing of the optimized method in large-scale sample. A) The DNA percentages of Y chromosome calculated by optimized method and original method. Here, the fetal DNA concentration is assumed as 1%. The percentage of Y chromosome calculated by the optimized method was marked as green point. The blue point illustrates the percentage of Y chromosome calculated by original method. B and C) the percentage of Y chromosome calculated by optimized method and original method in 1590 samples. Here, black and red dots represent the percentage of Y chromosome in female fetuses and male fetus, respectively. Error prediction results were marked by blue and green dots in Fig C in which two male fetuses were predicted to be female and marked by blue; a female fetus predicted to be male and marked by green.",
           ]

# http://journals.plos.org/plosone/article?id=10.1371%2Fjournal.pone.0159580
# fig6 **      6,4,3,11,8,5,10,9,2,7,1
PLOS_24 = ["Bone healing involves a variety of different cell types and biological processes. Although certain key molecules have been identified, the molecular interactions of the healing progress are not completely understood. Moreover, a clinical routine for predicting the quality of bone healing after a fracture in an early phase is missing. This is mainly due to a lack of techniques to comprehensively screen for cytokines, growth factors and metabolites at their local site of action. Since all soluble molecules of interest are present in the fracture hematoma, its in-depth assessment could reveal potential markers for the monitoring of bone healing. Here, we describe an approach for sampling and quantification of cytokines and metabolites by using microdialysis, combined with solid phase extractions of proteins from wound fluids. By using a control group with an isolated soft tissue wound, we could reveal several bone defect-specific molecular features. In bone defect dialysates the neutrophil chemoattractants CXCL1, CXCL2 and CXCL3 were quantified with either a higher or earlier response compared to dialysate from soft tissue wound. Moreover, by analyzing downstream adaptions of the cells on protein level and focusing on early immune response, several proteins involved in the immune cell migration and activity could be identified to be specific for the bone defect group, e.g. immune modulators, proteases and their corresponding inhibitors. Additionally, the metabolite screening revealed different profiles between the bone defect group and the control group. In summary, we identified potential biomarkers to indicate imbalanced healing progress on all levels of analysis.",
           "Table 1. List of used enzyme-linked immunosorbent assay (ELISA) kits. All kits were purchased as indicated and used according to the manufacture's guide.",
           "Fig 1. Concentration of chemokines and cytokines in the dialysate from bone defect (BD) and soft tissue wound (STW) 0-24 h after injury. Samples were analyzed by ELISA. Values are mean + SD. IL-6, interleukin-6; CXCL2, chemokine (C-X-C motif) ligand 2; CXCL7, chemokine (C-X-C motif) ligand 7; CXCL3, chemokine (C-X-C motif) ligand 3; CXCL1, chemokine (C-X-C motif) ligand 1; CXCL4, chemokine (C-X-C motif) ligand 4; IL-10, inteleukin-10; TGF-B1, transforming growth factor-B1; SDF-1, stromal-cell derived factor-1. * p < 0.05, ** p < 0.01, *** p < 0.001.",
           "Fig 2. Analysis of the fracture hematoma from BD 12 h and 24 h after surgery. (A) Number of mononucleated and multinucleated cells in the fracture hematoma. Cells were counted in 10 randomly distributed squares per section on three subsequent slices per fracture hematoma. (B) Pappenheim staining of fracture hematoma 12 hours after surgery. (C) Pappenheim staining of a fracture hematoma 24 h after surgery. Magnification 100x. *, neutrophil granulocyte; #, monocyte; black arrow, basophil granulocyte; white arrow, eosinophil granulocyte.",
           "Fig 3. Comparative analysis of protein adsorbates sampled from bone defect (BD) and soft tissue wound (STW) at 12 h after injury. n = 4 for each defect scenario. (A) Volcano plot of proteins quantified in both defect scenarios: Data points represent the Log10-transformed ratios of the mean quantitative values gained from BD or STW and the -Log10-transformed p-value of the comparative analysis. (B) Proportions of unique and specific proteins for a certain wound scenario among all quantified proteins. The numbers in brackets represent the number of proteins assigned to the corresponding groups.",
           "Fig 4. Principle component analysis (PCA). n = 4 for each defect scenario. PCA based on protein quantitative values gained from each replicate of bone defect (BD) and soft tissue wound (STW) at 12 h after injury.",
           "Fig 5. Signaling pathway analysis. n = 4 for each defect scenario. Regulation of selected pathways focusing on cellular migration (A) and immune response (B) revealed by Ingenuity Pathway Analysis (IPA). The quantitative values of all proteins were used. Z-score reflects the pathway activity with orange indicating an increased activity in the bone defect group and blue a decreased activity, respectively.",
           "Table 2. Identified proteins clustered to extracellular matrix proteins. Identified proteins from proteomic analysis at 12 h after injury. Gene symbols and protein names correspond to Uniprot knowledgebase. Quantitative values are given as Log10-transformed fold changes BD/STW. Significances are indicated as asterisks with * p-value < 0.05, ** p-value < 0.01 and *** p-value < 0.001. n = 4 for each defect scenario.",
           "Table 3. Identified proteins clustered to immune modulators. Identified proteins from proteomic analysis at 12 h after injury. Gene symbols and protein names correspond to Uniprot knowledgebase. Quantitative values are given as Log10-transformed fold changes BD/STW. Significances are indicated as asterisks with * p-value < 0.05, ** p-value < 0.01 and *** p-value < 0.001. n = 4 for each defect scenario.",
           "Table 4. Identified proteins clustered to proteases. Identified proteins from proteomic analysis at 12 h after injury. Gene symbols and protein names correspond to Uniprot knowledgebase. Quantitative values are given as Log10-transformed fold changes BD/STW. Significances are indicated as asterisks with * p-value < 0.05, ** p-value < 0.01 and *** p-value < 0.001. n = 4 for each defect scenario.",
           "Table 5. Identified proteins clustered to protease inhibitors. Identified proteins from proteomic analysis at 12 h after injury. Gene symbols and protein names correspond to Uniprot knowledgebase. Quantitative values are given as Log10-transformed fold changes BD/STW. Significances are indicated as asterisks with * p-value < 0.05, ** p-value < 0.01 and *** p-value < 0.001. n = 4 for each defect scenario.",
           "Fig 6. Metabolomic results in the dialysate from bone defect (BD) and soft tissue wound (STW) 12-24 h after injury. (A) PCA comparisons of BD vs. STW after 12-15h (I) and 21-24h (II) as well as monitoring of STW (III) and BD after 12-15h vs. 15-18h (IV). n = 6 for soft tissue wound and n = 3 for bone defect. (B) Absolute concentration of serine in STW and BD wound fluids (I). Summarized concentrations of all monitored amino acids, acylcarnitines, and phosphatidylcholines-aa are shown in II-IV. Bar charts are depicted as mean + SD, * indicates a p-value < 0.05. n = 6 for soft tissue wound and n = 2 for bone defect."]

# http://journals.plos.org/plosone/article?id=10.1371%2Fjournal.pone.0159764
# fig 7 ** 0,2,1,4,8,3,6,9,10,7,5
PLOS_25 = ["After traumatic spinal cord injury, an acute phase triggered by trauma is followed by a subacute phase involving inflammatory processes. We previously demonstrated that peripheral serum cytokine expression changes depend on neurological outcome after spinal cord injury. In a subsequent intermediate phase, repair and remodeling takes place under the mediation of growth factors such as Insulin-like Growth Factor 1 (IGF-1). IGF-1 is a promising growth factor which is thought to act as a neuroprotective agent. Since previous findings were taken from animal studies, our aim was to investigate this hypothesis in humans based on peripheral blood serum. Forty-five patients after traumatic spinal cord injury were investigated over a period of three months after trauma. Blood samples were taken according to a fixed schema and IGF-1 levels were determined. Clinical data including AIS scores at admission to the hospital and at discharge were collected and compared with IGF-1 levels. In our study, we could observe distinct patterns in the expression of IGF-1 in peripheral blood serum after traumatic spinal cord injury regardless of the degree of plegia. All patients showed a marked increase of levels seven days after injury. IGF-1 serum levels were significantly different from initial measurements at four and nine hours and seven and 14 days after injury, as well as one, two and three months after injury. We did not detect a significant correlation between fracture and the IGF-1 serum level nor between the quantity of operations performed after trauma and the IGF-1 serum level. Patients with clinically documented neurological remission showed consistently higher IGF-1 levels than patients without neurological remission. This data could be the base for the establishment of animal models for further and much needed research in the field of spinal cord injury.",
           "Table 1. Blood sample protocol. Numbered list of blood sample collection with corresponding time-points.",
           "Table 2. Demographic and clinical characteristics of subjects. Abbreviations: NLI = Neurological Level of Injury; AO = AO-Classification; AIS = ASIA (American Spinal Injury Association) Impairment Scale; Paralysis: cP = complete Paraplegia, iC = incomplete Paraplegia, cT = complete Tetraplegia, iT = incomplete Tetraplegia; GCS = Glasgow Coma Scale. Age is expressed as mean years + standard deviation. Neurological remission was defined as improvement in AIS. Four patients had posttraumatic spinal cord injury (SCI) without vertebral fractures.",
           "Table 3. Odds ratios, CIs' and p-values of variables included the preferred logistic regression models I (Table 3A) and II (Table 3B). Abbreviations: Paralysis: cP = complete Paraplegia, iC = incomplete Paraplegia, cT = complete Tetraplegia, iT = incomplete Tetraplegia; NLI = Neurological Level of Injury; MV = mean IGF-1 serum levels of the first 24 h; A_DB = dichotomized initial AIS = ASIA (American Spinal Injury Association) Impairment Scale, 0 = A, 1 = B-D; AGE = age of the subjects.",
           "Fig 1. Mean IGF-1 serum levels of all traumatic SCI patients. During observation period of 3 months after injury, expressed as mean values + standard error of the mean. The Wilcoxon signed-rank test assessed significant differences from the admission level (0h), *p < 0.05, **p < 0.001.",
           "Fig 2. IGF-1 serum level Boxplots and Whiskers of all traumatic SCI patients. IGF-1 serum levels during 3-month observation period are displayed as boxplots and whiskers. Whiskers are displayed as 97,5 (upper whisker) and 2,5 (lower whisker) confidence intervals. Dots are indicating outliers.",
           "Fig 3. Mean IGF-1 serum level comparison of all patients with (G1) and without (G2) neurological remission. Expressed as mean values + standard error of the mean. The Mann-Whitney-U-Test assessed significant differences between both groups at each particular time-point, *p < 0.05. For better illustration mean values of all traumatic SCI patients had been added to the figure.",
           "Fig 4. IGF-1 serum level comparison of all AIS A classified patients and all AIS B-D classified patients. Expressed as mean values + standard error of the mean. The Mann-Whitney-U-Test assessed significant differences between both groups at each particular time-point, *p < 0.05.",
           "Fig 5. Location shifts between IGF-1 levels of 3 to 7 days after injury within all patients, G1 and G2. IGF-1 serum levels are expressed as mean values + standard error of the mean. The Wilcoxon signed-rank test assessed significant differences from the IGF-1 level 3 days to 7 days after injury, *p < 0.05.",
           "Fig 6. ROC Analysis of Model 1. Favored predictive model including all variables as predictors shown in Model 1 Table 3A (such as Paralysis: cP = complete Paraplegia, iC = incomplete Paraplegia, cT = complete Tetraplegia, iT = incomplete Tetraplegia; NLI = Neurological Level of Injury; MV = mean IGF-1 serum levels of the first 24 h; A_DB = dichotomized initial AIS = ASIA Impairment Scale, 0 = A, 1 = B-D; AGE = age of the subjects) which enables us to differentiate between neurological remission and no neurological remission in 81.8% [CI: 68.4%-95.1%] of cases. Grey highlighted area represents the particular confidence interval. Confidence interval of the model's AUC in brackets. Model is based on n = 45 cases.",
           "Fig 7. ROC Analysis of Model 2. Predictive model including the mean IGF-1 serum levels of the first 24h as predictor shown in Model 2 Table 3B. The model enables us to differentiate between neurological remission and no neurological remission in 59.5% [CI: 41.9%-77.1%] of cases. Confidence interval of the model's AUC in brackets. Model is based on n = 45 cases."]

# http://journals.plos.org/plosone/article?id=10.1371%2Fjournal.pone.0159601
# fig6** 1,3,2,6,4,5
PLOS_26 = ["Identification of dysregulated microRNAs (miRNAs) in prostate cancer is critical not only for diagnosis, but also differentiation between the aggressive and indolent forms of the disease. miR-9 was identified as an oncomiR through both miRNA panel RT-qPCR as well as high-throughput sequencing analysis of the human P69 prostate cell line as compared to its highly tumorigenic and metastatic subline M12, and found to be consistently upregulated in other prostate cell lines including DU-145 and PC3. While miR-9 has been characterized as dysregulated either as an oncomiR or tumour suppressor in a variety of other cancers including breast, ovarian, and nasopharyngeal carcinomas, it has not been previously evaluated and proven as an oncomiR in prostate cancer. miR-9 was confirmed an oncomiR when found to be overexpressed in tumour tissue as compared to adjacent benign glandular epithelium through laser-capture microdissection of radical prostatectomy biopsies. Inhibition of miR-9 resulted in reduced migratory and invasive potential of the M12 cell line, and reduced tumour growth and metastases in male athymic nude mice. Analysis showed that miR-9 targets e-cadherin and suppressor of cytokine signalling 5 (SOCS5), but not NF-kB mRNA. Expression of these proteins was shown to be affected by modulation in expression of miR-9.",
           "Fig 1. miR-9 levels are upregulated in all prostate cancer cell line models. A: miR-9 levels were significantly upregulated in M12 cells as relative to the parental P69 line (p = .002). Data is the mean of 3 independent experiments, each performed in duplicate (F6) or triplicate (P69, M2182, M12). B: miR-9 levels in DU-145 and PC3 cells are upregulated as relative to P69 cells (p <.001). Data was normalized to RNU48 (dCT) and expressed relative to P69 using the comparative CT method. Data is the mean of three technical replicates. C: miR-9 expression is upregulated in 75% of positive patient tumours. miR-9 levels were measured in tumour and benign tissue separated from prostate biopsies using laser-captured microdissection (LCM). Data was normalized to RNU48 and expressed relative to the benign tissue using the comparative CT method, and is the average of 3 technical replicates. miR-9 was undetected in both benign and tumour of the fifth patient, and thus was not included in the analysis.",
           "Fig 2. Inhibition of miR-9 significantly reduces migratory and invasive potential, but has no effect on cell proliferation. M12 cells were stably transformed with scrambled control or miR-9 inhibitor, plated on a ThinCert transwell membrane (with basement membrane added for invasion assay) and assessed for (A) migratory (p < 0.0001), (B) invasive potential (p < 0.0001), and (C) proliferation. Data is the mean of 3 independent experiments, each performed in triplicate (A&B) or one representative of three independent experiments (C).",
           "Fig 3. Tumour growth and metastasis is reduced upon miR-9 inhibition. A: Subcutaneous injections into the flank of the respective cell lines into nude athymic mice. Tumour reduction was significantly reduced in mice injected with M12 cells transformed with miR-9 inhibitor (p <.0001). Results are reported as the average ratio of tumour volume to the final average M12 tumour volume. N = 5 mice for each treatment. B: Intraprostatic (IP) injection into nude athymic mice resulted in 0 metastatic lesions when miR-9 was inhibited, as compared to 7 lesions in the mouse injected with M12 cells alone.",
           "Fig 4. messenger RNA and protein levels of CDH1 are increased upon miR-9 inhibition. A: RT-qPCR analysis shows that mRNA levels are impacted by miR-9, and miR-9 inhibition relieves mRNA and protein levels (p<0.03 for P69 and miR-9 inhibited lines vs. M12). mRNA is normalized to GAPDH and reported as relative to the M12 cell line using the comparative CT method. Results are compiled data from two biological replicates, each performed in triplicate. B: Proven binding site for miR-9 in e-cadherin. Adapted from RNAHybrid [21,22] analysis of CDH1 mRNA (NM_004360.3). miR-9 is the left sequence, and hybridized on the right is the complementary region of the 3'-UTR of CDH1. C: Western blot analysis and D: quantitation. Blot is representative of 5 independent experiments, Quantitation the average of 3 independent experiments normalized to B-Actin and reported as relative to the M12 cell line (p<0.05). (E) e-cadherin expression is suppressed by miR-9. M12 cells were transiently transfected with a firefly luciferase reporter construct containing a portion of the CDH1 3'-UTR, with the wild type or mutated miR-9 binding seed region along with a renilla luciferase plasmid. Firefly luciferase expression is reported as normalized to renilla luciferase activity and relative to mutated seed region expression. Results are the mean of 2 independent experiments, each performed in triplicate. (p <0.01). Western blot analysis shows that NF-kB levels do not change significantly between M12 or M12 cells stably transfected with miR-9 inhibitor. Blot (F) is representative of 3 independent experiments and quantitation (G) is from one representative experiment.",
           "Fig 5. While protein levels of SOCS5 are increased upon miR-9 inhibition, messenger RNA levels are not significantly different. SOCS5 expression in the P69, M12, and M12 stably transformed with vector expressing miR-9 inhibitor. A: messenger RNA relative quantitation shows that mRNA levels are not significantly impacted by miR-9. mRNA is normalized to GAPDH and reported as relative to the M12 cell line; results are compiled data from three independent lysates, each performed in triplicate. Regardless of similar SOCS5 mRNA levels, (B) Western blot analysis and (C) quantitation show that miR-9 inhibition results in increased levels of SOCS5. Blot is representative of 6 independent experiments, Quantitation the average of 3 independent experiments normalized to B-Actin and reported as relative to the M12 cell line (p<0.05). (D) The proven SOCS5 mRNA:miR-9 hybrid [21,22]. miR-9 is the top sequence, and hybridized below is the complementary region of the 3'-UTR of SOCS5. (E) SOCS5 expression is suppressed by miR-9. M12 cells were transiently transfected with a dual luciferase reporter construct containing a portion of the SOCS5 3'-UTR, with the wild type or mutated miR-9 binding seed region. Firefly luciferase expression is reported as normalized to renilla luciferase activity and relative to mutated seed region expression. Results are the mean of 2 independent experiments, each performed in triplicate. (p = .016).",
           "Fig 6. miR-9's mode of action on tumour progression through e-cadherin and SOCS5. In the presence of miR-9 (left), e-cadherin message is cleaved and suppressed, resulting in less protein production. This allows for accumulation of B-catenin (grey 'L'), which can then diffuse into the nucleus, activate transcription factors and drive pro-survival, pro-proliferation transcription including c-Myc and Cyclin D1. c-Myc then initiates additional miR-9 message through a positive feedback signal. SOCS5 translation is likewise suppressed when miR-9 is overexpressed, resulting in less protein production. This allows phosphorylation and signal transduction, resulting in p-STAT transcriptional activation of pro-survival, proliferation, and invasion/mestatasis oncogenes. In the noncancerous tissue in which miR-9 expression levels are low (right panel), e-cadherin is produced and sequesters B-catenin, preventing activation of transcriptional events. Likewise, SOCS5 is produced and works to prevent phosphorylation of JAk kinase and STAT while also promoting ubiquitination, thus attenuating the JAk/STAT signaling cascade, and resulting in decreased transcription of survival, proliferation, and invasion genes."]

# http://journals.plos.org/plosone/article?id=10.1371%2Fjournal.pone.0159360
# fig3  5,3,2,4,1
PLOS_27 = ["Quantifying and predicting the antigenic characteristics of a virus is something of a holy grail for infectious disease research because of its central importance to the emergence of new strains, the severity of outbreaks, and vaccine selection. However, these characteristics are defined by a complex interplay of viral and host factors so that phylogenetic measures of viral similarity are often poorly correlated to antigenic relationships. Here, we generate antigenic phylogenies that track the phenotypic evolution of two serotypes of foot-and-mouth disease virus by combining host serology and viral sequence data to identify sites that are critical to their antigenic evolution. For serotype SAT1, we validate our antigenic phylogeny against monoclonal antibody escape mutants, which match all of the predicted antigenic sites. For serotype O, we validate it against known sites where available, and otherwise directly evaluate the impact on antigenic phenotype of substitutions in predicted sites using reverse genetics and serology. We also highlight a critical and poorly understood problem for vaccine selection by revealing qualitative differences between assays that are often used interchangeably to determine antigenic match between field viruses and vaccine strains. Our approach provides a tool to identify naturally occurring antigenic substitutions, allowing us to track the genetic diversification and associated antigenic evolution of the virus. Despite the hugely important role vaccines have played in enhancing human and animal health, vaccinology remains a conspicuously empirical science. This study advances the field by providing guidance for tuning vaccine strains via site-directed mutagenesis through this high-resolution tracking of antigenic evolution of the virus between rare major shifts in phenotype.",
           "Fig 1. Phylogenetic tree for serotypes O and SAT1. (a) Phylogeny of all serotype O viruses studied, with alternate shading showing geographically-isolated subtypes (topotypes), labelled with their names in red (1, 2, 3 and 4 are East Africa topotypes EA-1, EA-2, EA-3 and EA-4, and W is the West Africa topotype, WA). The three unshaded viruses are all sole representatives of their topotype. (b) Phylogeny of all serotype SAT1 viruses studied (including those from[25]), with alternate shading showing topotypes, labelled with their numbers in red. The two unshaded viruses are sole representatives of their topotype. Phylogenies are time-resolved, with branch lengths measured in evolutionary time rather than substitutions.",
           "Table 1. Residues identified by antigenic site analysis as forming part of FMDV epitopes.",
           "Fig 2. Antigenic sites on serotypes O and SAT1. The molecular surface of (a) serotype O and (b) serotype SAT1 icosahedral protomers is shown using pymol (Schrodinger LLC), with the molecular surface of neighbouring protomers shown as a light grey mesh (refer to inset icosahedral representation of an FMDV capsid for orientation). The molecular surface is white, with known antigenic sites on this serotype in orange (red when they were identified in this study), and known antigenic sites for any other serotype in yellow (see Table 1 and S3 Table for details). The location of the residues on the VP1 BG-BH loop could not be resolved on the serotype SAT1 structure, and so sites on that loop are not shown. Blue highlights mark the beginning (VP1 139) and end (VP1 165, with pale dot) of the disordered area, and the approximate locations of all four surface-exposed regions identified in Table 1 are also circled in blue and marked as before with *1-*4. Residues identified in this study but not in these regions are individually labelled with their positions on the common alignment. The three-fold and five-fold axes of symmetry are shown by triangles and pentagons respectively.",
           "Table 2. Antigenic impact of introduced substitutions on VNT and LPBE titres.",
           "Fig 3. Antigenic evolution of Serotypes O and SAT1. Phylogeny of all serotype O viruses (a) and serotype SAT1 (b) are shown with shading and labelling of topotypes as Fig 1A-1B. Different branch colours denote antigenic dissimilarity with respect to VNT between the viruses derived from analyses of the sequence and serological data and ancestral state reconstruction. Conversely, the same colours in the same phylogeny denote the same antigenic phenotype. +Change in antigenic phenotype for which no causative substitution has been identified. The broadest such group defined by VNT is starred for serotype O (spanning 5 topotypes) and SAT1 (spanning 2 topotypes). As in Fig 1, phylogenies are time-resolved, with branch lengths measured in evolutionary time rather than substitutions."]

# http://journals.plos.org/plosone/article?id=10.1371%2Fjournal.pone.0159069
# fig12 16,15,3,4,1,2,5,6,7,13,14,8,10,9,11,12
PLOS_28 = ["Wireless sensor networks consist of resource limited devices. Most crucial of these resources is battery life, as in most applications like battle field or volcanic area monitoring, it is often impossible to replace or recharge the power source. This article presents an energy efficient collaborative communication system based on spread spectrum to achieve energy efficiency as well as immunity against jamming, natural interference, noise suppression and universal frequency reuse. Performance of the proposed system is evaluated using the received signal power, bit error rate (BER) and energy consumption. The results show a direct proportionality between the power gain and the number of collaborative nodes as well as BER and signal-to-noise ratio (Eb/N0). The analytical and simulation results of the proposed system are compared with SISO system. The comparison reveals that SISO perform better than collaborative communication in case of small distances whereas collaborative communication performs better than SISO in case of long distances. On the basis of these results it is safe to conclude that collaborative communication in wireless sensor networks using wideband systems improves the life time of nodes in the networks thereby prolonging the network's life time.",
           "Table 1. Details of off-the-shelf products with data and parameters used in simulation.",
           "Fig 1. Geometry of sensor nodes.",
           "Fig 2. System model.",
           "Fig 3. Normalized average received power vs. number of collaborative nodes with Rayleigh fading.",
           "Fig 4. Average received power/N vs. number of collaborative nodes with Rayleigh fading.",
           "Fig 5. BER over interval {-0.1pi to 0.1pi} for different number of nodes with fading and total transmitted energy Eb/N0.",
           "Fig 6. BER over interval {-0.2pi to 0.2pi} for different number of nodes with fading and total transmitted energy Eb/N0.",
           "Fig 7. Percentage energy savings and break-even distances over {-0.1pi to 0.1pi}, for AT86RF212, with different number of nodes.",
           "Fig 8. Percentage energy savings and break-even distances over {-0.3pi to 0.3pi} for AT86RF212, with different number of nodes.",
           "Fig 9. Percentage energy savings and break-even distance over {-0.1pi to 0.1pi} for CC2420, with different number of nodes.",
           "Fig 10. Percentage energy savings and break-even distance ove {-0.3pi to 0.3pi} for CC2420, with different number of nodes.",
           "Table 2. Break-even distance for CC2420 and AT86RF212.",
           "Table 3. Percentage energy preservation based on the parameters of CC2420.",
           "Table 4. Percentage energy preservation based on the parameters of AT86RF212.",
           "Fig 11. Multihop system in case of node failure.",
           "Fig 12. collaborative communication in case of node failure.",
           ]

# http://journals.plos.org/plosone/article?id=10.1371%2Fjournal.pone.0158868
# fig 5** 4,5,1,3,2
PLOS_29 = ["Borneol is a compound widely used in ophthalmic preparations in China. Little is known about its exact role in treating eye diseases. Here we report that transient receptor potential melastatin 8 (TRPM8) channel is a pharmacological target of borneol and mediates its therapeutic effect in the eyes. Ca2+ measurement and electrophysiological recordings revealed that borneol activated TRPM8 channel in a temperature- and dose-dependent manner, which was similar to but less effective than the action of menthol, an established TRPM8 agonist. Borneol significantly increased tear production in guinea pigs without evoking nociceptive responses at 25C, but failed to induce tear secretion at 35C. In contrast, menthol evoked tearing response at both 25 and 35C. TRPM8 channel blockers N-(3-Aminopropyl)-2-[(3-methylphenyl)methoxy]-N-(2-thienylmethyl)benzamide hydrochloride (AMTB) and N-(4-tert-butylphenyl)-4-(3-chloropyridin-2-yl)piperazine-1-carboxamide (BCTC) abolished borneol- and menthol-induced tear secretion. Borneol at micromolar concentrations did not affect the viability of human corneal epithelial cells. We conclude that borneol can activate the cold-sensing TRPM8 channel and modestly increase ocular surface wetness, which suggests it is an active compound in ophthalmic preparations and particularly useful in treating dry eye syndrome.",
           "Fig 1. Borneol stimulated Ca2+ influx in HEk293 cells overexpressing TRPM8 channel. (A-B) In the presence of 1.5 mM extracellular Ca2+, borneol and menthol induced an increase of cytosolic Ca2+ in TRPM8-expressing cells, but not in the control HEk293 cells without TRPM8 expression at 25 and 35C. The rise of Ca2+ was inhibited by 10 uM AMTB, a selective TRPM8 blocker. (C) Efficacy of 100 uM borneol and menthol on inducing Ca2+ influx into TRPM8-expressing cells at 25 and 35C. (D) In the absence of extracellular Ca2+, borneol did not change the intracellular Ca2+ level, while subsequent addition of extracellular Ca2+ resulted in robust Ca2+ influx. (E) Comparison between the effects of borneol and menthol at different concentrations on the induction of Ca2+ influx in TRPM8-expressing cells. ** P<0.01, *** P<0.001. Sample numbers are shown in the bars.",
           "Fig 2. Borneol potentiated TRPM8 currents in transfected HEk293 cells. (A-B) Acute stimulation of whole-cell currents in TRPM8-expressing cells by 100 uM borneol at 25 and 35C. Subsequent superfusion of 10 uM AMTB potently inhibited the currents. The inset shows current-voltage (I-V) relationships collected from the time points 1, 2 and 3. (C) Borneol (100 uM) showed no effect on the whole-cell currents in non-transfected HEk293 cells. (D-E) Menthol (100 uM) potently augmented the whole-cell currents in TRPM8-expressing cells at 25 and 35C. (F) Traces for the mean values of whole-cell current density-voltage relationships at conditions indicated. (G) Statistical results for values extracted from +80mV in panel F. Sample numbers are shown in the bars. (H) Dose-dependency of borneol on the activation of whole-cell TRPM8 currents at 25C. The ratio between borneol-induced peak current and the basal current before borneol application was used.",
           "Fig 3. Effect of borneol on TRPM8 currents in transfected HEk293 cells recorded under inside-out configuration. (A) Typical recordings of single channel currents at +100 mV and 25C. Borneol at 100 uM was used. Dashed arrows indicate expansions from underlined region above. (B) Amplitude histograms of the single channel activity. The histograms could not be fitted with a simple Gaussian function, reflecting the existence of multiple conducting states.",
           "Fig 4. Borneol evoked tear secretion without nociceptive effect in guinea pigs. (A) RT-PCR of TRPM8 mRNA expressed in the cornea of guinea pig. N.C., negative control. (B) 100 uM borneol and menthol were applied to the ocular surface at 25 and 35C to induce tear production (** P<0.01, *** P<0.001 vs vehicle at the same temperature; ## P<0.01, ### P<0.001 between groups indicated; n = 10 in each group). (C) TRPM8 channel blockers AMTB (10 uM) and BCTC (20 uM) inhibited borneol- and menthol-induced tear production (*** P<0.001 vs control; n = 10 in each group). (D) Total time spent for eye wiping with paws by the animals within 5 min after the drop of 100 uM borneol and menthol solution to the corneas. No significantly difference was observed between groups (n = 4 in each group).",
           "Fig 5. Effects of borneol and menthol on viability of human corneal epithelial (HCE) cells and TRPM8-expressing HEk293 cells. (A) Both borneol and menthol at 1 mM significantly reduced the viability of HCE cells. (B) Borneol at 0.1-1000 uM did not affect the viability of TRPM8-HEk293 cells, while menthol at 100 and 1000 uM showed strong inhibition on cell viability. (*** P<0.001 vs control; ## P<0.01, ### P<0.001 between indicated columns; n = 8 in each column)."]

# http://journals.plos.org/plosone/article?id=10.1371%2Fjournal.pone.0157744
# fig 3 **1,4,3,5,6,2
PLOS_30 = ["Given the public health importance of improving dietary behavior in chronic disease prevention in low- and middle-income countries it is crucial to understand the factors influencing dietary behavior in these settings. This study tested the validity of a conceptual framework linking individual and environmental factors to dietary behavior among Ecuadorian adolescents aged 10-16 years. A cross-sectional survey was conducted in 784 school-going Ecuadorian adolescents in urban and rural Southern Ecuador. Participants provided data on socio-economic status, anthropometry, dietary behavior and its determining factors. The relationships between individual (perceived benefits and barriers, self-efficacy, habit strength, and a better understanding of healthy food) and environmental factors (physical environment: accessibility to healthy food; social environment: parental permissiveness and school support), and their association with key components of dietary behavior (fruit and vegetables, sugary drinks, breakfast, and unhealthy snack intake) were assessed using structural equation modeling. The conceptual model performed well for each component of eating behavior, indicating acceptable goodness-of-fit for both the measurement and structural models. Models for vegetable intake and unhealthy snacking showed significant and direct effects of individual factors (perceived benefits). For breakfast and sugary drink consumption, there was a direct and positive association with socio-environmental factors (school support and parental permissiveness). Access to healthy food was associated indirectly with all eating behaviors (except for sugary drink intake) and this effect operated through socio-environmental (parental permissiveness and school support) and individual factors (perceived benefits). Our study demonstrated that key components of adolescents' dietary behaviors are influenced by a complex interplay of individual and environmental factors. The findings indicate that the influence of these factors varied by type of dietary behavior.",
           "Fig 1. Conceptual framework for healthy dietary behavior in an Ecuadorian population.",
           "Table 1. Participant characteristics.",
           "Table 2. key components of dietary behaviour as measured by two 24h dietary recalls.",
           "Table 3. Cronbach's alpha and Intra-class Correlation Coefficient for retained constructs in the model at individual and environmental level.",
           "Fig 2. Specification of a SEM for predicting dietary behavior, including mediation effects. Rectangles indicate observed variables, and ellipses latent variables. HE: Healthy Eating.",
           "Fig 3. (A) Individual and environmental influences on sugary drink consumption. Only statistically significant paths at P < 0.05 are shown. Goodness of Fit-statistics: X^2(303) = 630.18, p< 0.001; RMSEA = 0.038; NNFI = 0.92; CFI = 0.93, GFI = 0.93. HE: Healthy Eating. (B) Individual and environmental influences on breakfast consumption. Only statistically significant paths at P < 0.05 are shown. Goodness of Fit-statistics: X^2(303) = 630.50, p< 0.001; RMSEA = 0.038; NNFI = 0.92; CFI = 0.93, GFI = 0.94. HE: Healthy Eating. (C) Individual and environmental influences on fruit, vegetable and fruit and vegetable consumption. Only statistically significant paths at P < 0.05 are shown. Goodness of Fit-statistics: X^2(303) = 623.69, p< 0.001; RMSEA = 0.038; NNFI = 0.92; CFI = 0.93, GFI = 0.94. a significant pathway for vegetable intake only. HE: Healthy Eating(D) Individual and environmental influences on unhealthy snacking. Only statistically significant paths at P < 0.05 are shown. Goodness of Fit-statistics: X^2(303) = 619.19, p< 0.001; RMSEA = 0.037; NNFI = 0.92; CFI = 0.93, GFI = 0.94. HE: Healthy Eating."]

# http://journals.plos.org/plosone/article?id=10.1371%2Fjournal.pone.0159387
# fig 3 ** 3,4,2,1,5
PLOS_31 = ["The genome of a laboratory-adapted strain of Leptospira interrogans serovar Hardjo was sequenced and analyzed. Comparison of the sequenced genome with that recently published for a field isolate of the same serovar revealed relatively high sequence conservation at the nucleotide level, despite the different biological background of both samples. Conversely, comparison of both serovar Hardjo genomes with those of L. borgpetersenii serovar Hardjo showed extensive differences between the corresponding chromosomes, except for the region occupied by their rfb loci. Additionally, comparison of the serovar Hardjo genomes with those of different L. interrogans serovars allowed us to detect several genomic features that may confer an adaptive advantage to L. interrogans serovar Hardjo, including a possible integrated plasmid and an additional copy of a cluster encoding a membrane transport system known to be involved in drug resistance. A phylogenomic strategy was used to better understand the evolutionary position of the Hardjo serovar among L. interrogans serovars and other Leptospira species. The proposed phylogeny supports the hypothesis that the presence of similar rfb loci in two different species may be the result of a lateral gene transfer event.",
           "Table 1. Genomes of Leptospira species, strains and serovars selected for the functional and phylogenomic studies presented in this article.",
           "Table 2. Basic statistics of the L. interrogans serovar Hardjo genome sequenced in this study, compared with those of L. interrogans serovar Lai (strain 56601), L. interrogans serovar Copenhageni (strain Fiocruz L1-130) and L. interrogans serovar Hardjo (strain Norma).",
           "Fig 1. Comparison of the two L. interrogans serovar Hardjo genomes included in this study with those of L. interrogans serovar Lai (strain 56601) and L. borgpetersenii serovar Hardjo (strain L550). Red bands indicate similar regions and blue bands indicate inversions. Sequences corresponding to the rfb loci typical of the Hardjo serovar are highlighted in yellow. Three regions present in the L. interrogans serovar Hardjo genome but absent from that of the Lai serovar are indicated by asterisks.",
           "Fig 2. Maximum likelihood tree based on concatenated protein sequences from 23 Leptospira genomes. The saprophytic non-pathogenic L. biflexa was used to root the tree. Branches highlighted in red are those leading to taxa whose genomes contain the additional cluster of RND transporter components described for serovar Hardjo (see main text). As there was not enough phylogenetic signal to separate individual L. interrogans serovars, those having this cluster are shaded in pink. Bootstrap values are shown for branches separating different species.",
           "Fig 3. Unrooted tree of selected L. interrogans serovars. This maximum likelihood tree was built following the same methodology described for the tree in Fig 2, but considering only the L. interrogans serovars. Branches highlighted in red are those corresponding to serovars whose genomes contain the additional cluster of RND transporter components. Bootstrap values are shown for branches separating different serovars."]

# http://journals.plos.org/plosone/article?id=10.1371%2Fjournal.pone.0159344
# fig 8  8,1,2,5,3,4,6,7
PLOS_32 = ["Cell cycle checkpoint is mediated by ATR and ATM kinases, as a prompt early response to a variety of DNA insults, and culminates in a highly orchestrated signal transduction cascade. Previously, we defined the regulatory role of nucleotide excision repair (NER) factors, DDB2 and XPC, in checkpoint and ATR/ATM-dependent repair pathway via ATR and ATM phosphorylation and recruitment to ultraviolet radiation (UVR)-induced damage sites. Here, we have dissected the molecular mechanisms of DDB2- and XPC- mediated regulation of ATR and ATM recruitment and activation upon UVR exposures. We show that the ATR and ATM activation and accumulation to UVR-induced damage not only depends on DDB2 and XPC, but also on the NER protein XPA, suggesting that the assembly of an active NER complex is essential for ATR and ATM recruitment. ATR and ATM localization and H2AX phosphorylation at the lesion sites occur as early as ten minutes in asynchronous as well as G1 arrested cells, showing that repair and checkpoint-mediated by ATR and ATM starts early upon UV irradiation. Moreover, our results demonstrated that ATR and ATM recruitment and H2AX phosphorylation are dependent on NER proteins in G1 phase, but not in S phase. We reasoned that in G1 the UVR-induced ssDNA gaps or processed ssDNA, and the bound NER complex promote ATR and ATM recruitment. In S phase, when the UV lesions result in stalled replication forks with long single-stranded DNA, ATR and ATM recruitment to these sites is regulated by different sets of proteins. Taken together, these results provide evidence that UVR-induced ATR and ATM recruitment and activation differ in G1 and S phases due to the existence of distinct types of DNA lesions, which promote assembly of different proteins involved in the process of DNA repair and checkpoint activation.",
           "Fig 1. ATR and ATM recruitment and rH2AX phosphorylation at UVR damage sites in asynchronous OSU-2 cells. OSU-2 cells were exposed to 100 J/m2 UVR through a 5 uM micropore filter and allowed to respond to the induced damage for short indicated periods. The cells were fixed and immunofluorescence was performed to decorate ATR, pATM, rH2AX, and XPC proteins with cognate antibodies. Images show representative ATR, pATM and rH2AX foci co-localizing at DNA damage sites that were marked by distinct XPC labeling. (A) ATR (B) pATM (C) rH2AX (D) rH2AX foci formation in XP-E, XP-C, and XP-A cells upon 20 min post-repair.",
           "Fig 2. ATR, ATM and their substrate phosphorylations are affected in cells defective in XPA function. A. ATR and ATM and their substrate phosphorylation are affected in XP-A cells. Normal OSU-2 and XP-A cells were exposed to 20 J/m2 UVR and allowed to repair for 1 h. Their protein extracts were used to determine the phosphorylation of ATR, ATM, Chk1, Chk2, and H2AX alongside their total protein levels. Western blots were performed as described in [45]. (B) ATR and ATM recruitment and H2AX phosphorylation at the damage site is affected in XP-A cells. OSU-2 and XP-A cells were exposed to 100 J/m2 UVR and 1 h after cellular UVR exposures, cells were processed for immunofluorescent detection of damage-specific ATR, pATM, and rH2AX. Images show representative cells for individual treatment. The top panel shows the co-localization of ATR with the damage marker rH2AX, and the bottom panel shows the co-localization of pATM with rH2AX. (C) The quantitative data of ATR, pATM, and rH2AX foci formation: Quantitation was done by using three independent experiments, and at least 100 random samples per experiment were scored in each group. All values were expressed as mean + SD. The difference between groups was tested using two-sided t-tests. The p-values <0.05 are considered significant.",
           "Fig 3. ATR and ATM recruitment and H2AX phosphorylation at UVR damage site in G1 cells. A. OSU-2 cells were arrested by in G1 phase of cell cycle by serum starvation for 48 h and the cell cycle arrest was examined by FACS analysis. B, C, &D. Experiments were done as described in Fig 1 using the G1 arrested cells.",
           "Fig 4. ATR and ATM and their substrate phosphorylation are affected in G1 arrested cells. OSU-2, XP-E, XP-C and XP-A cells were arrested in G1 phase of cell cycle as described in A, exposed to 20 J/m2 UV irradiation, and after 1 h post-repair time, extracts were isolated and processed by Western blotting as described in [45].",
           "Fig 5. ATR, ATM recruitment and H2AX phosphorylation is affected in XP cells during G1. (A) Colocalization of ATR with DNA damage marker rH2AX in G1 arrested cells. OSU-2, XP-E, XP-C and XP-A cells were arrested in G1 phase of cell cycle by serum starvation for 48 h and exposed to 100 J/m2 UV irradiation through a 5 um micropore filter. At 1 h post-exposure immunofluorescence was performed to determine the extent of co-localization. The quantitative data of ATR and rH2AX foci formation were determined as described in Fig 2B (B) Colocalization of pATM with rH2AX in G1 arrested cells. Experiments were done as described in A.",
           "Fig 6. ATM recruitment and H2AX phosphorylation at damage sites is affected in XP cells during G1. (A) rH2AX foci formation at the CPD damage sites. OSU-2, XP-E, XP-C and XP-A cells were arrested in G1 by serum starvation for 48 h and exposed to 100 J/m2 UV irradiation using a 5 um micropore filter. After 1 h post-exposure time immunofluorescence was performed as in Fig 5. The quantitative data of rH2AX and pATM foci formation were determined as described in Fig 2B. (B) pATM foci formation at the CPD damage sites. Experiments were done as described in A.",
           "Fig 7. ATM recruitment and H2AX phosphorylation at damage sites is not affected in XP cells during S. (A) rH2AX foci formation in EdU positive cells. Asynchronously growing OSU-2, XP-E, XP-C and XP-A cells were exposed to 100 J/m2 UVR through a 5 um micropore filter. S phase cells were differentiated from non-S phase cells by nuclear EdU labeling as described in methods. The H2AX was detected by immunofluorescence as described in Fig 1. The quantitative data of rH2AX and pATM foci formation in S phase EdU-positive cells were determined as described in Fig 2B. (B) pATM foci formation in EdU positive cells. Experiments were done as described in A.",
           "Fig 8. Schematic model of pre-incision and pre-checkpoint UVR response events. Only the predominant pathways involved in post-UVR regulation of ATR and ATM recruitment during G1 and S phase of cell cycle are considered. G1 phase cells possess UVR-induced lesions which primarily recruit core components of NER pathway. Formation of the NER complex in turn facilitates the recruitment of ATR and ATM to initiate repair and in parallel the downstream events leading to G1 checkpoint. In S phase cells, replication fork encounters the UVR-induced lesions which result in the formation of ssDNA breaks and recruitment of ATR to processed single strand ends. Replication fork collapse results in DSBs where ATM is recruited. Both of these SSBs and DSBs lead to the initiation of S phase replication checkpoint and are repaired by HR pathway."]

# http://journals.plos.org/plosone/article?id=10.1371%2Fjournal.pone.0159369
# fi3 ** 2,3,1
PLOS_33 = ["Reporter genes are powerful technologies that can be used to directly inform on the fate of transplanted cells in living subjects. Imaging reporter genes are often employed to quantify cell number, location(s), and viability with various imaging modalities. To complement this, reporters that are secreted from cells can provide a low-cost, in vitro diagnostic test to monitor overall cell viability at relatively high frequency without knowing the locations of all cells. Whereas protein-based secretable reporters have been developed, an RNA-based reporter detectable with amplification inherent PCR-based assays has not been previously described. MicroRNAs (miRNAs) are short non-coding RNAs (18-22 nt) that regulate mRNA translation and are being explored as relatively stable blood-based disease biomarkers. We developed an artificial miRNA-based secreted reporter, called Sec-miR, utilizing a coding sequence that is not expressed endogenously and does not have any known vertebrate target. Sec-miR was detectable in both the cells and culture media of transiently transfected cells. Cells stably expressing Sec-miR also reliably secreted it into the culture media. Mice implanted with parental HeLa cells or HeLa cells expressing both Sec-miR and the bioluminescence imaging (BLI) reporter gene Firefly luciferase (FLuc) were monitored over time for tumor volume, FLuc signal via BLI, and blood levels of Sec-miR. Significantly (p<0.05) higher Sec-miR was found in the blood of mice bearing Sec-miR-expressing tumors compared to parental cell tumors at 21 and 28 days after implantation. Importantly, blood Sec-miR reporter levels after day 21 showed a trend towards correlation with tumor volume (R2 = 0.6090; p = 0.0671) and significantly correlated with FLuc signal (R2 = 0.7067; p<0.05). Finally, we could significantly (p<0.01) amplify Sec-miR secretion into the cell media by chaining together multiple Sec-miR copies (4 instead of 1 or 2) within an expression cassette. Overall, we show that a novel complement of BLI together with a unique Sec-miR reporter adds an in vitro RNA-based diagnostic to enhance the monitoring of transplanted cells. While Sec-miR was not as sensitive as BLI for monitoring cell number, it may be more sensitive than clinically-relevant positron emission tomography (PET) reporter assays. Future work will focus on improving cell detectability via improved secretion of Sec-miR reporters from cells and more sensitive detection platforms, as well as, exploring other miRNA sequences to allow multiplexed monitoring of more than one cell population at a time. Continued development may lead to more refined and precise monitoring of cell-based therapies.",
           "Fig 1. An artificial secreted miRNA reporter (Sec-miR) is detectable in the media of transiently transfected cultured cells. A) Vector map of pcDNA 6.2-GW/EmGFP-miR-neg. The pre-miR (miR-neg), necessary for mature Sec-miR generation, is encoded within the 3' untranslated regions (3'-UTR) of the pCMV-EmGFP expression cassette. Spec-R-Spectinomycin resistance; BSD-R-blasticidin resistance; pUC origin-origin of replication. B) Fluorescence/phase images of HeLa cells 24 hours after transfection with pcDNA 6.2-GW/EmGFP-miR-neg (top) or mock-transfected (bottom). C) Sec-miR levels measured in HeLa cells (top) or media (bottom) transfected with pcDNA 6.2-GW/EmGFP-miR-neg (or mock transfected) 72 hours after transfection. Significantly higher Sec-miR levels were detected in transfected cells compared to mock cells in both the cell pellets (p<0.001) and the media (p<0.001). Sec-miR data from transfected cells is normalized to data from mock transfected cells and expressed as mean + SD. D) Vector map of pcDNA 6.2-GW/ pEF1-Luc2-Sec-miR-miR-neg. miR-neg is encoded within the 3' untranslated regions (3'-UTR) of the pEF1-Luc2 expression cassette. E) Significantly (p<0.001) higher Sec-miR levels are measured in cells and media of cells stably expressing Sec-miR/FLuc versus parental HeLa cells. Sec-miR data from Sec-miR/FLuc cells is normalized to data from parental cells and data is expressed as mean + SD.",
           "Fig 2. Sec-miR is detectable in the serum of animals bearing Sec-miR-expressing cells and can be used as a complement to imaging reporters. A) Tumor volume measurements over time of mice implanted with parental (n = 5) or Sec-miR/FLuc (n = 5) HeLa cells. Data is expressed as mean + SD. No differences in tumor volume are seen between the two cell lines. B) Bioluminescence imaging (BLI) of a representative Sec-miR/FLuc HeLa mouse over time. Data is expressed as mean + SD. C) Quantitative ROI tumor analysis of BLI images was performed showing the increase in average radiance over time. D) Correlation of tumor volume and BLI measures. E) Sec-miR levels in the serum over time in mice bearing parental or Sec-miR/FLuc-expressing HeLa tumors. Significantly (p<0.05) higher Sec-miR levels are detected on days 21 and 28 in Sec-miR/FLuc mice versus controls. Data from Sec-miR/FLuc HeLa mice are normalized to data from parental HeLa mice and expressed as mean + SD. Correlation of Sec-miR levels (1/delta-Ct) and tumor volume (F) and FLuc BLI signal (G) for Sec-miR/FLuc HeLa mice.",
           "Fig 3. Expression of multiple Sec-miR copies within a construct can amplify reporter secretion from cells. A) Vector maps of Sec-miR-expressing constructs containing 1, 2, or 4 copies of miR-neg (A to C). B) Sec-miR levels in cell culture media in HeLa cells (left to right) after transient transfection with the vectors shown in A-C and a Renilla luciferase (RLuc) expressing plasmid to control for transfection efficiency. All raw qPCR data was normalized to Renilla luciferase (RLuc) activity and then across groups to values in cells transfected with the 1 miR-neg copy vector. Data is expressed as mean + SD. All constructs resulted in detectable levels of Sec-miR in the media and significantly higher levels were detectable for the construct containing 4 copies of miR-neg versus the other constructs (p<0.01)."]

# http://journals.plos.org/plosone/article?id=10.1371%2Fjournal.pone.0159336
# fig 7 ** 3,1,2,5,8,7,4,9,6
PLOS_34 = ["Pigeons have shown suboptimal gambling-like behavior when preferring a stimulus that infrequently signals reliable reinforcement over alternatives that provide greater reinforcement overall. As a mechanism for this behavior, recent research proposed that the stimulus value of alternatives with more reliable signals for reinforcement will be preferred relatively independently of their frequencies. The present study tested this hypothesis using a simplified design of a Discriminative alternative that, 50% of the time, led to either a signal for 100% reinforcement or a blackout period indicative of 0% reinforcement against a Nondiscriminative alternative that always led to a signal that predicted 50% reinforcement. Pigeons showed a strong preference for the Discriminative alternative that remained despite reducing the frequency of the signal for reinforcement in subsequent phases to 25% and then 12.5%. In Experiment 2, using the original design of Experiment 1, the stimulus following choice of the Nondiscriminative alternative was increased to 75% and then to 100%. Results showed that preference for the Discriminative alternative decreased only when the signals for reinforcement for the two alternatives predicted the same probability of reinforcement. The ability of several models to predict this behavior are discussed, but the terminal link stimulus value offers the most parsimonious account of this suboptimal behavior.",
           "Fig 1. Design of the 4-stimulus suboptimal choice experiment using a spatial discrimination between two white keys. The left alternative shows possible outcomes for choosing the Discriminative alternative while the right alternative shows the Nondiscriminative outcome.",
           "Fig 2. Design of the 3-stimulus suboptimal choice experiment using a spatial discrimination between two white keys. The left alternative shows possible outcomes for choosing the Discriminative alternative while the right alternative shows the Nondiscriminative outcome.",
           "Fig 3. Design of the 2-stimulus suboptimal choice experiment using a visual discrimination. The left alternative shows possible outcomes for choosing the Discriminative alternative while the right alternative shows the Nondiscriminative outcome.",
           "Fig 4. Experiment 1: Mean percentage choice (+ SEM) of the Discriminative alternative as a function of session for Phases 1-3. Vertical lines indicate phase changes, the horizontal line indicates indifference between the two altrnatives (50%), and the solid horizontal line indicates sessions where the left response key intermittently malfunctioned.",
           "Table 1. Average food reinforcers earned from each alternative across the last five sessions of Phases 1-3 in Experiment 1.",
           "Fig 5. Experiment 1: Mean logged latencies (+ SEM) to choose (left panel) and terminal link responses (right panel) for Phases 1-3.",
           "Fig 6. Experiment 2: Mean percentage choice (+ SEM) of the Discriminative alternative as a function of session for Phases 1-3. Vertical lines indicate phase changes while the horizontal line indicates chance (50%).",
           "Table 2. Average food reinforcers earned from each altenrative across the last five sessions Phases 1-3 in Experiment 2.",
           "Fig 7. Experiment 2: Mean logged latencies (+ SEM) to choose (left panel) and terminal link responses (right panel) to the terminal link stimuli for Phases 1-3."
            ]

# http://journals.plos.org/plosone/article?id=10.1371%2Fjournal.pone.0159302
#fig7 **7,4,5,6,8,2,9,1,3
PLOS_35 = ["In this paper we apply machine learning methods for predicting protein interactions in fungal secretion pathways. We assume an inter-species transfer setting, where training data is obtained from a single species and the objective is to predict protein interactions in other, related species. In our methodology, we combine several state of the art machine learning approaches, namely, multiple kernel learning (MkL), pairwise kernels and kernelized structured output prediction in the supervised graph inference framework. For MkL, we apply recently proposed centered kernel alignment and p-norm path following approaches to integrate several feature sets describing the proteins, demonstrating improved performance. For graph inference, we apply input-output kernel regression (IOkR) in supervised and semi-supervised modes as well as output kernel trees (Ok3). In our experiments simulating increasing genetic distance, Input-Output kernel Regression proved to be the most robust prediction approach. We also show that the MkL approaches improve the predictions compared to uniform combination of the kernels. We evaluate the methods on the task of predicting protein-protein-interactions in the secretion pathways in fungi, S.cerevisiae, baker's yeast, being the source, T. reesei being the target of the inter-species transfer learning. We identify completely novel candidate secretion proteins conserved in filamentous fungi. These proteins could contribute to their unique secretion capabilities.",
           "Fig 1. Frequency distribution of percentage of amino acid sequence identity between natural S. cerevisiae sequences and (1) sets of artificial sequences created from from S. cerevisiae with different Blosum matrices, (2) natural T. reesei sequences.",
           "Fig 2. Schematic representation of the duality between (A) the PPI network and (B) the adjacency matrix for the proteins in the training set (blue) and testing set (yellow) and their interactions: training interactions (black), training-testing interactions (gray) and testing interactions (white).",
           "Table 1. Confusion matrix indicating True positive (TP), False positive (FP), False negative (FN) and True negative predictions.",
           "Fig 3. ROC curves for predicting PPIs in different artificial data sets with Output kernel trees (Ok3), Tensor kernels on protein pairs (Tensor kernel on PP), and supervised and semi-supervised Input-Output kernel Regression (IOkR). AUROC statistic of the associated curve is depicted in the figure legend (standard deviation in parenthesis).",
           "Fig 4. Precision-Recall (PR) curves for predicting PPIs in different artificial data sets with Output kernel trees (Ok3), Tensor kernels on protein pairs (Tensor kernel on PP), and supervised and semisupervised Input-Output kernel Regression (IOkR). AUPR statistic is shown in the legend for each curve (standard devation in parenthesis.",
           "Fig 5. ROC curves and Precision-Recall (PR) curves for predicting secretory PPIs from the full S. cerevisiae genome with Output kernel trees (Ok3), Tensor kernels on protein pairs (Tensor kernel on PP), and supervised and semi-supervised Input-Output kernel Regression (IOkR). AUCROC and AUPR statistics are shown in the legend for each curve.",
           "Fig 6. ROC curves and Precision-Recall (PR) curves for predicting secretory PPIs from the full S. cerevisiae genome with semi-supervised Input-Output kernel Regression (IOkR) and different Multiple kernel Learning (MkL) methods compared to no MkL (UNIMkL). AUCROC and AUPR statistics are shown in the legend for each curve.",
           "Fig 7. Predicted T. reesei secretion network. A) The proteins annotated as secretory (242) and unknown (14) are included. Proteins are nodes and they are labelled with best matching S. cerevisiae protein name or if no match was found with T. reesei gene ID number. Thick edges signify either negative (red) or positive (green) absolute Pearson correlation of > 0.3 in transcriptomic data. Pink nodes do not have any interactions in STRING. B) Pie chart of functional classes of the 320 proteins included in the T. reesei secretion network.",
           "Table 2. Unknown genes and genes without any interactions in STRING in predicted T. reesei secretion network. Column 'Gene' contains the T. reesei gene ID. 'In STRING' tells if the gene has interactions in STRING. Columns 'Btw' and 'Deg' denote the betweenness and degree network statistics of the corresponding gene. Columns 'Class' and 'Putative secretion pathway component' are author assigned classifications. 'Taxon specificity' gives the largest taxonomic group the gene was found in."]

# http://journals.plos.org/plosone/article?id=10.1371%2Fjournal.pone.0159195
# fig 3 ** 1*,4,3,5,6,2
PLOS_36 = ["High-throughput sequencing (HTS) techniques have suggested the existence of a wealth of species with very low relative abundance: the rare biosphere. We attempted to exhaustively map this rare biosphere in two water samples by performing an exceptionally deep pyrosequencing analysis (~500,000 final reads per sample). Species data were derived by a 97% identity criterion and various parametric distributions were fitted to the observed counts. Using the best-fitting Sichel distribution we estimate a total species richness of 1,568-1,669 (95% Credible Interval) and 5,027-5,196 for surface and deep water samples respectively, implying that 84-89% of the total richness in those two samples was sequenced, and we predict that a quadrupling of the present sequencing effort would suffice to observe 90% of the total richness in both samples. Comparing the HTS results with a culturing approach we found that most of the cultured taxa were not obtained by HTS, despite the high sequencing effort. Culturing therefore remains a useful tool for uncovering marine bacterial diversity, in addition to its other uses for studying the ecology of marine bacteria.",
           "Table 1. Summary of location and depth (m) of samples, total sequences before cleaning (Raw Reads) and after cleaning (Final Reads), observed richness (Sobs) computed as the total number of Operational Taxonomic Units (OTUs) clustered at 97% identity, and the percentage of singletons. Total richness (S) was estimated using the Chao1 lower bound estimator [39] and using the Sichel distribution fitted to the count frequency data by the Bayesian method of [33] and selected from four alternative candidate models using the Deviance Information Criterion (DIC). Using the Sichel distribution, point estimates and 95% credible intervals (CIs) for S were obtained from the mean and (2.5%, 97.5%) quantiles of the posterior distribution sampled 150000 times by Markov Chain Monte Carlo (after a burn-in period of 100000 samples, see 33). The Required Sequencing Effort (RSE) to sequence 90% of the total richness was predicted by hierarchical simulation (see Materials and methods) and is quoted in terms of the number of final reads and as a multiple of the present sequencing effort. Point estimates and 95% prediction intervals (PIs) for RSE were calculated as the mean and (2.5%, 97.5%) quantiles from a set of 80 simulations using the Sichel distribution.",
           "Fig 1. OTU collector's curves of the surface (orange line) and bottom (green line) samples. Black dashed lines indicate the 95% confidence intervals (95% CI).",
           "Fig 2. Rank-abundance plots of surface (A) and bottom (B) samples. The red line is the rank-abundance plot calculated with the actual data. The dark blue line shows the estimates of the sequencing effort necessary to retrieve 90% of the total richness calculated by simulation from the best-approximating Sichel distribution (posterior mean estimate). The vertical black line separates the real data (left) from the estimates (right). In (A) the percentage of cultured isolates found in the 454-pyrosequencing dataset is indicated at the left side of the black vertical line. The percentage of cultured isolates not found in the 454-pyrosequencing dataset, and that would presumably be found by increasing the sequencing effort, is indicated at the right of the black vertical line. Insert pictures show some of the bacterial cultures grown from the surface sample. Font size and pictures are scaled according to the percentage of cultured isolates found or not found in the 454-pyrosequencing dataset.",
           "Fig 3. Rank-abundance plot of the 38 isolated bacterial species. The orange triangles indicate the cultured isolates found in the 454-pyrosequencing (HTS) dataset and the white circles indicate the cultures that were not found in the HTS dataset. The isolated bacterial species are listed in Tables 2 and 3.",
           "Table 2. Cultured isolates with matching HTS sequences. Columns show the isolates' closest relatives according to the BLAST results, the percentage of identity with the BLAST reference strain (identity BLAST), the GenBank accession number of the BLAST reference strain, the number of HTS reads matching the isolate sequences in the surface sample (Reads in Surface), the percentage of the total HTS reads in the surface sample represented by the isolate sequences (% Surface), and the number of isolates of each taxa sequenced. Abbreviations are: Actino (Actinobacteria), Bact (Bacteroidetes), Alpha-P (Alpha-Proteobacetria) and Gamma-P (Gamma-Proteobacteria).",
           "Table 3. Cultured isolates without matching HTS sequences. Columns show the isolates' closest relatives according to the BLAST results, the % of identity with the BLAST reference strain (identity BLAST), the GenBank accession number of the BLAST reference strain and the number of isolates of each taxa sequenced. Abbreviations are: Actino (Actinobacteria), Bact (Bacteroidetes), Firm (Firmicutes), Alpha-P (Alpha-Proteobacetria) and Gamma-P (Gamma-Proteobacteria)."]

# http://journals.plos.org/plosone/article?id=10.1371%2Fjournal.pone.0159188
# fig 10 ** 1,8,4,10,11,3,9, 5,2,6,7
PLOS_37 = ["Researches on sequential vocalization often require analysis of vocalizations in long continuous sounds. In such studies as developmental ones or studies across generations in which days or months of vocalizations must be analyzed, methods for automatic recognition would be strongly desired. Although methods for automatic speech recognition for application purposes have been intensively studied, blindly applying them for biological purposes may not be an optimal solution. This is because, unlike human speech recognition, analysis of sequential vocalizations often requires accurate extraction of timing information. In the present study we propose automated systems suitable for recognizing birdsong, one of the most intensively investigated sequential vocalizations, focusing on the three properties of the birdsong. First, a song is a sequence of vocal elements, called notes, which can be grouped into categories. Second, temporal structure of birdsong is precisely controlled, meaning that temporal information is important in song analysis. Finally, notes are produced according to certain probabilistic rules, which may facilitate the accurate song recognition. We divided the procedure of song recognition into three sub-steps: local classification, boundary detection, and global sequencing, each of which corresponds to each of the three properties of birdsong. We compared the performances of several different ways to arrange these three steps. As results, we demonstrated a hybrid model of a deep convolutional neural network and a hidden Markov model was effective. We propose suitable arrangements of methods according to whether accurate boundary detection is needed. Also we designed the new measure to jointly evaluate the accuracy of note classification and boundary detection. Our methods should be applicable, with small modification and tuning, to the songs in other species that hold the three properties of the sequential vocalization.",
           "Fig 1. Three arrangements of methods for birdsong recognition. Flow diagrams for the three arrangements compared in this study. (a) BD - LC - GS arrangement. The colored letters A, B, and C indicate the note classes, and the white regions indicate the detected inter-note silent intervals. (b) LC - BD & GS arrangement. The white letter S indicates the silent intervals. (c) LC & GS - BD & GS arrangement.",
           "Fig 2. Note & timing error rate. An example of an input, a true sequence, output sequences, and correctly recognized intervals that would be obtained in computing note & timing error rate. (a) An example spectrogram and the true labels and boundaries. Note classes are indicated by letters. (b) An example recognition output (upper), and the correctly recognized intervals (lower). In the correctly recognized intervals, colored bars indicate the correctly recognized note intervals, longest overlaps with the true label intervals. Gray bars indicate the correctly recognized silent intervals. The correctly recognized intervals appear to capture the performances properly even in such cases that a single note is recognized as two (notes B) and two notes are recognized as one (notes C). In both cases either of two overlapping intervals (the one with a longer overlap) was counted as correctly recognized intervals. There could be a case in which no matched interval is assigned (note A). (c) Another example showing the recognition outputs and the correctly recognized intervals with lower note & timing error rate.",
           "Fig 3. Boundary detection by thresholding. In the BD - LC - GS arrangement, note boundaries were detected using sound amplitude and interval duration. From top to bottom: input spectrogram, amplitude envelope (black line) and threshold (orange line), non-background intervals (orange bars), detected note intervals (blue bars).",
           "Fig 4. Local classification with a deep convolutional neural network. Local classification was conducted with a deep convolutional neural network. Input, intermediate, and output layers were shown in gray rectangles. The range of the input used for computing a single time point in the following layer is shown in colored lines. (a) BD - LC - GS arrangement. (b) LC - BD & GS arrangement. (c) LC & GS - BD & GS arrangement.",
           "Fig 5. Syntax models used in HMMs. Schematic diagrams of song syntax modeled with a second-order Markov model. In this figure examples with two note classes (A & B) are shown. (a) A transition diagram in the BD - LC - GS arrangement. The initial state is indicated by the letter 'e'. The transition probabilities of orange arrows were computed from the training data sets. Those of black arrows were uniformly distributed (ie. all transition probabilities from states 'e', 'A', and 'B' are 0.5). Sequence generation is allowed to stop at any states. (b) In the LC - BD & GS and the LC & GS - BD & GS arrangements, each state in (a) except the initial state was divided into four. The letter 'X' and 'Y' denote any note classes or the initial state.",
           "Table 1. Average note ERs, timing ERs, and note and timing ERs.",
           "Fig 6. Validation errors. (a) Note ERs of the results trained on two and eight minutes of training data sets. ERs in each bird are shown in open circles. (b) Timing ERs. (c) Note & timing ERs. ***: p < 0.001.",
           "Fig 7. Recognition results in the BD - LC - GS arrangement. (a) A recognition result in one bird. From upper to lower: an input spectrogram, amplitude, outputs of local classification, recognized note intervals, true note intervals, and correctly recognized intervals. Rows in the classification outputs correspond to the note classes. The black areas are putative silent intervals detected in the boundary detection step. Gray rectangles with letters indicate note intervals and classes. The correctly recognized intervals are indicated by black bars. (b) A result in another bird with poorer recognition accuracy.",
           "Fig 8. Sub-divisions in notes. (a) An example input spectrogram (upper) and the true note intervals (lower). Gray rectangles with letters indicate note intervals with note classes. (b) Example outputs of a DCNN without sub-division in notes (upper) and the recognized sequence (lower). First three rows in the DCNN outputs correspond to three note classes and the last to the class for silence. (c) Example outputs with notes divided into two parts. First six rows in the DCNN outputs correspond to three note classes with two sub-divisions. The last row corresponds to the silence. (d) Example outputs with notes divided into three parts. First nine rows in the DCNN outputs correspond to the three note classes with three sub-divisions.",
           "Fig 9. Recognition results in the LC - BD & GS arrangement. (a) A recognition result in one bird. From upper to lower: an input spectrogram, outputs of local classification, recognized note intervals, true note intervals, and correctly recognized intervals. Rows in the classification outputs correspond to twelve note classes with three sub-divisions. The bottom row indicates the class for the background noise. Gray rectangles with letters indicate note intervals and classes. The correctly recognized intervals are indicated by black bars. (b) A result in another bird.",
           "Fig 10. Recognition results in the LC & GS - BD & GS arrangement. (a) A recognition result in one bird. From upper to lower: an input spectrogram, outputs of local classification, recognized note intervals, true note intervals, and correctly recognized intervals. Rows in the classification outputs correspond to twelve note classes with three sub-divisions. The bottom row indicates the class for the background noise. Gray rectangles with letters indicate note intervals and classes. The correctly recognized intervals are indicated by black bars. (b) A result in another bird."]

# http://journals.plos.org/plosone/article?id=10.1371%2Fjournal.pone.0158962
# fig3 4,2,1,3
PLOS_38 = ["Rugose projections on the anterior and posterior aspects of vertebral neural spines appear throughout Amniota and result from the mineralization of the supraspinous and interspinous ligaments via metaplasia, the process of permanent tissue-type transformation. In mammals, this metaplasia is generally pathological or stress induced, but is a normal part of development in some clades of birds. Such structures, though phylogenetically sporadic, appear throughout the fossil record of non-avian theropod dinosaurs, yet their physiological and adaptive significance has remained unexamined. Here we show novel histologic and phylogenetic evidence that neural spine projections were a physiological response to biomechanical stress in large-bodied theropod species. Metaplastic projections also appear to vary between immature and mature individuals of the same species, with immature animals either lacking them or exhibiting smaller projections, supporting the hypothesis that these structures develop through ontogeny as a result of increasing bending stress subjected to the spinal column. Metaplastic mineralization of spinal ligaments would likely affect the flexibility of the spinal column, increasing passive support for body weight. A stiff spinal column would also provide biomechanical support for the primary hip flexors and, therefore, may have played a role in locomotor efficiency and mobility in large-bodied species. This new association of interspinal ligament metaplasia in Theropoda with large body size contributes additional insight to our understanding of the diverse biomechanical coping mechanisms developed throughout Dinosauria, and stresses the significance of phylogenetic methods when testing for biological trends, evolutionary or not.",
           "Fig 1. Dorsal vertebrae showing evidence of metaplastic rugosities on the neural spine. (A) Dorsal vertebra from a Troodon (MOR 553-8.20.92.305), a small-bodied theropod. (B) A dorsal vertebra from an Allosaurus (MOR 693), a large bodied theropod. Note the expanded metaplastic rugosities in Allosaurus (highlighted by 50% transparency) compared with Troodon. of the mature individual compared with the smaller rugosities of the immature animal. Both dorsal vertebrae in left lateral view. Scale bars = 10 cm.",
           "Table 1. Femur length and the presence or absence of rugose projections that extend from the anterior and posterior blades of the neural spines in the dorsal region.",
           "Fig 2. Histological section of the interspinous ligament enthesis in a dorsal vertebra in Alligator and Tyrannosaurus. (A) The enthesis in Alligator (H&E staining, 40x) exhibiting an undulating interface surface between the lighter ligament (marked 'En.' on the left of the panel) and bone of the neural spine (marked 'N.S.'). (B) The enthesis in Tyrannosaurus (100x) shows rough-bundled metaplastic tissue in the area of the enthesis, which deeply interdigitates with the neural spine.",
           "Fig 3. (A) Phylogenetic tree onto which femur length (Ln cm) is mapped using maximum likelihood. Species with metaplastic rugosities on neural spines are bolded. Note that Spinosaurus is large-bodied, but lacks rugose neural spines, most likely owing to its elongate neural spines. (B) Distribution of femur length (Ln cm) in theropods grouped by the absence (top quartile plot) or presence (bottom quartile plot) of rugose neural spines. Quantiles are shown in gray boxes with the minimum, 25%, median, 75%, and maximum values shown for each group. The color code of data matches the gradient from the phylogenetic mapping in panel A, which correspond to natural log femur length with red representing the longest and purple representing the shortest. A phylogenetic t-test supports a substantial difference in average body size between species that exhibit or lack neural spine rugosities (n = 56, p-value < 1.0e-10)."]


# http://journals.plos.org/plosone/article?id=10.1371%2Fjournal.pone.0158865
# fig8 8,5,7,1,4,2,6,3
PLOS_39 = ["Protein kinase C (PkC) plays a significant role in thrombin-induced loss of endothelial cell (EC) barrier integrity; however, the existence of more than 10 isozymes of PkC and tissue-specific isoform expression has limited our understanding of this important second messenger in vascular homeostasis. In this study, we show that PkCD isoform promotes thrombin-induced loss of human pulmonary artery EC barrier integrity, findings substantiated by PkCD inhibitory studies (rottlerin), dominant negative PkCD construct and PkCD silencing (siRNA). In addition, we identified PkCD as a signaling mediator upstream of both thrombin-induced MLC phosphorylation and Rho GTPase activation affecting stress fiber formation, cell contraction and loss of EC barrier integrity. Our inhibitor-based studies indicate that thrombin-induced PkCD activation exerts a positive feedback on Rho GTPase activation and contributes to Rac1 GTPase inhibition. Moreover, PkD (or PkCu) and CPI-17, two known PkCD targets, were found to be activated by PkCD in EC and served as modulators of cytoskeleton rearrangement. These studies clarify the role of PkCD in EC cytoskeleton regulation, and highlight PkCD as a therapeutic target in inflammatory lung disorders, characterized by the loss of barrier integrity, such as acute lung injury and sepsis.",
           "Fig 1. The role of PkCD in thrombin-induced endothelial barrier disruption. (A) Cells were treated with thrombin (1 U/ml, 15 min) or PMA (100 nM, 30 min) with unstimulated cells (Unstim) used as controls. Cytosolic (C) and membrane (M) fractions were immunoblotted with various anti-PkC isoform-specific antibodies. Both thrombin and PMA increased PkCD in the membrane fraction and correspondingly decreased it in the cytosol fraction thereby indicating that PkCD is activated and translocated to membranes. (B) Cells were treated as Fig 1A, Membrane fractions were immunoblotted with PkCD and VE-Cadherin specific antibodies. (C) Normalized densitometry of PkCD in membrane fraction is shown (n > 3/condition, * p < 0.05). (D) Human pulmonary artery EC were grown to full confluency on gold microelectrodes and then treated with 100-1000 nM BisA, a putative activator of PkCD. BisA induced a dose-dependent decrease in TER corresponding to increased permeability. Representative traces are shown. (E) TER values in BisA-treated cells at 180 min are shown (n = 4/condition, * p < 0.05 compared to control).",
           "Fig 2. Effects of rottlerin pretreatment and expression of dominant negative PkCD on thrombin-induced endothelial barrier dysfunction. (A) EC monolayer leakiness was assessed by measuring TER in EC pretreated with rottlerin (10 uM, 30 min), a PkCD inhibitor, followed by thrombin stimulation (1 U/ml, 15 min). Error bars = + SE. (B) EC were infected with either control adenovirus (AdLacZ) or Ad PkCD-DN (MOI: multiplicity of infection). After 3 d, cells were lysed and immunoblotted. (C) Infected EC (MOI = 25) were then challenged with thrombin (1 U/ml) and TER (normalized to basal resistance) measured over time. Error bars = + SE. (D) Maximum decrease in TER (normalized to basal resistance) in EC treated with either rottlerin or infected with Ad PkCD-DN is shown. (n = 6/condition, * p < 0.05).",
           "Fig 3. Inhibition of PkCD abrogates thrombin-induced stress fiber formation. Untreated (top left panel), rottlerin pretreated (10 u;M, 30 min; top right panel), thrombin treated (bottom left panel), and rottlerin pretreated EC exposed to thrombin (bottom right panel) were fixed, permeabilized, stained for F-actin with Texas-Red phalloidin and imaged by fluorescence microscopy (representative images shown).",
           "Fig 4. Effect of PkCD inhibition on thrombin-induced MLC phosphorylation. (A) EC were pretreated with rottlerin (5-10 uM, 30 min) and subsequently stimulated with thrombin (1U/ml, 5 min). Whole cell lysates were immunoblotted with anti-phospho-MLC (Thr18/Ser19) antibody. (B) Relative MLC phosphorylation by densitometry is shown (n > 3/condition, * p < 0.05).",
           "Fig 5. Effect of PkCD inhibition on thrombin-induced changes in Rho and Rac GTPase activity. EC were pretreated with rottlerin (10 uM, 30 min) followed by exposure to thrombin (1 U/ml, 5 min). Unstimulated cells (Un) were used as controls. (A) Membrane fractions were immunoblotted with RhoA and VE-Cadherin specific antibodies. (B) Normalized densitometry of RhoA in membrane fraction is shown. (C) Activated RhoA (RhoA-GTP) was determined as described in the methods. (D) Normalized RhoA activity based on densitometric analysis is shown. (E) Using the same experimental conditions, Rac1 activation (Rac1-GTP) was determined as described in the methods. (F) Normalized Rac1 activity based on densitometric analysis is shown. n> 3/condition, * p < 0.05.",
           "Fig 6. Effect of rottlerin pretreatment on thrombin-induced PkD and CPI-17 phosphorylation. (A) EC were pretreated with rottlerin (10 uM, 30 min) followed by stimulation with thrombin (1U/ml, 5 min) and whole cell lysates were immunoblotted with anti-phospho-PkD and anti-PkD antibody. Unstimulated cells (Un) were used as controls. (B) Normalized densitometry analysis of PkD phosphorylation is shown (n > 3/condition, * p < 0.05). (C) In subsequent experiments using the same conditions, whole cell lysates were immunoblotted with anti-phospho-CPI-17 and anti-CPI-17 antibody. (D) Normalized densitometry of CPI-17 phosphorylation is shown. n > 3/condition, * p < 0.05.",
           "Fig 7. PkCD silencing attenuates thrombin-induced permeability increase and phosphorylation of MLC, PkD and CPI-17. (A) EC were transfected with 100 nM small interfering RNA of PkCD (siPkCD) or scramble RNA control (siCTRL). 72 h later whole cell lysates were immunoblotted with antibodies as indicated. (B) EC transfected with PkCD or control siRNA, were treated with thrombin (1 U/ml) for 1 h, and permeability was measured, the plot shows the fluorescence intensity of FITC-dextran (n = 3, * p < 0.05). (C) EC transfected with PkCD or control siRNA were treated with thrombin (1 U/ml) for 10 min, whole cell lysates were immunoblotted with antibodies as indicated. (D) Normalized densitometry of MLC, PkD and CPI-17 phosphorylation is shown (n > 3/condition, * p < 0.05).",
           "Fig 8. Proposed roles of PkCD in thrombin-induced human lung EC barrier disruption through a pathway involving MLC and Rho GTPase. Thrombin activates PkCD that then leads to the activation of PkD and CPI-17. In turn, PkD activates the RhoA pathway inducing MLC activation, while PkCD phosphorylates CPI-17, a potent MLCP inhibitor, thereby resulting in increased MLC phosphorylation. MLC phosphorylation mediates actin stress fiber formation and contraction. In addition, PkCD inhibits Rac1 activity blocking its barrier protective effects upon thrombin stimulation."
           ]

non_unicoe = "-"

PLOS_40 = ["Change is a fundamental ingredient of interaction patterns in biology, technology, the economy, and science itself: Interactions within and between organisms change; transportation patterns by air, land, and sea all change; the global financial flow changes; and the frontiers of scientific research change. Networks and clustering methods have become important tools to comprehend instances of these large-scale structures, but without methods to distinguish between real trends and noisy data, these approaches are not useful for studying how networks change. Only if we can assign significance to the partitioning of single networks can we distinguish meaningful structural changes from random fluctuations. Here we show that bootstrap resampling accompanied by significance clustering provides a solution to this problem. To connect changing structures with the changing function of networks, we highlight and summarize the significant structural changes with alluvial diagrams and realize de Solla Price's vision of mapping change in science: studying the citation pattern between about 7000 scientific journals over the past decade, we find that neuroscience has transformed from an interdisciplinary specialty to a mature and stand-alone discipline.",
           "Figure 1. Significance clustering of networks. The standard approach to cluster networks is to minimize an objective function over possible partitions of the network, as in the left side of the diagram. By repeated resampling of the weighted links from the original network, we create a 'bootstrap world' of resampled networks. By clustering these as well, and comparing to the clustering of the original network, we can estimate the degree of support that the data provide in assigning each node to a cluster. In the bottom network, the darker nodes are clustered together in at least 95% of the 1000 bootstrap networks.",
           "Figure 2. Mapping change in networks. An alluvial diagram (bottom), with clusters ordered by size, reveals changes in network structures over time. Here the height of each block represents the volume of flow through the cluster, with significant subsets in darker color. The orange module merges with the red module, but the nodes are not clustered together in 95% of the bootstrap networks. The blue module splits, but the significant nodes in the blue and purple modules are clustered together in more than 5% of the bootstrap networks. With a 5% significance threshold, neither change is significant.",
           "Figure 3. Mapping change in science. This set of scientific fields show the major shifts in the last decade of science. Each significance clustering for the citation networks in years 2001, 2003, 2005, and 2007 occupies a column in the diagram and is horizontally connected to preceding and succeeding significance clusterings by stream fields. Each block in a column represents a field and the height of the block reflects citation flow through the field. The fields are ordered from bottom to top by their size with mutually nonsignificant fields placed together and separated by half the standard spacing. We use a darker color to indicate the significant subset of each cluster. All journals that are clustered in the field of neuroscience in year 2007 are colored to highlight the fusion and formation of neuroscience.",
           "Figure 4. Significance clustering and alluvial diagram for mapping change in large networks. By repeatedly resampling of the weighted links from the original networks, we create 'bootstrap worlds' of 1000 resampled networks. By clustering these bootstrap networks, and comparing to the clustering of the original networks, we can estimate the degree of support that the data provide in assigning each node to a cluster. In the bottom networks, the darker colors represent nodes that are clustered together in at least 95% of the 1000 bootstrap networks. The alluvial diagram highlights and summarizes the structural changes between the time 1 and time 2 significance clusters. The height of each block represents the volume of flow through the cluster. The clusters are ordered from bottom to top by their size, with mutually nonsignificant clusters placed together and separated by a third of the standard spacing. The orange module merges with the red module, but the nodes are not clustered together in 95% of the bootstrap networks. The blue module splits, but the significant nodes in the blue and purple modules are clustered together in more than 5% of the bootstrap networks. Neither change is significant."]

# http://www.nature.com/ncomms/2016/160720/ncomms12246/full/ncomms12246.html
# 1,2,4,3
NA_COM_1 = ["Magnetic interactions in topological insulators cause essential modifications in the originally mass-less surface states. They offer a mass gap at the Dirac point and/or largely deform the energy dispersion, providing a new path towards exotic physics and applications to realize dissipation-less electronics. The nonequilibrium electron dynamics at these modified Dirac states unveil additional functions, such as highly efficient photon to spin-current conversion. Here we demonstrate the generation of large zero-bias photocurrent in magnetic topological insulator thin films on mid-infrared photoexcitation, pointing to the controllable band asymmetry in the momentum space. The photocurrent spectra with a maximal response to the intra-Dirac-band excitations can be a sensitive measure for the correlation between Dirac electrons and magnetic moments.",
            "Figure 1: Schematic energy dispersion and transport properties. (a) Electronic structure of magnetic topological insulator: the mass gap 2D at the Dirac point, and the effects of in-plane magnetic field above the critical value Bc inducing finite in-plane portion of the dopant spin Sy. (b) Experimental setup with the coordinates used in this work. The film is illuminated at normal incidence, and the photocurrent along x direction is measured. (c) Longitudinal and (d) Hall resistance of the Cr0.3(Bi0.22Sb0.78)1.7Te3/InP(111) film, indicating the magnetic transition temperature TC and ferromagnetic response of the surface Dirac states with uniaxial anisotropy.",
            "Figure 2: Photocurrent characteristics of the magnetic topological insulator thin film. (a) Zero-bias photocurrent at 250 meV (and 350 nJ) excitation under varying in-plane magnetic field By. (b) Normalized photocurrent under in-plane (By) and out-of-plane (Bz) magnetic fields, plotted together with the magnetization under By. (c) Photocurrent spectra at 20 k under By.",
            "Figure 3: Temperature dependence of the zero-bias photocurrent. (a) Normalized zero-bias photocurrent under in-plane magnetic field By at varying temperature. (b) Magnetization under By. (c) Photocurrent at +0.5 T as a function of temperature, plotted together with magnetization.",
            "Figure 4: Mechanism of zero-bias photocurrent generation. Schematics for the energy dispersion in the magnetic topological insulator for the out-of-plane Sz (a) and in-plane Sy (b,c) alignment of dopant spin. Corresponding energy contours calculated from equation (1) are depicted in (d,e). Possible imbalance in the photoexcitation at the opposite kx is shown in b: grey and orange arrows for low- and high-photon-energy excitations, respectively. The scattering between the surface and bulk states is shown in c."]

# http://www.nature.com/ncomms/2016/160720/ncomms12195/full/ncomms12195.html
# fig 1   1,2,3,4
NA_COM_2 = ["New exotic phenomena have recently been discovered in oxides of paramagnetic Ir4+ ions, widely known as 'iridates'. Their remarkable properties originate from concerted effects of the crystal field, magnetic interactions and strong spin-orbit coupling, characteristic of 5d metal ions. Despite numerous experimental reports, the electronic structure of these materials is still challenging to elucidate, and not attainable in the isolated, but chemically inaccessible, [IrO6]8- species (the simplest molecular analogue of the elementary {IrO6}8- fragment present in all iridates). Here, we introduce an alternative approach to circumvent this problem by substituting the oxide ions in [IrO6]8- by isoelectronic fluorides to form the fluorido-iridate: [IrF6]2-. This molecular species has the same electronic ground state as the {IrO6}8- fragment, and thus emerges as an ideal model for iridates. These results may open perspectives for using fluorido-iridates as building-blocks for electronic and magnetic quantum materials synthesized by soft chemistry routes.",
            "Figure 1: Oxido-iridates versus fluorido-iridates. Ball and stick representations of (a) the oxido IrIV-based layer in Sr2IrO4, (b) its smallest {IrO6}8- unit and the isoelectronic fluoride counterpart, [IrF6]2-. (c) Energy level diagram for IrIV (5d5; t2g5 electronic configuration). The octahedral ligand field splits the 5d orbitals into eg and t2g levels, and SO coupling further lifts the degeneracy of the t2g levels into filled jeff=3/2 and half-filled jeff=1/2 levels. Note that this one-electron picture is equivalent to the SO splitting of the 2T2g (Oh) term into an Jeff=1/2 ground state and excited Jeff=3/2 state.",
            "Figure 2: X-ray structure view of the molecular iridates. Thermal ellipsoid plots of 1 (a), 2 (b) and 3 (c) are shown at 80% probability level. The counterions for 1 and 3 and the auxiliary parts of the 1-vinylimidazole ligands in 2 are omitted for clarity. Colour code: Ir, yellow; Zn, grey; Cl, dark green; F, pale green; O, red; N, blue; H, white. Selected bond lengths (A) and angles  for 1: Ir-F 1.9339(8)-1.9510(8), F-Ir-Fcis 88.89(4)-91.35(4); for 2: Ir-F 1.9281(1), 1.9449(1) (only two crystallographically different bond lengths), F-Ir-Fcis 90, 90.0112(1); for 3: Ir-Cl 2.3205(3)-2.3370(3), Cl-Ir-Clcis 88.54(1)-91.47(1).",
            "Figure 3: X-ray spectroscopy. X-ray spectra of 1-3 showing the isotropic XAS (positive values) and XMCD (negative values) obtained in a magnetic field of +17 T (at 2.7, 2.6 and 2.5 k for 1, 2 and 3 respectively). The spectra of 2 and 3 were shifted horizontally (35 and 70 eV, respectively) and vertically for clarity. The filled patterns are the integrals used for the sum rule analysis. Inset: Field dependence of the magnetization, M versus u0HT-1, of a polycrystalline sample of 2 at T=2.0 k. 'XMCD' designates the field dependence of the XMCD maximum signal at the Ir L3 edge at T=2.6 k. The green line is the best fit of the magnetization and scaled XMCD data to the Brillouin function.",
            "Figure 4: Magnetization dynamics. Two-dimensional frequency/temperature maps of the real (X'T, a) and imaginary (XT, b) components of the a.c. susceptibility-temperature product for a polycrystalline sample of 2 obtained under u0H=75 mT. Left part: representative 4-k data of the X'T and X''T frequency dependence with the solid black line being the best fit to the generalized Debye model27, that was used for each temperature to determine the relaxation time (tho) shown as green dots on the contour plots. The best fit of tho-1 versus T is shown as solid white lines (see text). Dashed lines are guides for the eyes.",
            ]

# http://www.nature.com/ncomms/2016/160722/ncomms12199/full/ncomms12199.html
# fig 8   8,7,6,1,4,5,3,2
NA_COM_3 = ["Orexins are associated with drug relapse in rodents. Here, we show that acute restraint stress in mice activates lateral hypothalamic (LH) orexin neurons, increases levels of orexin A and 2-arachidonoylglycerol (2-AG) in the ventral tegmental area (VTA), and reinstates extinguished cocaine-conditioned place preference (CPP). This stress-induced reinstatement of cocaine CPP depends on type 1 orexin receptors (OX1Rs), type 1 cannabinoid receptors (CB1Rs) and diacylglycerol lipase (DAGL) in the VTA. In dopaminergic neurons of VTA slices, orexin A presynaptically inhibits GABAergic transmission. This effect is prevented by internal GDP-B-S or inhibiting OX1Rs, CB1Rs, phospholipase C or DAGL, and potentiated by inhibiting 2-AG degradation. These results suggest that restraint stress activates LH orexin neurons, releasing orexins into the VTA to activate postsynaptic OX1Rs of dopaminergic neurons and generate 2-AG through a Gq-protein-phospholipase C-DAGL cascade. 2-AG retrogradely inhibits GABA release through presynaptic CB1Rs, leading to VTA dopaminergic disinhibition and reinstatement of cocaine CPP.",
            "Figure 1: Orexin A depressed IPSCs in VTA slices through OX1Rs, but not OX2Rs, via a presynaptic mechanism. (a-c) Time courses of the effects of orexin A on IPSC amplitude in representative slices (a) alone, or in combination (b) with SB-334867, an OX1R antagonist, or (c) with TCS-OX2-29, an OX2R antagonist. Inward IPSCs evoked at 0.05 Hz were recorded at -70 mV with a kCl-based internal solution in the presence of 2 mM kynurenic acid, and every three IPSCs were averaged. Shown are representative traces taken at the indicated time points. (d) Summarized effects of orexin A (100 nM) on IPSC amplitude without (black bars) or with 3 uM SB-334867 (n=7, P<0.001 ; t(6)=6.818) or 30 uM TCS-OX2-29 (n=7, P=0.0814; t(6)=2.091). Ten IPSCs at 10-15 min after drug application, when the response had stabilized, were averaged. The averaged amplitude of 10 IPSCs before drug treatment in each neuron was taken as 100% baseline. Data are presented as mean+s.e.m. The number of recorded neurons is denoted above each bar. Grouped bars represent the data from the same group with different treatments conducted sequentially. **P<0.01, ***P<0.001 versus 100% (one sample t-test); ###P<0.001 versus orexin A alone: n=10 (paired t-test). (e) The effect of orexin A (100 nM) on the PPR of IPSCs. Paired IPSCs were evoked by 50 ms-separated paired pulses every 20 s. Left panel: a representative paired IPSCs recorded in a neuron before and 10 min after orexin A treatment. Scale bars, 50 ms and 200 pA. Right panel: average PPR of IPSCs in four neurons recorded before (control) and after orexin A treatment. (n=4, P=0.026; t(3)=4.089). *P<0.05 versus control (paired t-test). Data are expressed as mean+s.e.m.",
            "Figure 2: Orexin A decreased the frequency, but not the amplitude, of mIPSCs. (a) Representative traces of mIPSCs recorded in the presence of 2 mM kynurenic acid and 1 uM tetrodotoxin in a neuron before (upper panel) and 10 min after (lower panel) treatment with 100 nM orexin A. Scale bars, 0.5 s and 50 pA. (b,c) Cumulative probability of the interval or amplitude of mIPSCs, sampled in a 5-min period, before and 10 min after treatment with 100 nM orexin A (black bars). Insets: (b) the average frequency (n=4, P=0.0028 ; t(3)=9.127) and (c) amplitude (n=4, P=0.1142 ; t(3)=2.209) of mIPSCs before and 10 min after treatment with orexin A in the same neurons. **P<0.01 versus control (paired t-test). Data are expressed as mean+s.e.m.",
            "Figure 3: Orexin A-induced IPSC depression was reversed by a CB1R antagonist, and was mimicked and occluded by a CB1R agonist. (a-c) The time course of the effect of (a) orexin A (100 nM) or (b) WIN 55,212-2 (3uM) on IPSC amplitude before and after further treatment with AM 251 (3 uM), or with (c) orexin A. (d) The effect of orexin A (black bar) or WIN 55,212-2 (grey bars) alone or in combination with AM 251 (slashed bars; n=7, P<0.001 ; t(6)=-10.433 versus n=4, P=0.0481; t(3)=-3.232) or WIN 55212-2+orexin A (crossed bars; n=7, P=0.569 ; t(6)=-0.602) on IPSC amplitude. Scale bars, 50 ms and 200 pA. *P<0.05, **P<0.01, ***P<0.001 versus 100% (One sample t-test); #P<0.05, ###P<0.001 versus Orexin A alone (paired t-test). Data are expressed as mean+s.e.m.",
            "Figure 4: Orexin A-induced IPSC depression was prevented by inhibitors of G-proteins, PLCB or DAGL, and was enhanced by a MAGL inhibitor. (a-c) The time course of the effect of orexin A on IPSC amplitude in slices pretreated with (a) GDP-B-S, a non-selective G protein inhibitor that was applied intracellularly through the patch pipette, (b) edelfosine (a PLCB inhibitor) or (c) THL (a DAGL inhibitor). Scale bars, 50 ms, and (a) 400 pA, (b) 100 pA and (c) 200 pA. (d) The time course of the effect of orexin A on IPSC amplitudes alone (filled circles) or in the presence of 1 uM JZL184(open squares), a selective inhibitor of MAGL, which is the major catabolic enzyme of 2-AG. (e) Summarized effects of orexin A on IPSC amplitude in the absence (n=9, black bar) or presence of 1 mM GDP-B-S (hatched bar; n=7, P=0.005 ; t(14)=-3.378), 5 uM edelfosine (right slashed grey bar; n=7, P=0.038; t(14)=2.286), 10 uM THL (left slashed grey bar; n=7, P<0.001 ; t(14)=-4.755) or 1 uM JZL184 (horizontal-lined bar; n=7, P<0.001 ; t(14)=-4.382), and the effect of the inhibitor alone (left or right slashed bar). ***P<0.001 (one sample t-test); #P<0.05, ##P<0.01, ###P<0.001 versus the group treated with orexin A alone (Student's t-test). Data are expressed as mean+s.e.m.",
            "Figure 5: Orexin A increased the firing rate of VTA dopaminergic neurons through CB1Rs. (a,b) The firing activity in a representative neuron recorded in the (a) whole cell or (b) cell-attached recording mode before and 10 min after treatment with 100 nM orexin A, and 10 min after further treatment with 3 uM AM 251. Scale bars, (a) 500 ms and 20 mV; (b) 5 s and 20 pA (c) The time course of the effect of orexin A before and after further treatment with AM 251 on the firing rate of a representative neuron recorded by cell-attached mode (d) Effects of orexin A alone or in combination with AM 251 on the average firing rate of VTA dopaminergic neurons. Neuronal firing activity was assessed in the whole cell and cell-attached recording mode. Note that 100 nM orexin A (black bar) significantly increased the average firing rate of VTA dopaminergic neurons, compared with baseline firing rate (open bar) and this effect was reversed by 3 uM AM 251 (gray). n=7, *P<0.05, **P<0.01 (repeat measures one-way ANOVA with Dunns post hoc test). Data are expressed as mean+s.e.m.",
            "Figure 6: Restraint stress or intra-VTA microinjection of orexin A reinstated extinguished cocaine-CPP in WT, but not in Cnr1 kO mice or in mice pretreated with an OX1R or CB1R antagonist. (a) The protocol for inducing, extinguishing, and reinstating cocaine-CPP. Four stages were analysed: preconditioning1, conditioning35, extinction and reinstatement. Mice received daily pairing of cocain (i.p. 20 mg kg-1) on days 2-4 to develop cocaine CPP, followed by a 3-day forced extinction training by daily paring of saline with the cocaine-paired chamber on days 6-8. (b-e) On day 10, (b-d) a 30-min restraint stress or (e) bilatera intra-VTA microinjection (i.vta.) of 0.1 nmol orexin A for 30 min was applied before the CPP test to reinstate extinguished cocaine CPP. The CPP test was conducted befor (day 1, PRE) and after (day 5, CON) cocaine-conditioning, after extinction (day 9, extinction) and after restraint stress/i.vta. orexin A (day 10, reinstatement) Antagonists were given 30 min before restraint stress by either i.p. injection or bilateral i.vta. microinjection The degree of cocaine seeking was determined by the CPP score as described in Methods section (b-d) CPP scores at various stages (b) in WT (n=10, black bars), Cnr1+/- (n=10, grey bars) or Cnr1-/- (n=8, open bars) mice, or in mice (c) i.p. or (d) i.vta. pretreated with vehicle (i.p.: n=7, i.vta: n=6, black bars), SB-334867 (i.p.: n=9, i.vta: n=6, left slashed bars), AM 251 (i.p.: n=7, i.vta: n=6, grey bars) or THL (n=6, right slashed bars) before receiving restraint stress. (e) CPP scores at various stages in mice, after extinction, i.vta. microinjection of orexin A (n=7, black bars), saline (n=7, open bars), orexin A+AM 251 (n=4, grey bars) or orexin A+vehicle (n=4, right slashed grey bars). *P<0.05, **P<0.01, ***P<0.01 versus the PRE stage ; #P<0.05, ##P<0.01, ###P<0.001 versus (b-d) the WT, or the vehicle or (e) the saline group (two-way ANOVA with Bonferroni post hoc test). Data are expressed as mean+s.e.m.",
            "Figure 7: Restraint stress increased the number of c-fos-expressing orexin neurons in the LH and orexin A levels in the VTA. (a,b) A representative hypothalamic section taken at the level of (a) the LH and PeF/DMH or (b) the LH (reproduced from ref. 52) from a pre-conditioned mouse. Scale bars:  (a) 3300 um, (b) 20 um. Orexin A (OX+) was labelled by DAB (ref. 18) and c-fos (Fos+) was labelled by DAB with nickel (black in nucleus). Arrows indicate neurons doubly immunoreactive for orexin A and c-fos (Fos+OX+). Arrowheads indicate orexin neurons without c-fos staining. 4 V: the fourth ventricle. (c) The number of Fos+OX+ neurons as a percentage of total OX+ neurons, which reflects the percentage of activated orexin neurons, in the LH or PeF/DMH at each stage (PRE: n=3; CON: n=3; extinction: n=9; reinstatement: n=9, *P<0.05; **P<0.01, one-way ANOVA with Newman-keuls post hoc test; #P<0.05, paired t-test). (d) The association between the number of activated LH orexin neurons and the degree of reinstatement, expressed by the CPP score difference between reinstatement and extinction stages. Note that there is a positive linear correlation between the number of activated orexin neurons and the degree of reinstatement of cocaine CPP in mice whether (green dots) or not (blue dots) restraint stress reinstated cocaine CPP. n=9, r=0.787, P<0.05. (e) The average level of orexin A in the VTA homogenate at each stage of cocaine CPP. (PRE: n=15; CON: n=14; extinction: n=14; reinstatement: n=14, *P<0.05; **P<0.01; ***P<0.001, one-way ANOVA with Newman-keuls post hoc test). Data are expressed as mean+s.e.m.",
            "Figure 8: A schematic illustration of how stress-released orexins induce an endocannabinoid-mediated relapse for cocaine-seeking. This scheme was illustrated by using the Biology Bundle of Motifolio PPT Drawing Toolkits (Motifolio Inc, MD) to describe what may occur in the LH and the VTA (a) before stress and (b) during stress. Right boxes are enlarged portions of synaptic events occurring in a GABAergic synapse onto a dopaminergic neuron in the VTA. During stress, LH orexin neurons are activated and release orexins. The released orexins then activate postsynaptic OX1 receptors on dopaminergic neurons in the VTA. Activation of the OX1 receptor, a Gq-protein coupled receptor, leads to PLC activation, generating DAG that is converted into 2-AG, an endocannabinoid, by DAGL. 2-AG travels retrogradely across the synapse to inhibit GABA release by activating presynaptic CB1 receptors on the GABAergic terminal. Inhibition of GABAergic synaptic neurotransmission onto dopaminergic neurons in the VTA results in activation of the mesolimbic dopaminergic circuitry, leading to a reinstatement of extinguished cocaine CPP, modelling cocaine relapse in humans. Finally, 2-AG is degraded by MAGL, which is located in GABAergic terminals. Thus, the MAGL inhibitor potentiates and prolongs orexin A-induced IPSC depression."]

# http://www.nature.com/ncomms/2016/160721/ncomms12277/full/ncomms12277.html
# fig1  6,2,4,3,5,1
NA_COM_4 = ["Nanoparticle-mediated delivery of functional macromolecules is a promising method for treating a variety of human diseases. Among nanoparticles, cell-derived exosomes have recently been highlighted as a new therapeutic strategy for the in vivo delivery of nucleotides and chemical drugs. Here we describe a new tool for intracellular delivery of target proteins, named 'exosomes for protein loading via optically reversible protein-protein interactions' (EXPLORs). By integrating a reversible protein-protein interaction modul controlled by blue light with the endogenous process of exosome biogenesis, we are able to successfully load cargo proteins into newly generated exosomes. Treatment with protein-loaded EXPLORs is shown to significantly increase intracellular levels of cargo proteins and their function in recipient cells in vitro and in vivo. These results clearly indicate the potential of EXPLORs as a mechanism for the efficient intracellular transfer of protein-based therapeutics into recipient cells and tissues.",
            "Figure 1: Generation of engineered EXPLOR. (a) Schematics of DNA constructs used for the production of EXPLOR. (b) Schematic showing fusion proteins and their proposed action. (c) HEk293T cells were transiently transfected with CIBN-EGFP-CD9 and mCherry-CRY2 expression vectors. The mCherry fluorescence was imaged before and after 488-nm laser stimulation (15 s in duration, 350 uW cm-2). Scale bars, 20 um (5 um for inset images). A representative result from at least 10 experiments. (d) HEk293T cells transiently transfected with CIBN-EGFP-CD9 and mCherry-CRY2 were imaged for time-lapse imaging of mCherry fluorescence for varying time periods (0-12 min) after a stimulation (black arrow of 488-nm light (15 s in duration, 350 uW cm-2). Scale bars, 5 um. A representative result of at least 10 experiments. (e) Quantification of mCherry fluorescence in the cytoplasm and at the plasma membrane. Data are presented as the mean+s.e.m. (n=3).",
            "Figure 2: Light-dependent loading of target proteins in EXPLORs. (a) Cells transiently transfected with CIBN-EGFP-CD9 and mCherry-CRY2 expression vectors were maintained under blue light illumination of varying powers for 48 h. Cell-derived exosomes were subject to immunoblot analysis using antibodies against mCherry, EGFP and CD63, an exosome marker. A representative result from three independent experiments. (b) The graph presents densitometry analysis for normalized amount of mCherry-CRY2 protein over CIBN-EGFP-CD9 protein from three independent experiments. Data are presented as the mean+s.e.m. (n=3), and Tukey's post hoc test was applied to significant group effects (**P<0.01, ***P<0.001) identified by analysis of variance.",
            "Figure 3: Comparison of the exosome-loading capacity of target proteins between various protein-loading methods. (a) HEk293T cells were transiently transfected with luciferase-mCherry-CRY2 expression vector alone, XPack-luciferase-mCherry expression vector or co-transfected with CIBN-EGFP-CD9 and luciferase-mCherry-CRY2 expression vectors After 24 h, cells were imaged by fluorescence microscopy for the expression profile of mCherry fusion proteins. A representative result from five independent experiments. Scale bars, 20 um. (b) Quantification of mCherry fluorescence Data are presented as the mean+s.e.m. (n=5), and Tukey's post hoc test was applied to significant group effects identified by analysis of variance (ANOVA). Control: untransfected HEk293T cells; OVER: cells transiently transfected with a luciferase-mCherry-CRY2 vector; XP: cells transfected with an XPACk-luciferase-mCherry vector; and EXPLOR: cells transfected with both luciferase-mCherry-CRY2 and CIBN-EGFP-CD9 vectors. (c) Cells transiently transfected with various vectors were maintained for 48 h. In the case of EXPLOR-producing cells, cells were maintained in the absence (OFF) or presence (ON) of blue light illumination; 5 x 108 particles of the isolated exosomes were analysed for luciferase activity. Data are presented as the mean+s.e.m. (n=3), and Tukey's post hoc test was applied to significant group effects (**P<0.01, ***P<0.001) identified by ANOVA. (d) Loading efficiency of various protein-loaded exosomes was calculated by dividing the number of luciferase molecules in exosomes with the number of luciferase molecules in the exosome-producing cells. Numbers of luciferase molecules were estimated from a standard curve using recombinant luciferase. Data are presented as the mean+s.e.m. (n=3), and Tukey's post hoc test was applied to significant group effects (***P<0.001) identified by ANOVA. NS, not significant.",
            "Figure 4: EXPLOR-mediated intracellular delivery of cargo proteins. (a,b) HeLa cells were incubated in the absence or presence of 5 x 109 particles of various isolated exosomes for 24 h and imaged by fluorescence microscopy. The fluorescence intensities of mCherry were quantified by two imaging processing tools, ImageJ and Cellprofiler. Data are presented as the mean+s.e.m. (n=15), and Tukey's post hoc test was applied to significant group effects (***P<0.001) identified by analysis of variance (ANOVA). Scale bars, 100 um. (c,d) HeLa cells were incubated in the absence or presence of 0.1 mg ml-1 mCherry:EXPLORs or Bax-mCherry:EXPLORs for 12 h, and fixed with 4% paraformaldehyde. Then, cytochrome c was stained with an antibody conjugated with Alexa Fluor 647 and imaged by confocal microscopy. The ratios of cytochrome c localization were analysed by cell counting. Data are presented as the mean+s.e.m. (n=3), and Tukey's post hoc test was applied to significant group effec (**P<0.01) identified by ANOVA. Scale bars, 20 um. (e) HeLa cells were incubated in the absence or presence of 0.1 mg ml-1 mCherry:EXPLORs or srIkB:EXPLORs for 12 h, treated with 10 ng ml-1 tumour necrosis factor-a (TNF-a) for an additional 30 min, and fixed with 4% paraformaldehyde. NF-kB p65 was stained with an antibody conjugated with Alexa Fluor 488 and imaged by confocal microscopy. (f) The nuclear extracts of cells were assayed for the DNA-binding activity of p65/c-Rel (NF-kB). Data are presented as the mean+s.e.m. (n=3), and Tukey's post hoc test was applied to significant group effects (**P<0.01) identified by ANOVA. Scale bars, 20 um. NS, not significant.",
            "Figure 5: EXPLOR-mediated delivery of Cre recombinase in vitro and in vivo. (a,b) Differentiated neurosphere-derived cells were incubated in the absence or presence of 2 x 1010 particles per ml of Cre:EXPLORs (0.16 mg ml-1) or transfected with pCMV-Cre vector for 72 h. Cells were fixed with 4% paraformaldehyde and immune-stained with antibodies against a neuron-specific class III beta-tubulin marker Tuj1, GFP and Hoechst 33342. The ratios of EGFP-expressing cells were analysed by cell counting. Data are presented as the mean+s.e.m (n=10 fields), and Tukey's post hoc test was applied to significant group effects identified by ANOVA. Scale bars, 100 um (c) An experimental scheme for the administration of Cre:EXPLORs in loxp-stop-loxp-eNpHR3.0-EYF transgenic mice. In total, 50 ul of Cre:EXPLORs (10 mg ml-1) were administered to pCAG-loxP-STOP-loxP-eNpHR3.0-EYFP transgenic mice by ventrolateral injection (d) Brain slices of EXPLOR-injected transgenic mice were fixed with 4% formaldehyde and imaged by fluorescence microscopy Green fluorescence indicates eNpHR3.0-EYFP protein expression and blue fluorescence indicates cell nuclei Inset images showed confocal microscopy images of the detailed cellular eNpHR3.0-EYFP expression in EXPLOR-administered mouse neurons of the zona incerta (ZI) region Scale bars, 500 um (50 um for inset confocal images). Hip, hippocampus; Th, thalamus. A representative of two independent experiments. (e) Representative image of NeuN/GFAP immunohistochemistry of the brain. Pink, neuronal-specific nuclear protein (NeuN)-positive neurons; Red, glial fibrillary acidic protein (GFAP)-positive astrocyte cells. Objective lens, x 40. Scale bar, 20 um. NS, not significant.",
            "Figure 6: Schematic diagram of EXPLOR technology. In EXPLOR-producing donor cells, CRY2 protein was fused to a cargo protein, and CIBN was conjugated with a representative marker of exosomes, CD9 protein. Blue light illumination induces the reversible PPI between CIBN and CRY2 fusion proteins. With continuous blue light irradiation, the cargo proteins are guided to the inner surface of the cell membrane or the surface of early endosomes. Mature multi-vesicular bodies (MVBs) then readily secrete cargo protein-carrying exosomes (EXPLORs) from the cells by membrane fusion with the plasma membrane. After exocytosis, EXPLORs can be easily isolated and purified in vitro. Purified EXPLORs can be used for delivery of the cargo proteins into target cells via membrane fusion or endocytosis processes. Bottom grey boxes highlight the essential steps from EXPLORs biogenesis to target cell delivery."]


test_1 = ["Transforming growth factors-? are not good biomarkers of chemopreventive efficacy in a preclinical breast cancer model system. Using a carcinogen-initiated rat model of mammary tumorigenesis, we tested the hypothesis that transforming growth factor (TGF)-?s are useful biomarkers of chemopreventive efficacy in the breast. The chemopreventive agents tested were tamoxifen and the retinoids 9-Using a carcinogen-initiated rat model of mammary tumorigenesis, we tested the hypothesis that transforming growth factor (TGF)-?s are useful biomarkers of chemopreventive efficacy in the breast. The chemopreventive agents tested were tamoxifen and the retinoids 9-ciscis-retinoic acid (9cRA) and NN-(4-hydroxyphenyl)retinamide (4-HPR), because both antiestrogens and retinoids have previously been shown to upregulate TGF-?s in vitroin vitro. Despite demonstrable chemopreventive efficacy in this model, none of these agents, alone or in combination, had any significant impact on the expression of TGF-?s in the mammary ductal epithelium or periductal stroma as determined by immunohistochemistry. These data suggest that TGF-?s are not likely to be useful biomarkers of chemopreventive efficacy in a clinical setting.",
          "Effect of chemopreventive agents on mammary tumor formation in the NMU rat model of mammary tumorigenesis. Rats were administered NMU at 8 weeks of age (time t = 0), and were randomized into six groups which were given the following chemopreventive agents continuously in their diet, starting 1 week after initiation: no additions (unfilled circle), 4-HPR (filled circle), 9cRA (unfilled square), tamoxifen (filled square), 4-HPR + tamoxifen (unfilled triangle), or 9cRA + tamoxifen (filled triangle). Rats were weighed and palpated for tumors each week. (a) Percentage of rats with palpable tumors. (b) Mean number of tumors per rat. (c) Mean weight of rat. The open arrow indicates the start of the chemopreventive intervention.",
          "Immunohistochemical analysis of TGF- expression in the rat mammary gland. The first thoracic (number 2) mammary gland was harvested from 15-week-old rats that had been treated with NMU at age 8 weeks. Sections were immunostained (see Materials and method), with the following antisera: (a) anti-TGF-1-LC (stains predominantly intracellular TGF-1); (b) anti-TGF-1-CC (stains extracellular TGF-1); (c) anti-TGF-2; (d) anti-TGF-3; (e) anti-LTBP; and (f) normal rabbit immunoglobulin. The brown stain indicates a positive immunoperoxidase reaction. Images were shot at 630 original magnification.",
          "Lack of effect of chemopreventive agents on expression of TGF-s in the NMU-initiated rat mammary gland. Immunohistochemical staining intensity for TGF-s and LTBP was determined after 6 weeks of treatment with the following chemopreventive agents: None (C); 4-HPR (H); 9cRA (R); tamoxifen (T); 4-HPR + tamoxifen (T/H); 9cRA + tamoxifen (T/R). Staining intensity was determined independently for the ductal epithelial cells and the periductal stroma (see Materials and method). Staining intensity ranged from 0 to 4 (max). Results are given as the mean SD for five or six rats/group. TGF-1-LC is an antibody that predominantly recognizes intracellular TGF-1, whereas TGF-1-CC recognizes extracellular TGF-1.",
          "Treatment with tamoxifen affects the histology of the rat mammary gland. Representative hematoxylin and eosin stained sections of the first thoracic gland of 15-week-old rats that had undergone the following treatments: (a, b) No treatment; moderate numbers of mammary gland lobules are present containing primary, secondary and tertiary ductules, as well as developing alveoli. (c, d) Initiation with NMU at 8 weeks of age; no significant histologic differences are noted in mammary gland development from that in untreated control animals. (e, f) Initiation with NMU at 8 weeks, followed by treatment with tamoxifen from 9 to 15 weeks of age; scant numbers of atrophic primary and secondary mammary gland ductules are noted, with no alveolar bud development evident. (a, c, e) Shot at 100; and (b, d, f) shot at 400 original magnification."
]

test_2 = ["Altered expression of estrogen receptor-? variant messenger RNAs     between adjacent normal breast and breast tumor tissues. Using semiquantitative reverse transcription-polymerase chain     reaction assays, we investigated the expression of variant messenger RNAs     relative to wild-type estrogen receptor (ER)-? messenger RNA in normal     breast tissues and their adjacent matched breast tumor tissues. Higher ER     variant truncated after sequences encoding exon 2 of the wild-type ER-?     (ERC4) messenger RNA and a lower exon 3 deleted ER-? variant (ERD3)     messenger RNA relative expression in the tumor compartment were observed in the     ER-positive/PR-positive and the ER-positive subsets, respectively. A     significantly higher relative expression of exon 5 deleted ER-? varient     (ERD5) messenger RNA was observed in tumor components overall. These data     demonstrate that changes in the relative expression of ER-? variant     messenger RNAs occur between adjacent normal and neoplastic breast tissues. We     suggest that these changes might be involved in the mechanisms that underlie     breast tumorigenesis.Using semiquantitative reverse transcription-polymerase chain     reaction assays, we investigated the expression of variant messenger RNAs     relative to wild-type estrogen receptor (ER)-? messenger RNA in normal     breast tissues and their adjacent matched breast tumor tissues. Higher ER     variant truncated after sequences encoding exon 2 of the wild-type ER-?     (ERC4) messenger RNA and a lower exon 3 deleted ER-? variant (ERD3)     messenger RNA relative expression in the tumor compartment were observed in the     ER-positive/PR-positive and the ER-positive subsets, respectively. A     significantly higher relative expression of exon 5 deleted ER-? varient     (ERD5) messenger RNA was observed in tumor components overall. These data     demonstrate that changes in the relative expression of ER-? variant     messenger RNAs occur between adjacent normal and neoplastic breast tissues. We     suggest that these changes might be involved in the mechanisms that underlie     breast tumorigenesis.", 
          "Comparison of the relative expression of estrogen receptor (ER) variant truncated after sequences encoding exon 2 of the wild-type (WT) ER- (ERC4) messenger RNAs between breast tumor and adjacent matched normal breast samples. (A) Total RNA extracted from frozen tissue sections from tumor (T) and adjacent normal (N) breast tissue samples was reverse transcribed and polymerase chain reaction (PCR) was amplified using ERU, ERL and C4L primers (see text). Radioactive PCR products were separated on a 6% acrylamide gel and visualized by autoradiography. Bands migrating at 149 and 536 base pairs were identified as corresponding to WT-ER and ERC4 variant messenger RNA, respectively. C, negative control (no complementary DNA added during the PCR reaction). (B) For each case, signals corresponding to ERC4 variant messenger RNA were quantified (see text) and expressed in arbitrary units for tumor (black column) and normal (white column) components. For each sample, the mean and the standard deviation of at least three different PCR assays are indicated. Cases are sorted by ER status (black bottom lane) and progesterone receptor (PR) status (gray bottom lane). The significance of the differences between tumor and normal matched components within each subgroup, as tested using the Wilcoxon matched-pair test, is indicated where P <0.05. M, molecular weight marker (fx174 Haelll digest, Gibco BRL, Grand Island, New York, NY).",
          "Comparison of the relative expression of exon 3 deleted estrogen receptor (ER) variant (ERD3) messenger RNA between breast tumor and adjacent matched normal breast samples. (A) Total RNA extracted from frozen tissue sections from tumor (T) and adjacent normal (N) breast tissue samples was reverse transcribed and polymerase chain reaction (PCR) was amplified using D3U and D3L primers. Radioactive PCR products were separated on a 6% acrylamide gel and visualized by autoradiography. Bands migrating at 354 and 237 base pairs were identified as corresponding to wild-type (WT)-ER and ERD3 variant messenger RNA, respectively. C, negative control (no complementary DNA added during the PCR reaction). (B) For each case, signals corresponding to ERD3 variant messenger RNA were quantified and expressed in arbitrary units for tumor (black column) and normal (white column) components. For each sample, the mean and the standard deviation of at least three different polymerase chain reaction (PCR) assays are indicated. Cases are sorted by ER status (black bottom lane) and progesterone receptor (PR) status (gray bottom lane). Samples that failed to have three measurable signals in the four experiments performed in both normal and neoplastic components were not included in the statistical analysis. The significance of the differences between tumor and normal matched components within each subgroup, as tested using the Wilcoxon matched-pair test, is indicated where P <0.05. M, molecular weight marker.",
          "Comparison of the relative expression of exon 3 deleted estrogen receptor (ER) variant (ERD3) messenger RNA between breast tumor and adjacent matched normal breast samples. (A) Total RNA extracted from frozen tissue sections from tumor (T) and adjacent normal (N) breast tissue samples was reverse transcribed and polymerase chain reaction (PCR) amplified using D5U and D5L primers. Radioactive PCR products were separated on a 6% acrylamide gel and visualized by autoradiography. Bands that migrated at 483 and 344 base pairs were identified as corresponding to wild-type (WT)-ER and exon 5 deleted ER variant (ERD5) messenger RNA, respectively. C, negative control (no complementary DNA added during the PCR reaction). (B) For each case, signals corresponding to ERD5 variant messenger RNA were quantified and expressed in arbitrary units for tumor (black column) and normal (white column) components. For each sample, the mean and the standard deviation of at least three different PCR assays are indicated. Cases are sorted by ER status (black bottom lane) and progesterone receptor (PR) status (gray bottom lane). Samples that failed to have three measurable signals in the four experiments performed in both normal and neoplastic components were not included in the statistical analysis. The significance of the differences between tumor and normal matched components within each subgroup, as tested using the Wilcoxon matched-pair test, is indicated where P <0.05. m, molecular weight marker."
]

# string = "abc10 ab"
# string = string.encode('ascii',errors='ignore')
# print string
# review_text = re.sub("[^a-zA-Z0-9]"," ", string)

# print review_text


stemmed = True
Model = NLPModel()

Model.create_models(PLOS_40, stemmed=stemmed)

# print Model.tokenize([PLOS_27[3]], stemmed=stemmed)

similarity = Model.get_similarity_score(PLOS_40, stemmed = stemmed)

similarity_sort = Model.sort_similarity_score(similarity)

 
for i, s in enumerate(similarity_sort):
    print s
#     
# for i, s in enumerate(similarity_sort):
#     for j, p in enumerate(s):
#         if p[0] == 0:
#             print i, j

def mds_transform(X, n_components, max_iter=100000, n_init=1):
    
    mds = manifold.MDS(n_components=2, max_iter=3000, eps=1e-9, dissimilarity="precomputed", n_jobs=1)
    return  mds.fit_transform(X)  


similarity[similarity>1] = 1
# print similarity

# distance = 1/(np.matrix(similarity) + 0.0001)
# for i in range(len(similarity_sort)):
#     distance[i,i] = 0
    



# MDS map
# distance = np.max(similarity) - np.matrix(similarity)
# for i, pos in enumerate(similarity):
#     distance[i,:] = np.max(pos) - pos
# 
# print distance
# 
# Y = mds_transform(distance, 2)
# print Y
# fig = plt.figure()
# ax = fig.add_subplot(111)
# ax.scatter(Y[:, 0], Y[:, 1])
# 
# lable = ['abstract', 'figure1', 'figure2', 'figure3', 'figure4', 'figure5', 'figure6', 'figure7', 'figure8', 'figure9', 'conclusion']
# for i, pos in enumerate(Y):
#     ax.text(pos[0], pos[1], lable[i])
# 
# plt.show()
# 
# fig = plt.figure()
# ax = fig.add_subplot(111)
# ax.scatter(pos[:, 0], pos[:, 1])
# 
# for i, p in enumerate(pos):
#     ax.text(p[0], p[1], i)
        







# texts = Model.tokenize(tt)
# Model.creat_dictionary(texts)
# corpus = Model.text2corpus(texts)
# Model.create_tfidf_model(tt)
# tfidf_corpus = Model.get_tfidf_corpus(corpus)
# Model.create_similarity_index_by_corpus(corpus)



# texts =  tokenize(tt)
# dictionary = corpora.Dictionary(texts)
# dictionary.save('/Users/sephon/Downloads/nlp/deerwester.dict')
# dictionary = corpora.Dictionary.load('/Users/sephon/Downloads/nlp/deerwester.dict')
# 
# corpus = [dictionary.doc2bow(text) for text in texts]
# 
# 
# tfidf = models.TfidfModel(corpus)
# corpus_tfidf = tfidf[corpus]
# # for doc in corpus_tfidf:
# #     print doc
#     
# len(dictionary)
# index = similarities.SparseMatrixSimilarity(tfidf[corpus], num_features=len(dictionary))
# 
# vec = [[(0, 1), (4, 1)], [(12, 1), (2, 1)]]
# print tfidf[vec][0]
# sims = index[tfidf[vec]]
# 
# for s in sims:
#     s = sorted(enumerate(s), key=lambda item: -item[1])
#     print(s)
#     
# print
# Model.get_similarity_score_from_corpus(vec)

# frequency = defaultdict(int)
# for text in texts:
#     for token in text:
#         frequency[token] += 1

# all_stems = sum(texts, [])
# stems_once = set(stem for stem in set(all_stems) if all_stems.count(stem) == 1)
# texts = [[stem for stem in text if stem not in stems_once] for text in texts]
# print texts


# stoplist = set(nltk.corpus.stopwords.words('english'))
# 
# texts = [[word for word in document.lower().split() if word not in stoplist] for document in documents]
# 
# frequency = defaultdict(int)
# for text in texts:
#     for token in text:
#         frequency[token] += 1
# 
# texts = [[token for token in text if frequency[token] > 1] for text in texts]
# from pprint import pprint
# 
# dictionary = corpora.Dictionary(texts)
# dictionary.save('/Users/sephon/Downloads/nlp/deerwester.dict')
# 
# print(dictionary.token2id)
# new_doc = "Human computer interaction"
# new_vec = dictionary.doc2bow(new_doc.lower().split())
# 
# corpus = [dictionary.doc2bow(text) for text in texts]
# corpora.MmCorpus.serialize('/Users/sephon/Downloads/nlp/deerwester.mm', corpus)